{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Using TensorFlow backend.\n",
      "[nltk_data] Downloading package stopwords to\n",
      "[nltk_data]     C:\\Users\\user\\AppData\\Roaming\\nltk_data...\n",
      "[nltk_data]   Package stopwords is already up-to-date!\n"
     ]
    }
   ],
   "source": [
    "import numpy as np\n",
    "import pandas as pd\n",
    "import os\n",
    "import re\n",
    "\n",
    "from sklearn.preprocessing import LabelEncoder\n",
    "from keras.utils.np_utils import to_categorical\n",
    "from sklearn.model_selection import train_test_split\n",
    "import matplotlib.pyplot as plt\n",
    "\n",
    "from gensim.models import Word2Vec\n",
    "from nltk.tokenize import word_tokenize, sent_tokenize\n",
    "from gensim.models import Phrases\n",
    "from gensim.models.doc2vec import Doc2Vec, TaggedDocument\n",
    "from gensim.models import FastText\n",
    "\n",
    "\n",
    "from keras.models import Sequential\n",
    "import gensim\n",
    "from gensim.models import FastText\n",
    "from keras.layers import Embedding,Dense,LSTM, Dropout, Flatten, Convolution1D, MaxPooling1D\n",
    "\n",
    "\n",
    "from nltk.tokenize import word_tokenize\n",
    "from nltk.corpus import stopwords\n",
    "import re\n",
    "import nltk\n",
    "nltk.download('stopwords')\n",
    "%matplotlib inline"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Fasttext Model"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "from gensim.models.wrappers import FastText\n",
    "model = FastText.load_fasttext_format('wiki.en/wiki.en.bin')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### preprocessing data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Preprocessing - removing unwanted characters, tokenization, stop-word removal\n",
    "def clean_data(txts):\n",
    "    x = re.sub(\"[^a-zA-Z]\", \" \",txts) \n",
    "    x = x.lower().split()                             \n",
    "    stops = set(stopwords.words(\"english\"))                  \n",
    "    words = [w for w in x if not w in stops] \n",
    "    \n",
    "    return( \" \".join(words))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "def clean_norm(txts):\n",
    "    x = re.sub('(OntoBiotope |)','',txts)\n",
    "    x = re.sub('(:|_)',' ',x)\n",
    "    x = re.sub('(Taxonomy)','taxon',x)\n",
    "    \n",
    "    return x"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Train_norm+Dev_norm appended (same as Train_norm+ner, Dev_norm+ner appended)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(3268, 7)\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>words</th>\n",
       "      <th>entity</th>\n",
       "      <th>norm1</th>\n",
       "      <th>norm2</th>\n",
       "      <th>norm3</th>\n",
       "      <th>norm4</th>\n",
       "      <th>norm5</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>selective broths based on hypertonic strontium...</td>\n",
       "      <td>Habitat</td>\n",
       "      <td>OntoBiotope OBT:000360</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>selective broths based on the bi-selenite ion</td>\n",
       "      <td>Habitat</td>\n",
       "      <td>OntoBiotope OBT:000360</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Salmonellae</td>\n",
       "      <td>Microorganism</td>\n",
       "      <td>NCBI_Taxonomy 590</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>egg products</td>\n",
       "      <td>Habitat</td>\n",
       "      <td>OntoBiotope OBT:001086</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>egg</td>\n",
       "      <td>Habitat</td>\n",
       "      <td>OntoBiotope OBT:001847</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                               words         entity  \\\n",
       "0  selective broths based on hypertonic strontium...        Habitat   \n",
       "1      selective broths based on the bi-selenite ion        Habitat   \n",
       "2                                        Salmonellae  Microorganism   \n",
       "3                                       egg products        Habitat   \n",
       "4                                                egg        Habitat   \n",
       "\n",
       "                    norm1 norm2 norm3 norm4 norm5  \n",
       "0  OntoBiotope OBT:000360                          \n",
       "1  OntoBiotope OBT:000360                          \n",
       "2       NCBI_Taxonomy 590                          \n",
       "3  OntoBiotope OBT:001086                          \n",
       "4  OntoBiotope OBT:001847                          "
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data = pd.read_csv('data.csv')\n",
    "\n",
    "data['norm2'].fillna(value='', inplace=True)\n",
    "data['norm3'].fillna(value='', inplace=True)\n",
    "data['norm4'].fillna(value='', inplace=True)\n",
    "data['norm5'].fillna(value='', inplace=True)\n",
    "\n",
    "data = data.drop(data[data.entity == 'Paragraph'].index)\n",
    "data = data.drop(data[data.entity == 'Title'].index)\n",
    "print(data.shape)\n",
    "data.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "data['words'] = data['words'].apply(lambda x: clean_data(x))\n",
    "\n",
    "# data['norm1'] = data['norm1'].apply(lambda x: clean_norm(x))\n",
    "# data['norm2'] = data['norm2'].apply(lambda x: clean_norm(x))\n",
    "# data['norm3'] = data['norm3'].apply(lambda x: clean_norm(x))\n",
    "# data['norm4'] = data['norm4'].apply(lambda x: clean_norm(x))\n",
    "# data['norm5'] = data['norm5'].apply(lambda x: clean_norm(x))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\numpy\\core\\fromnumeric.py:3118: RuntimeWarning: Mean of empty slice.\n",
      "  out=out, **kwargs)\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\numpy\\core\\_methods.py:85: RuntimeWarning: invalid value encountered in double_scalars\n",
      "  ret = ret.dtype.type(ret / rcount)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(3268, 300)\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "(3268,)"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "train_words_ner = data['words'].tolist()\n",
    "train_entity_ner = data['entity'].tolist()\n",
    "#======================================================================\n",
    "X_train_ner= np.zeros((len(train_words_ner),300)) \n",
    "\n",
    "for i in range(len(train_words_ner)):\n",
    "    words = word_tokenize(train_words_ner[i])\n",
    "    emb = [model.wv[w] for w in words] \n",
    "    X_train_ner[i] = np.mean(emb, axis=0)\n",
    "    \n",
    "df_train_ner= pd.DataFrame(X_train_ner)\n",
    "df_train_ner.fillna(value=0,inplace=True)\n",
    "\n",
    "X_train_ner = np.array(df_train_ner)\n",
    "print(X_train_ner.shape)\n",
    "\n",
    "#======================================================================\n",
    "le_ner = LabelEncoder()\n",
    "Y_train_ner = le_ner.fit_transform(train_entity_ner)\n",
    "\n",
    "Y_train_ner.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(3268, 600)\n",
      "(3268, 600)\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "(3268,)"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "train_words_norm = data['words'].tolist()\n",
    "train_entity_norm = data['entity'].tolist()\n",
    "\n",
    "#======================================================================\n",
    "\n",
    "X1_train_norm= np.zeros((len(train_words_norm),300)) \n",
    "X2_train_norm= np.zeros((len(train_entity_norm ),300)) \n",
    "\n",
    "for i in range(len(train_words_norm)):\n",
    "    words = word_tokenize(train_words_norm[i])\n",
    "    emb = [model.wv[w] for w in words] \n",
    "    X1_train_norm[i] = np.mean(emb, axis=0) \n",
    "\n",
    "\n",
    "for i in range(len(train_entity_norm)):\n",
    "    X2_train_norm[i] = model.wv[train_entity_norm[i]]\n",
    "    \n",
    "X_train_norm= np.concatenate((X1_train_norm,X2_train_norm),axis=1)\n",
    "print(X_train_norm.shape)\n",
    "\n",
    "df_train_norm= pd.DataFrame(X_train_norm)\n",
    "df_train_norm.fillna(value=0,inplace=True)\n",
    "X_train_norm = np.array(df_train_norm)\n",
    "print(X_train_norm.shape)\n",
    "\n",
    "#======================================================================\n",
    "\n",
    "test_norm = data['norm1'].tolist()\n",
    "\n",
    "#======================================================================\n",
    "\n",
    "le_norm = LabelEncoder()\n",
    "Y_train_norm = le_norm.fit_transform(test_norm)\n",
    "\n",
    "Y_train_norm.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "RandomForestClassifier(bootstrap=True, class_weight='balanced',\n",
       "            criterion='entropy', max_depth=None, max_features='auto',\n",
       "            max_leaf_nodes=None, min_impurity_decrease=0.0,\n",
       "            min_impurity_split=None, min_samples_leaf=1,\n",
       "            min_samples_split=2, min_weight_fraction_leaf=0.0,\n",
       "            n_estimators=30, n_jobs=None, oob_score=False,\n",
       "            random_state=None, verbose=0, warm_start=False)"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "from sklearn.ensemble import RandomForestClassifier\n",
    "from sklearn.metrics import accuracy_score, confusion_matrix, classification_report\n",
    "\n",
    "model_ner = RandomForestClassifier(n_estimators=30,criterion='entropy',class_weight='balanced')\n",
    "model_ner.fit(X_train_ner,Y_train_ner)\n",
    "\n",
    "model_norm = RandomForestClassifier(n_estimators=30,criterion='entropy',class_weight='balanced')\n",
    "model_norm .fit(X_train_norm ,Y_train_norm )\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [],
   "source": [
    "import nltk\n",
    "from nltk.tokenize import word_tokenize\n",
    "from nltk.tag import pos_tag"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [],
   "source": [
    "def preprocess(sent):\n",
    "    sent = nltk.word_tokenize(sent)\n",
    "    sent = nltk.pos_tag(sent)\n",
    "    return sent"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [],
   "source": [
    "def norm_test_data(df_test,X_test_ner):\n",
    "    \n",
    "#     test_words_norm  = df_test['words'].tolist()\n",
    "    test_entity_norm = df_test['entity'].tolist()\n",
    "        \n",
    "    \n",
    "#     X1_test_norm = np.zeros((len(test_words_norm ),300)) \n",
    "    X2_test_norm= np.zeros((len(test_entity_norm ),300)) \n",
    "    print(X2_test_norm.shape) \n",
    "    \n",
    "#     for i in range(len(test_words_norm )):\n",
    "#         words = word_tokenize(test_words_norm [i])\n",
    "#         emb = [model.wv[w] for w in words] \n",
    "#         X1_test_norm [i] = np.mean(emb, axis=0) \n",
    "\n",
    "    X1_test_norm  = X_test_ner\n",
    "        \n",
    "    for i in range(len(test_entity_norm)):\n",
    "        X2_train_norm[i] = model.wv[test_entity_norm[i]]\n",
    "    \n",
    "    print(X1_test_norm.shape,X2_test_norm.shape)\n",
    "    X_test_norm  = np.concatenate((X1_test_norm ,X2_test_norm ),axis=1)\n",
    "    df= pd.DataFrame(X_test_norm )\n",
    "    df.fillna(value=0,inplace=True)\n",
    "    X_test_norm  = np.array(df)\n",
    "    #print(X_test.shape)\n",
    "    predict_norm = model_norm.predict(X_test_norm )\n",
    "    norm = le_norm.inverse_transform(predict_norm)\n",
    "    \n",
    "    return norm"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [],
   "source": [
    "def preprocess_test_data(df_dat,org1 ):\n",
    "#     df_dat = df_dat.drop(df_dat[df_dat.entity == 'Paragraph'].index)\n",
    "    T_index = df_dat['words'].shape[0]\n",
    "    df_dat = df_dat.drop(df_dat[df_dat.words == 'Title.'].index)\n",
    "    df_dat['words'] = df_dat['words'].apply(lambda x: clean_data(x))\n",
    "    test_words = df_dat['words'].tolist()\n",
    "#     test_entity = df_dat['entity'].tolist()\n",
    "    \n",
    "    sent=[]\n",
    "    b=[]\n",
    "    ner_words=[]\n",
    "    df_test = pd.DataFrame(columns = ['text','words','entity','norm'])\n",
    "    p=1\n",
    "    X_test_ner =[]\n",
    "    pattern = 'NP: {<JJ|JJR|JJS>*<NN|NNS|NNP>*|<NN>|<JJ>|<VBP>}'\n",
    "    for sen in test_words:\n",
    "        sent = preprocess(sen)\n",
    "        cp = nltk.RegexpParser(pattern)\n",
    "        cs = cp.parse(sent)\n",
    "        for i,w1 in enumerate(cs):\n",
    "            if type(w1)==nltk.tree.Tree:\n",
    "                if w1.label()=='NP':\n",
    "                    X1_test=[]\n",
    "                    words=''\n",
    "                    #print(i+3,w1.leaves())\n",
    "                    \n",
    "                    for j,w2 in enumerate(w1):\n",
    "                        #print(j,w2)\n",
    "                        words=words+w2[0]+' '\n",
    "                        X1_test.append(model.wv[w2[0]])\n",
    "                    X_test_ner.append(np.mean(X1_test,axis=0))\n",
    "                    df_test = df_test.append({'text':f'T{p + T_index}','words':words}, ignore_index = True)\n",
    "                    print(words,p + T_index)\n",
    "                    p=p+1\n",
    "#             else:\n",
    "#                 b.append(i)\n",
    "    \n",
    "    X_test_ner =np.array(X_test_ner)\n",
    "#     print(X_test_ner.shape,org1 )\n",
    "    predict_ner = model_ner.predict(X_test_ner)\n",
    "    ner_entity=le_ner.inverse_transform(predict_ner)\n",
    "    df_test['entity'] = ner_entity\n",
    "    print(ner_entity)\n",
    "    print(X_test_ner.shape,ner_entity.shape)\n",
    "    print(df_test.head())\n",
    "    print(org1 )\n",
    "    \n",
    "    \n",
    "    norm = norm_test_data(df_test,X_test_ner)\n",
    "    df_test['norm'] = norm\n",
    "    print(norm)\n",
    "    return df_dat,df_test\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [],
   "source": [
    "def create_a2_file(df_test):\n",
    "    annotation=[]\n",
    "    text = df_test['text'].tolist()\n",
    "    norm = df_test['norm'].tolist()\n",
    "    for i in range(0,len(norm)):\n",
    "        if 'NCBI' in norm[i]:\n",
    "            annotation.append(f'N{i+1}\\tNCBI_Taxonomy Annotation:{text[i]} Referent:{os.path.splitext(norm[i])[0][14:]}')\n",
    "        \n",
    "        elif 'OBT' in norm[i]:\n",
    "            annotation.append(f'N{i+1}\\tOntoBiotope Annotation:{text[i]} Referent:{os.path.splitext(norm[i])[0][12:]}')\n",
    "            \n",
    "    sub = pd.DataFrame(annotation)\n",
    "    \n",
    "    return sub"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [],
   "source": [
    "def predict_test_data(path):\n",
    "    \n",
    "    org_dat = os.listdir(path)\n",
    "    z=[]\n",
    "    for i in range(len(org_dat)):\n",
    "        \n",
    "        df_dat = pd.DataFrame(columns = ['text','words','entity'])\n",
    "        org1 = org_dat[i]\n",
    "        if org1.endswith('.txt'):\n",
    "            org_path1 = os.path.join(path,org1)\n",
    "            text1 = pd.read_table(org_path1,header = None)\n",
    "            z.append(org1)\n",
    "            for j in range(0,text1.shape[0]):\n",
    "                df_dat = df_dat.append({'words':text1[0][j]}, ignore_index = True) \n",
    "           \n",
    "            \n",
    "            df_dat,df_test = preprocess_test_data(df_dat,org1)\n",
    "            #print(X_test.shape)\n",
    "            \n",
    "            sub = create_a2_file(df_test)\n",
    "            sub.to_csv(f\"submission_norm+ner/{os.path.splitext(org1)[0]}.a2\",index=False,header=False)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "listeria monocytogenes  3\n",
      "acta  4\n",
      "protein interacts  5\n",
      "phosphatidylinositol  6\n",
      "bisphosphate vitro  7\n",
      "n terminal region listeria monocytogenes  8\n",
      "acta  9\n",
      "protein conjunction host cell factors  10\n",
      "sufficient actin polymerization  11\n",
      "bacterial surface  12\n",
      "previous data  13\n",
      "acta  14\n",
      "barbed ends  15\n",
      "proteins  16\n",
      "hypothesis actin polymerization experiments presence  17\n",
      "acta  18\n",
      "n terminal fragment  19\n",
      "protein acta  20\n",
      "protect  21\n",
      "ends  22\n",
      "protein contrast polypeptide prevents  23\n",
      "pip  24\n",
      "activity  25\n",
      "protein  26\n",
      "gel filtration tryptophan fluorescence experiments  27\n",
      "purified acta  28\n",
      "n terminal fragment binds  29\n",
      "pip  30\n",
      "pip  31\n",
      "phosphoinositides  32\n",
      "novels  33\n",
      "functional domain  34\n",
      "acta phosphoinositide  35\n",
      "n terminal region acta  36\n",
      "conformational changes  37\n",
      "acta  38\n",
      "facilitate binding cell components  39\n",
      "important acta  40\n",
      "actin polymerization  41\n",
      "['Microorganism' 'Habitat' 'Phenotype' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Phenotype' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Phenotype' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Phenotype' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat']\n",
      "(39, 300) (39,)\n",
      "  text                    words         entity norm\n",
      "0   T3  listeria monocytogenes   Microorganism  NaN\n",
      "1   T4                    acta         Habitat  NaN\n",
      "2   T5       protein interacts       Phenotype  NaN\n",
      "3   T6    phosphatidylinositol         Habitat  NaN\n",
      "4   T7      bisphosphate vitro         Habitat  NaN\n",
      "BB-norm+ner-10618167.txt\n",
      "(39, 300)\n",
      "(39, 300) (39, 300)\n",
      "['NCBI_Taxonomy 1639' 'NCBI_Taxonomy 10508' 'OntoBiotope OBT:000019'\n",
      " 'NCBI_Taxonomy 1255621' 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'NCBI_Taxonomy 10508' 'OntoBiotope OBT:000062' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000113' 'NCBI_Taxonomy 10508'\n",
      " 'NCBI_Taxonomy 1582' 'OntoBiotope OBT:000061' 'OntoBiotope OBT:000019'\n",
      " 'NCBI_Taxonomy 10508' 'NCBI_Taxonomy 263' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000360'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 263' 'NCBI_Taxonomy 263'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1308' 'OntoBiotope OBT:000055'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1313' 'OntoBiotope OBT:000019'\n",
      " 'NCBI_Taxonomy 10508' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 727'\n",
      " 'OntoBiotope OBT:000019']\n",
      "genotypic characterization drug  7\n",
      "resistant mycobacterium tuberculosis  8\n",
      "peru  9\n",
      "epidemiological unrelated  10\n",
      "multidrug resistant mycobacterium tuberculosis mdr tb strains  11\n",
      "peruvian patients  12\n",
      "molecular genetics  13\n",
      "mdr  14\n",
      "tb strains  15\n",
      "latin american country  16\n",
      "antimicrobial agent susceptibility  17\n",
      "major genetic group designation  18\n",
      "deoxyribonucleic acid  19\n",
      "regions  20\n",
      "katg  21\n",
      "rpob embb gyra pnca genes mutations  22\n",
      "drug resistance  23\n",
      "nineteen isolates  24\n",
      "multidrug resistant susceptibility  25\n",
      "unique  26\n",
      "drug resistance seventy  27\n",
      "percent isoniazid  28\n",
      "resistant strains  29\n",
      "ser  30\n",
      "thr amino  31\n",
      "acid  32\n",
      "change katg ninety  33\n",
      "percent rifampin  34\n",
      "resistant  35\n",
      "amino acid replacements  36\n",
      "rifampin  37\n",
      "resistance  38\n",
      "region rpob  39\n",
      "ethambutol  40\n",
      "resistant strains  41\n",
      "embb  42\n",
      "alterations  43\n",
      "pyrazinamide  44\n",
      "resistant strains  45\n",
      "distinct mutations  46\n",
      "pnca  47\n",
      "organisms  48\n",
      "drug resistance  49\n",
      "drug resistance  50\n",
      "mutations  51\n",
      "similar changes  52\n",
      "isolates areas world  53\n",
      "sequence  54\n",
      "strategies  55\n",
      "rapid detection drug resistance  56\n",
      "mutants  57\n",
      "applicable organisms  58\n",
      "peru  59\n",
      "areas  60\n",
      "latin  61\n",
      "america  62\n",
      "['Phenotype' 'Phenotype' 'Habitat' 'Habitat' 'Phenotype' 'Habitat'\n",
      " 'Habitat' 'Phenotype' 'Microorganism' 'Habitat' 'Phenotype' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Phenotype' 'Habitat'\n",
      " 'Phenotype' 'Habitat' 'Habitat' 'Habitat' 'Phenotype' 'Habitat' 'Habitat'\n",
      " 'Phenotype' 'Habitat' 'Habitat' 'Phenotype' 'Phenotype' 'Habitat'\n",
      " 'Phenotype' 'Habitat' 'Habitat' 'Phenotype' 'Habitat' 'Habitat'\n",
      " 'Phenotype' 'Phenotype' 'Habitat' 'Habitat' 'Habitat' 'Phenotype'\n",
      " 'Phenotype' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Phenotype'\n",
      " 'Phenotype' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat']\n",
      "(56, 300) (56,)\n",
      "  text                                              words     entity norm\n",
      "0   T7                   genotypic characterization drug   Phenotype  NaN\n",
      "1   T8              resistant mycobacterium tuberculosis   Phenotype  NaN\n",
      "2   T9                                              peru     Habitat  NaN\n",
      "3  T10                         epidemiological unrelated     Habitat  NaN\n",
      "4  T11  multidrug resistant mycobacterium tuberculosis...  Phenotype  NaN\n",
      "BB-norm+ner-10645449.txt\n",
      "(56, 300)\n",
      "(56, 300) (56, 300)\n",
      "['OntoBiotope OBT:000019' 'OntoBiotope OBT:002829'\n",
      " 'OntoBiotope OBT:001828' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:002806'\n",
      " 'OntoBiotope OBT:000360' 'OntoBiotope OBT:000598' 'NCBI_Taxonomy 777'\n",
      " 'OntoBiotope OBT:003334' 'OntoBiotope OBT:002732'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000232' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000598' 'NCBI_Taxonomy 28875'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000064'\n",
      " 'OntoBiotope OBT:000097' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000152' 'NCBI_Taxonomy 267818' 'NCBI_Taxonomy 562'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1752'\n",
      " 'OntoBiotope OBT:002337' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:002337' 'OntoBiotope OBT:002869'\n",
      " 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 1173061' 'OntoBiotope OBT:000152'\n",
      " 'NCBI_Taxonomy 294' 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000152' 'OntoBiotope OBT:000029'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000114'\n",
      " 'OntoBiotope OBT:000598' 'OntoBiotope OBT:000598'\n",
      " 'OntoBiotope OBT:000464' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:001828' 'NCBI_Taxonomy 1111734' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:003269' 'NCBI_Taxonomy 263' 'NCBI_Taxonomy 1697'\n",
      " 'OntoBiotope OBT:001828' 'OntoBiotope OBT:003188'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:002806']\n",
      "display  3\n",
      "active subtilisin phage analysis parameters  4\n",
      "selection subtilisin variants  5\n",
      "substrate specificity libraries  6\n",
      "inhibitors  7\n",
      "many attempts  8\n",
      "enzymes new catalytic activities  9\n",
      "general strategy  10\n",
      "creation random  11\n",
      "combinatorial libraries mutants  12\n",
      "associated efficient  13\n",
      "selection scheme phage  14\n",
      "facilitate selection polypeptides  15\n",
      "properties  16\n",
      "close link  17\n",
      "gene encodes selection phage  18\n",
      "new catalytic activities  19\n",
      "challenge aim study display  20\n",
      "serine protease subtilisin savinase bacillus  21\n",
      "lentus surface  22\n",
      "filamentous fd phage  23\n",
      "selection schemes  24\n",
      "allow  25\n",
      "extraction subtilisin variants  26\n",
      "substrate specificity libraries  27\n",
      "secreted preproenzyme mature  28\n",
      "active enzyme  29\n",
      "broad substrate specificity exhibit  30\n",
      "significant preference hydrophobic residues  31\n",
      "limited reactivity  32\n",
      "residues  33\n",
      "p  34\n",
      "site substrate show savinase  35\n",
      "displayed phage presence proteic inhibitor  36\n",
      "free enzyme "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " 37\n",
      "complex ci  38\n",
      "addition  39\n",
      "anionic detergent las phage enzyme  40\n",
      "streptavidin beads  41\n",
      "reaction biotin  42\n",
      "n epsilon aminocaproyl cystamine  43\n",
      "n glutaryl l ala l ala l p ro phe p diphenyl ester reactions  44\n",
      "residues  45\n",
      "part pocket  46\n",
      "biotin n epsilon aminocaproyl cystamine  47\n",
      "n glutaryl alpha l lys l la  48\n",
      "l pro phe p diphenylester  49\n",
      "select enzymes  50\n",
      "specific activity substrate  51\n",
      "lysine p parameters  52\n",
      "selection instance efficiency maturation mutant  53\n",
      "libraries  54\n",
      "['Habitat' 'Habitat' 'Habitat' 'Habitat' 'Phenotype' 'Habitat' 'Phenotype'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Phenotype' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Phenotype' 'Microorganism'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Phenotype' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism'\n",
      " 'Habitat' 'Phenotype' 'Phenotype' 'Habitat' 'Habitat' 'Phenotype'\n",
      " 'Habitat' 'Phenotype' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Phenotype' 'Habitat' 'Phenotype' 'Habitat']\n",
      "(52, 300) (52,)\n",
      "  text                                         words     entity norm\n",
      "0   T3                                      display     Habitat  NaN\n",
      "1   T4  active subtilisin phage analysis parameters     Habitat  NaN\n",
      "2   T5                selection subtilisin variants     Habitat  NaN\n",
      "3   T6              substrate specificity libraries     Habitat  NaN\n",
      "4   T7                                   inhibitors   Phenotype  NaN\n",
      "BB-norm+ner-10656819.txt\n",
      "(52, 300)\n",
      "(52, 300) (52, 300)\n",
      "['NCBI_Taxonomy 1256511' 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:002806' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1977402'\n",
      " 'OntoBiotope OBT:002337' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000029' 'OntoBiotope OBT:000061'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1173061' 'OntoBiotope OBT:003188'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 37763' 'NCBI_Taxonomy 71237'\n",
      " 'OntoBiotope OBT:002806' 'OntoBiotope OBT:000113' 'NCBI_Taxonomy 1697'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 585'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000113' 'NCBI_Taxonomy 1582'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000006'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1256511'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003188'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:001480' 'NCBI_Taxonomy 1182512'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 29412'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019']\n",
      "evaluation  3\n",
      "brucella  4\n",
      "soluble antigens  5\n",
      "indirect elisa discriminate  6\n",
      "cattle  7\n",
      "chain  8\n",
      "hot water  9\n",
      "polysaccharide ps  10\n",
      "brucella abortus b melitensis  11\n",
      "native hapten  12\n",
      "indirect enzyme  13\n",
      "immunosorbent assay  14\n",
      "elisa  15\n",
      "groups  16\n",
      "cattle  17\n",
      "sera sera  18\n",
      "sera cows  19\n",
      "b abortus b sera non  20\n",
      "cattle  21\n",
      "c  22\n",
      "sera non  23\n",
      "heifers  24\n",
      "b abortus strain sensitivity  25\n",
      "se  26\n",
      "specificity sp capability discriminate  27\n",
      "cattle adv  28\n",
      "ps antigen se sp  29\n",
      "highest sp  30\n",
      "chain  31\n",
      "nh antigen se sp  32\n",
      "antigens  33\n",
      "statistical analysis  34\n",
      "significant differences chain ps chain  35\n",
      "nh  36\n",
      "agreement  37\n",
      "antigens  38\n",
      "kappa coefficient chain ps chain  39\n",
      "nh ps nh  40\n",
      "['Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Microorganism' 'Habitat' 'Phenotype' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism'\n",
      " 'Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Microorganism' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat']\n",
      "(38, 300) (38,)\n",
      "  text                         words         entity norm\n",
      "0   T3                   evaluation         Habitat  NaN\n",
      "1   T4                     brucella   Microorganism  NaN\n",
      "2   T5             soluble antigens         Habitat  NaN\n",
      "3   T6  indirect elisa discriminate         Habitat  NaN\n",
      "4   T7                       cattle         Habitat  NaN\n",
      "BB-norm+ner-10665543.txt\n",
      "(38, 300)\n",
      "(38, 300) (38, 300)\n",
      "['OntoBiotope OBT:003220' 'OntoBiotope OBT:000375' 'NCBI_Taxonomy 543'\n",
      " 'OntoBiotope OBT:000113' 'OntoBiotope OBT:003532'\n",
      " 'OntoBiotope OBT:001623' 'OntoBiotope OBT:000468'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:002714' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 114277' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:003532' 'OntoBiotope OBT:000524'\n",
      " 'OntoBiotope OBT:000524' 'NCBI_Taxonomy 585' 'OntoBiotope OBT:003532'\n",
      " 'NCBI_Taxonomy 263' 'OntoBiotope OBT:000007' 'OntoBiotope OBT:002806'\n",
      " 'OntoBiotope OBT:003220' 'OntoBiotope OBT:003188'\n",
      " 'OntoBiotope OBT:000029' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000215'\n",
      " 'OntoBiotope OBT:001623' 'OntoBiotope OBT:000375'\n",
      " 'OntoBiotope OBT:002995' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003269' 'NCBI_Taxonomy 1752'\n",
      " 'OntoBiotope OBT:002995' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:003220']\n",
      "immune response infection salmonella typhimurium mice  3\n",
      "infection mice salmonella typhimurium results  4\n",
      "systemic infection disease  5\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "similar  6\n",
      "humans  7\n",
      "infection typhi innate immune system  8\n",
      "restrict replication typhimurium  9\n",
      "certain degree effective control eradication bacteria  10\n",
      "immunity essential salmonella infection induces generation  11\n",
      "specific cd cd cells  12\n",
      "cell  13\n",
      "populations  14\n",
      "important protection  15\n",
      "primary secondary responses  16\n",
      "mechanisms  17\n",
      "cell  18\n",
      "protection  19\n",
      "understood infection typhimurium  20\n",
      "results  21\n",
      "strong antibody response salmonella  22\n",
      "contrast  23\n",
      "intracellular bacteria antibody response  24\n",
      "protection  25\n",
      "summary response typhimurium  26\n",
      "cell  27\n",
      "immunity mechanisms  28\n",
      "lymphocyte populations  29\n",
      "important control  30\n",
      "primary infection protection  31\n",
      "secondary infection  32\n",
      "['Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat']\n",
      "(30, 300) (30,)\n",
      "  text                                              words   entity norm\n",
      "0   T3  immune response infection salmonella typhimuri...  Habitat  NaN\n",
      "1   T4     infection mice salmonella typhimurium results   Habitat  NaN\n",
      "2   T5                        systemic infection disease   Habitat  NaN\n",
      "3   T6                                           similar   Habitat  NaN\n",
      "4   T7                                            humans   Habitat  NaN\n",
      "BB-norm+ner-10770276.txt\n",
      "(30, 300)\n",
      "(30, 300) (30, 300)\n",
      "['OntoBiotope OBT:002806' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:002806' 'NCBI_Taxonomy 585' 'OntoBiotope OBT:002488'\n",
      " 'OntoBiotope OBT:002806' 'NCBI_Taxonomy 263' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:001623'\n",
      " 'OntoBiotope OBT:000706' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000113' 'OntoBiotope OBT:000706' 'NCBI_Taxonomy 1639'\n",
      " 'NCBI_Taxonomy 12721' 'OntoBiotope OBT:000038' 'OntoBiotope OBT:000019'\n",
      " 'NCBI_Taxonomy 29412' 'OntoBiotope OBT:000113' 'NCBI_Taxonomy 1639'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000706'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1639' 'OntoBiotope OBT:000007'\n",
      " 'NCBI_Taxonomy 33892' 'OntoBiotope OBT:002806']\n",
      "role fcgamma receptor  3\n",
      "polymorphisms  4\n",
      "c  5\n",
      "immune defence neisseria  6\n",
      "meningitidis  7\n",
      "complement deficient individuals  8\n",
      "individuals  9\n",
      "late c complement component deficiency lccd  10\n",
      "properdin deficiency  11\n",
      "risk  12\n",
      "meningococcal disease  13\n",
      "due serogroups  14\n",
      "w  15\n",
      "anti meningococcal defence lccd persons  16\n",
      "properdin  17\n",
      "deficient individuals  18\n",
      "bactericidal antibodies  19\n",
      "phagocytosis  20\n",
      "types  21\n",
      "opsonin receptors  22\n",
      "phagocytosis polymorphonuclear cells  23\n",
      "pmn represent polymorphic fcgammariia cd  24\n",
      "fcgammariiib cd b receptors  25\n",
      "c  26\n",
      "receptor cr cd  27\n",
      "cd distribution fcgammariia fcgammariiib allotypes  28\n",
      "lccd properdin deficient patients  29\n",
      "previous meningococcal disease  30\n",
      "combination fcgammariia r r fcgammariiib  31\n",
      "na allotypes  32\n",
      "previous meningococcal disease odds  33\n",
      "ratio  34\n",
      "test p relation  35\n",
      "properdin deficient patients importance fcgammariia allotypes  36\n",
      "vitro phagocytosis assays  37\n",
      "pmn  38\n",
      "fcgammariia r r  39\n",
      "homozygous donors  40\n",
      "igg opsonized meningococci w  41\n",
      "pmn fcgammariia h h donors  42\n",
      "properdin  43\n",
      "deficient serum  44\n",
      "observed reconstitution properdin  45\n",
      "enhanced pmn phagocytosis w meningococci p  46\n",
      "phagocytosis  47\n",
      "increase c deposition  48\n",
      "meningococci w r p conclude occurrence  49\n",
      "meningococcal disease lccd patients  50\n",
      "certain fcgammar allotypes  51\n",
      "properdin  52\n",
      "deficient individuals  53\n",
      "susceptible meningococcal disease  54\n",
      "insufficient c deposition surface meningococci  55\n",
      "insufficient phagocytosis  56\n",
      "['Habitat' 'Habitat' 'Microorganism' 'Habitat' 'Microorganism' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Phenotype' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Habitat' 'Microorganism'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Phenotype'\n",
      " 'Habitat' 'Habitat']\n",
      "(54, 300) (54,)\n",
      "  text                      words         entity norm\n",
      "0   T3     role fcgamma receptor         Habitat  NaN\n",
      "1   T4             polymorphisms         Habitat  NaN\n",
      "2   T5                         c   Microorganism  NaN\n",
      "3   T6  immune defence neisseria         Habitat  NaN\n",
      "4   T7              meningitidis   Microorganism  NaN\n",
      "BB-norm+ner-10792385.txt\n",
      "(54, 300)\n",
      "(54, 300) (54, 300)\n",
      "['OntoBiotope OBT:000019' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 263'\n",
      " 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 487' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:001623' 'NCBI_Taxonomy 12721'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 263' 'OntoBiotope OBT:002694'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:002732'\n",
      " 'OntoBiotope OBT:000029' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:001623' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 263' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000029' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:002806'\n",
      " 'OntoBiotope OBT:001134' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 263' 'OntoBiotope OBT:000152'\n",
      " 'OntoBiotope OBT:000031' 'OntoBiotope OBT:001480'\n",
      " 'OntoBiotope OBT:002732' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003220'\n",
      " 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 263' 'OntoBiotope OBT:002694'\n",
      " 'NCBI_Taxonomy 12721' 'OntoBiotope OBT:000006' 'NCBI_Taxonomy 1279']\n",
      "symbiotic induction pyruvate dehydrogenase genes sinorhizobium meliloti  3\n",
      "genes  4\n",
      "components  5\n",
      "pyruvate dehydrogenase pdh multienzyme  6\n",
      "complex pdhc sinorhizobium meliloti alfalfa symbiont  7\n",
      "isolated basis  8\n",
      "high expression  9\n",
      "symbiotic bacteria  10\n",
      "elp  11\n",
      "component pdh  12\n",
      "genes  13\n",
      "pdhaalpha  14\n",
      "bp pdhabeta bp situation  15\n",
      "alpha proteobacteria  16\n",
      "rickettsia  17\n",
      "prowazekii zymomonas mobilis  18\n",
      "positive bacteria  19\n",
      "mitochondria  20\n",
      "pdhaalpha pdhabeta precede pdhb bp  21\n",
      "e p component dihydrolipoamide acetyltransferase pdhc gene  22\n",
      "e component lipoamide dehydrogenase  23\n",
      "immediate vicinity pdha pdhb genes  24\n",
      "pdhaalpha  25\n",
      "pdhabeta pdhb  26\n",
      "likely constitute operon  27\n",
      "provide  28\n",
      "evidence pdha expression  29\n",
      "symbiotic stage  30\n",
      "free living conditions  31\n",
      "demonstrate  32\n",
      "symbiotic expression pdha genes  33\n",
      "depend  34\n",
      "fix lj regulatory cascade  35\n",
      "nitrogen fixation respiration gene expression  36\n",
      "symbiotic meliloti cells induction pdha expression  37\n",
      "free living conditions  38\n",
      "addition  39\n",
      "pyruvate culture medium induction pyruvate  40\n",
      "symbiotic activation pdh gene expression  41\n",
      "place promoter  42\n",
      "['Microorganism' 'Microorganism' 'Habitat' 'Phenotype' 'Microorganism'\n",
      " 'Habitat' 'Habitat' 'Phenotype' 'Habitat' 'Habitat' 'Microorganism'\n",
      " 'Habitat' 'Habitat' 'Microorganism' 'Microorganism' 'Microorganism'\n",
      " 'Phenotype' 'Habitat' 'Habitat' 'Habitat' 'Phenotype' 'Phenotype'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat']\n",
      "(40, 300) (40,)\n",
      "  text                                              words         entity norm\n",
      "0   T3  symbiotic induction pyruvate dehydrogenase gen...  Microorganism  NaN\n",
      "1   T4                                             genes   Microorganism  NaN\n",
      "2   T5                                        components         Habitat  NaN\n",
      "3   T6            pyruvate dehydrogenase pdh multienzyme       Phenotype  NaN\n",
      "4   T7  complex pdhc sinorhizobium meliloti alfalfa sy...  Microorganism  NaN\n",
      "BB-norm+ner-10796014.txt\n",
      "(40, 300)\n",
      "(40, 300) (40, 300)\n",
      "['OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 263'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000055' 'NCBI_Taxonomy 1046' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 780'\n",
      " 'OntoBiotope OBT:002806' 'OntoBiotope OBT:000061'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 777'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000029'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000113'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000328'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003188' 'NCBI_Taxonomy 1582'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 5052' 'OntoBiotope OBT:003269'\n",
      " 'NCBI_Taxonomy 1582' 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:002694']\n",
      "expression analysis phytoplasma protein translocation genes  3\n",
      "genes  4\n",
      "seca secy  5\n",
      "essential components  6\n",
      "sec  7\n",
      "translocation system  8\n",
      "onion yellows  9\n",
      "phytoplasma  10\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unculturable plant  11\n",
      "pathogenic bacterium seca gene  12\n",
      "nucleotides  13\n",
      "amino acid protein kda  14\n",
      "highest similarity seca bacillus subtilis  15\n",
      "anti  16\n",
      "seca rabbit antibody  17\n",
      "purified partial seca protein histidine tag  18\n",
      "escherichia coli  19\n",
      "western blot analysis  20\n",
      "seca protein  21\n",
      "infected plants  22\n",
      "immunohistochemical thin sections  23\n",
      "optical microscopy  24\n",
      "seca  25\n",
      "present plant phloem tissues  26\n",
      "phytoplasma secy gene  27\n",
      "nucleotides  28\n",
      "amino acid protein kda  29\n",
      "highest similarity secy b subtilis results  30\n",
      "suggest  31\n",
      "presence  32\n",
      "functional sec system  33\n",
      "phytoplasmas phytoplasmas  34\n",
      "endocellular bacteria  35\n",
      "cell walls system  36\n",
      "bacterial proteins  37\n",
      "host  38\n",
      "cytoplasm study  39\n",
      "believe  40\n",
      "first report sequence expression analysis phytoplasma genes  41\n",
      "membrane proteins  42\n",
      "function  43\n",
      "['Phenotype' 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Habitat' 'Phenotype'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Phenotype' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Phenotype' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat']\n",
      "(41, 300) (41,)\n",
      "  text                                              words         entity norm\n",
      "0   T3  expression analysis phytoplasma protein transl...      Phenotype  NaN\n",
      "1   T4                                             genes   Microorganism  NaN\n",
      "2   T5                                         seca secy         Habitat  NaN\n",
      "3   T6                              essential components         Habitat  NaN\n",
      "4   T7                                               sec         Habitat  NaN\n",
      "BB-norm+ner-11551069.txt\n",
      "(41, 300)\n",
      "(41, 300) (41, 300)\n",
      "['OntoBiotope OBT:000019' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 53482'\n",
      " 'OntoBiotope OBT:000061' 'NCBI_Taxonomy 5082' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000360' 'NCBI_Taxonomy 1299' 'OntoBiotope OBT:000524'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000055'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000143'\n",
      " 'OntoBiotope OBT:000933' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000061' 'NCBI_Taxonomy 1045010' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:002337' 'NCBI_Taxonomy 4952'\n",
      " 'NCBI_Taxonomy 1117' 'NCBI_Taxonomy 260680' 'NCBI_Taxonomy 197'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000055'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000113'\n",
      " 'OntoBiotope OBT:000113' 'OntoBiotope OBT:003334'\n",
      " 'OntoBiotope OBT:000061' 'NCBI_Taxonomy 1350' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000113' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000061' 'NCBI_Taxonomy 1359']\n",
      "emergence resistance vancomycin  3\n",
      "resistant enterococcus faecium  4\n",
      "thermal injury patient  5\n",
      "quinupristin dalfopristin  6\n",
      "epithelial autografts  7\n",
      "closure  8\n",
      "vancomycin  9\n",
      "resistant enterococcus faecium faecalis vre  10\n",
      "major complication  11\n",
      "ill patients year  12\n",
      "old patient  13\n",
      "total body surface area burns  14\n",
      "several e faecium  15\n",
      "admission  16\n",
      "resistant vancomycin  17\n",
      "chloramphenicol standard treatment time drug  18\n",
      "organism  19\n",
      "e faecium  20\n",
      "susceptible quinupristin dalfopristin  21\n",
      "data  22\n",
      "discontinue chloramphenicol therapy quinupristin dalfopristin therapy  23\n",
      "initial reduction  24\n",
      "white blood cell  25\n",
      "continued next days  26\n",
      "day  27\n",
      "quinupristin dalfopristin therapy return fever elevation  28\n",
      "white blood cell count  29\n",
      "repeated e faecium blood culture  30\n",
      "sudden resistance quinupristin dalfopristin bauer  31\n",
      "size mm chloramphenicol  32\n",
      "patient  33\n",
      "period days  34\n",
      "indigenous vre  35\n",
      "exposure quinupristin dalfopristin  36\n",
      "recent past  37\n",
      "resistance  38\n",
      "commercial use  39\n",
      "agent burn treatment center  40\n",
      "high dose chloramphenicol treatment  41\n",
      "appear  42\n",
      "impair engraftment  43\n",
      "epithelial autografts  44\n",
      "cea  45\n",
      "patient "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " 46\n",
      "['Phenotype' 'Microorganism' 'Habitat' 'Phenotype' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Habitat' 'Phenotype' 'Habitat' 'Habitat' 'Microorganism'\n",
      " 'Phenotype' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Phenotype' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat']\n",
      "(44, 300) (44,)\n",
      "  text                             words         entity norm\n",
      "0   T3  emergence resistance vancomycin       Phenotype  NaN\n",
      "1   T4   resistant enterococcus faecium   Microorganism  NaN\n",
      "2   T5           thermal injury patient         Habitat  NaN\n",
      "3   T6        quinupristin dalfopristin       Phenotype  NaN\n",
      "4   T7            epithelial autografts         Habitat  NaN\n",
      "BB-norm+ner-12417169.txt\n",
      "(44, 300)\n",
      "(44, 300) (44, 300)\n",
      "['OntoBiotope OBT:003269' 'OntoBiotope OBT:000113'\n",
      " 'OntoBiotope OBT:002480' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000334' 'OntoBiotope OBT:000118' 'NCBI_Taxonomy 1246'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:002714'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 114277' 'OntoBiotope OBT:001341'\n",
      " 'OntoBiotope OBT:002874' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000113' 'NCBI_Taxonomy 1352' 'NCBI_Taxonomy 1280'\n",
      " 'NCBI_Taxonomy 36656' 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000097'\n",
      " 'OntoBiotope OBT:000061' 'OntoBiotope OBT:001480'\n",
      " 'OntoBiotope OBT:000031' 'OntoBiotope OBT:003220' 'NCBI_Taxonomy 263'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:003220'\n",
      " 'OntoBiotope OBT:002480' 'OntoBiotope OBT:000113' 'NCBI_Taxonomy 562'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:002869'\n",
      " 'OntoBiotope OBT:002480' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000360' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:002806' 'OntoBiotope OBT:000334' 'NCBI_Taxonomy 1255580'\n",
      " 'OntoBiotope OBT:003220']\n",
      "pathological therapeutic significance  3\n",
      "cellular invasion proteus mirabilis  4\n",
      "enterocystoplasty infection stone model  5\n",
      "proteus mirabilis infection  6\n",
      "stone formation  7\n",
      "bacterium mucin adhesion invasion  8\n",
      "intracellular crystal formation  9\n",
      "antibiotic sensitivity  10\n",
      "frequent stone formation enterocystoplasties  11\n",
      "intestinal caco ht ht  12\n",
      "n ht fu ht  13\n",
      "mtx  14\n",
      "ureter cell line sv huc  15\n",
      "artificial urine  16\n",
      "proteus mirabilis  17\n",
      "cell  18\n",
      "facs laser  19\n",
      "microscopy electron microscopy  20\n",
      "cellular adhesion invasion pathologic changes  21\n",
      "mitochondria  22\n",
      "p mirabilis mucin colocalization muc muc ac mtt thiazolyl  23\n",
      "blue tetrazolium bromide  24\n",
      "assay  25\n",
      "facs analysis caspase  26\n",
      "cellular response  27\n",
      "cells  28\n",
      "antibiotics dosages  29\n",
      "urinary concentrations year  30\n",
      "old kg child  31\n",
      "evaluate  32\n",
      "bacterial invasion survival cell lines  33\n",
      "colocalization p mirabilis  34\n",
      "human colonic mucin e muc  35\n",
      "human gastric mucin e muc  36\n",
      "ac correlation membrane mucin expression invasion  37\n",
      "significant opposite sv huc ht mtx  38\n",
      "invasion p mirabilis  39\n",
      "intracellular crystal formation  40\n",
      "mitochondrial damage  41\n",
      "double membranes  42\n",
      "bacteria  43\n",
      "intestinal cells  44\n",
      "relative resistance cotrimoxazole augmentin  45\n",
      "presence  46\n",
      "epithelial cells  47\n",
      "ciprofloxacin  48\n",
      "gentamicin  49\n",
      "effective membrane mucin expression  50\n",
      "relative antibiotic resistance cell invasion p mirabilis mucin cell type  51\n",
      "distribution response differences  52\n",
      "indicate  53\n",
      "bacterial tropism  54\n",
      "crystal formation mucosal presence  55\n",
      "bacterial invasion  56\n",
      "cell  57\n",
      "type dependent mechanisms  58\n",
      "prolong bacterial survival  59\n",
      "antibiotic therapy  60\n",
      "new target  61\n",
      "therapeutic optimalization  62\n",
      "antibiotic treatment  63\n",
      "['Habitat' 'Microorganism' 'Habitat' 'Microorganism' 'Habitat' 'Phenotype'\n",
      " 'Habitat' 'Phenotype' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Phenotype' 'Habitat' 'Phenotype' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Phenotype' 'Habitat' 'Phenotype' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Phenotype']\n",
      "(61, 300) (61,)\n",
      "  text                                     words         entity norm\n",
      "0   T3    pathological therapeutic significance         Habitat  NaN\n",
      "1   T4      cellular invasion proteus mirabilis   Microorganism  NaN\n",
      "2   T5  enterocystoplasty infection stone model         Habitat  NaN\n",
      "3   T6              proteus mirabilis infection   Microorganism  NaN\n",
      "4   T7                          stone formation         Habitat  NaN\n",
      "BB-norm+ner-12438382.txt\n",
      "(61, 300)\n",
      "(61, 300) (61, 300)\n",
      "['OntoBiotope OBT:000061' 'OntoBiotope OBT:000062'\n",
      " 'OntoBiotope OBT:003188' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:001480' 'OntoBiotope OBT:002675'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:002806'\n",
      " 'OntoBiotope OBT:000113' 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 208964'\n",
      " 'OntoBiotope OBT:000152' 'NCBI_Taxonomy 10912' 'OntoBiotope OBT:000706'\n",
      " 'OntoBiotope OBT:000108' 'NCBI_Taxonomy 294' 'OntoBiotope OBT:002806'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 272630'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000801'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:003188' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 35818'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 35818' 'OntoBiotope OBT:001480'\n",
      " 'OntoBiotope OBT:002806' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000113' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:001623' 'OntoBiotope OBT:002790'\n",
      " 'OntoBiotope OBT:002762' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 120652'\n",
      " 'OntoBiotope OBT:000055' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000152' 'OntoBiotope OBT:000706' 'NCBI_Taxonomy 263'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:003269']\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "detection cryptosporidium parvum  3\n",
      "secondary effluents  4\n",
      "probable number polymerase chain reaction  5\n",
      "assay  6\n",
      "polymerase chain reaction pcr  7\n",
      "detect cryptosporidium parvum  8\n",
      "secondary effluent samples  9\n",
      "sludge facilities  10\n",
      "serial dilutions  11\n",
      "nucleic acid extracts samples  12\n",
      "pcr  13\n",
      "conducted estimate c  14\n",
      "parvum  15\n",
      "oocyst concentration  16\n",
      "poisson distribution  17\n",
      "probable number mpn  18\n",
      "degree oocysts  19\n",
      "wastewater particles  20\n",
      "sensitivity mpn pcr  21\n",
      "assay  22\n",
      "oocysts  23\n",
      "pcr  24\n",
      "unit detection limit concentration extraction purification protocols  25\n",
      "phosphate  26\n",
      "buffer saline  27\n",
      "concentration oocysts  28\n",
      "oocysts  29\n",
      "l detection limit wastewater samples  30\n",
      "oocysts  31\n",
      "l  32\n",
      "extent inhibition sample recovery efficiency oocysts  33\n",
      "samples oocysts  34\n",
      "samples concentrations oocysts  35\n",
      "mpn pcr method  36\n",
      "filtrate grab samples  37\n",
      "oocysts  38\n",
      "association  39\n",
      "c parvum oocysts  40\n",
      "particles  41\n",
      "secondary effluent drawn wastewater plants  42\n",
      "conditions  43\n",
      "weak correlation degree association mean cell residence time system  44\n",
      "['Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Phenotype' 'Microorganism'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Phenotype' 'Habitat'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Phenotype' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat']\n",
      "(42, 300) (42,)\n",
      "  text                                       words         entity norm\n",
      "0   T3           detection cryptosporidium parvum   Microorganism  NaN\n",
      "1   T4                        secondary effluents         Habitat  NaN\n",
      "2   T5  probable number polymerase chain reaction         Habitat  NaN\n",
      "3   T6                                      assay         Habitat  NaN\n",
      "4   T7              polymerase chain reaction pcr       Phenotype  NaN\n",
      "BB-norm+ner-12934822.txt\n",
      "(42, 300)\n",
      "(42, 300) (42, 300)\n",
      "['OntoBiotope OBT:000007' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000113'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 1311' 'OntoBiotope OBT:001444'\n",
      " 'NCBI_Taxonomy 813' 'NCBI_Taxonomy 1255618' 'NCBI_Taxonomy 1051'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000031'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 10508'\n",
      " 'OntoBiotope OBT:001444' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 1255670' 'OntoBiotope OBT:000011'\n",
      " 'NCBI_Taxonomy 10508' 'OntoBiotope OBT:003269' 'NCBI_Taxonomy 10508'\n",
      " 'NCBI_Taxonomy 29412' 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000113'\n",
      " 'OntoBiotope OBT:000304' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000372' 'NCBI_Taxonomy 10508' 'OntoBiotope OBT:003269'\n",
      " 'NCBI_Taxonomy 53345' 'OntoBiotope OBT:000477' 'OntoBiotope OBT:000007'\n",
      " 'NCBI_Taxonomy 1255580' 'OntoBiotope OBT:000019']\n",
      "efficacies  3\n",
      "vancomycin  4\n",
      "arbekacin gentamicin  5\n",
      "combination methicillin  6\n",
      "resistant staphylococcus aureus  7\n",
      "infective endocarditis model  8\n",
      "vitro infective endocarditis model iviem  9\n",
      "compare  10\n",
      "efficacy  11\n",
      "vancomycin van arbekacin abk gentamicin gen  12\n",
      "combination  13\n",
      "strains  14\n",
      "isolated methicillin  15\n",
      "resistant staphylococcus  16\n",
      "gen  17\n",
      "susceptible gs  18\n",
      "gen  19\n",
      "resistant gr fibrin clots  20\n",
      "suspended iviem antibiotics  21\n",
      "boluses  22\n",
      "h q h q h q  23\n",
      "h  24\n",
      "continuous infusion van q h q h  25\n",
      "abk q h q h gen combination treatment van q h  26\n",
      "abk q h van q h  27\n",
      "gen q h  28\n",
      "fibrin clots  29\n",
      "model h  30\n",
      "bacterial densities  31\n",
      "number colonies  32\n",
      "fibrin clot  33\n",
      "study groups  34\n",
      "control groups  35\n",
      "p  36\n",
      "van abk  37\n",
      "alone number colonies  38\n",
      "lower gs gr  39\n",
      "h administration p  40\n",
      "lowest gs abk  41\n",
      "q h p h  42\n",
      "abk van  43\n",
      "equivalent bacterial reductions  44\n",
      "frequency gen resistance gr van  45\n",
      "abk  46\n",
      "additive effect till  47\n",
      "van  48\n",
      "gen  49\n",
      "indifference data  50\n",
      "suggest  51\n",
      "abk  52\n",
      "alternative van gen  53\n",
      "resistant staphylococcal endocarditis  54\n",
      "additive effect  55\n",
      "van abk  56\n",
      "gen  57\n",
      "resistant strains  58\n",
      "h  59\n",
      "studies  60\n",
      "clinical application combination  61\n",
      "['Habitat' 'Microorganism' 'Habitat' 'Phenotype' 'Microorganism' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Habitat' 'Microorganism'\n",
      " 'Phenotype' 'Phenotype' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Microorganism' 'Microorganism' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Microorganism' 'Phenotype' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Phenotype' 'Habitat' 'Habitat' 'Habitat' 'Phenotype'\n",
      " 'Microorganism' 'Habitat' 'Habitat']\n",
      "(59, 300) (59,)\n",
      "  text                             words         entity norm\n",
      "0   T3                       efficacies         Habitat  NaN\n",
      "1   T4                       vancomycin   Microorganism  NaN\n",
      "2   T5             arbekacin gentamicin         Habitat  NaN\n",
      "3   T6          combination methicillin       Phenotype  NaN\n",
      "4   T7  resistant staphylococcus aureus   Microorganism  NaN\n",
      "BB-norm+ner-14638480.txt\n",
      "(59, 300)\n",
      "(59, 300) (59, 300)\n",
      "['OntoBiotope OBT:000521' 'NCBI_Taxonomy 1246' 'NCBI_Taxonomy 273384'\n",
      " 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 1280' 'NCBI_Taxonomy 12814'\n",
      " 'NCBI_Taxonomy 246432' 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 12721'\n",
      " 'NCBI_Taxonomy 28875' 'NCBI_Taxonomy 83560' 'OntoBiotope OBT:000061'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:002817'\n",
      " 'OntoBiotope OBT:002870' 'OntoBiotope OBT:000113'\n",
      " 'OntoBiotope OBT:002870' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 244319' 'OntoBiotope OBT:001480'\n",
      " 'OntoBiotope OBT:002712' 'NCBI_Taxonomy 471872' 'NCBI_Taxonomy 263'\n",
      " 'NCBI_Taxonomy 1182512' 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000019'\n",
      " 'NCBI_Taxonomy 1582' 'OntoBiotope OBT:000113' 'OntoBiotope OBT:003188'\n",
      " 'OntoBiotope OBT:000152' 'OntoBiotope OBT:003188'\n",
      " 'OntoBiotope OBT:003188' 'NCBI_Taxonomy 585' 'NCBI_Taxonomy 294'\n",
      " 'NCBI_Taxonomy 260680' 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000491' 'NCBI_Taxonomy 294'\n",
      " 'NCBI_Taxonomy 12721' 'OntoBiotope OBT:000007' 'OntoBiotope OBT:003321'\n",
      " 'NCBI_Taxonomy 1243' 'NCBI_Taxonomy 29412' 'OntoBiotope OBT:002870'\n",
      " 'NCBI_Taxonomy 29412' 'OntoBiotope OBT:000007' 'OntoBiotope OBT:003321'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:002817'\n",
      " 'OntoBiotope OBT:003269' 'NCBI_Taxonomy 294' 'OntoBiotope OBT:002870'\n",
      " 'OntoBiotope OBT:000152' 'OntoBiotope OBT:002712'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000062']\n",
      "mesonia algae gen  3\n",
      "nov sp nov  4\n",
      "novel marine bacterium family  5\n",
      "green alga acrosiphonia sonderi k tz kornm  6\n",
      "taxonomic position  7\n",
      "heterotrophic aerobic gram  8\n",
      "negative non motile  9\n",
      "halophilic marine bacteria  10\n",
      "green alga acrosiphonia sonderi k tz kornm  11\n",
      "rdna sequence analysis  12\n",
      "strains  13\n",
      "members family  14\n",
      "flavobacteriaceae  15\n",
      "form  16\n",
      "distinct lineage basis phenotypic  17\n",
      "chemotaxonomic genotypic  18\n",
      "phylogenetic data novel bacteria  19\n",
      "mesonia algae gen  20\n",
      "nov sp nov type strain kmm kctc ccug  21\n",
      "['Habitat' 'Microorganism' 'Habitat' 'Microorganism' 'Habitat' 'Phenotype'\n",
      " 'Phenotype' 'Microorganism' 'Microorganism' 'Microorganism'\n",
      " 'Microorganism' 'Habitat' 'Microorganism' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Habitat']\n",
      "(19, 300) (19,)\n",
      "  text                                        words         entity norm\n",
      "0   T3                           mesonia algae gen         Habitat  NaN\n",
      "1   T4                                  nov sp nov   Microorganism  NaN\n",
      "2   T5               novel marine bacterium family         Habitat  NaN\n",
      "3   T6  green alga acrosiphonia sonderi k tz kornm   Microorganism  NaN\n",
      "4   T7                          taxonomic position         Habitat  NaN\n",
      "BB-norm+ner-14657131.txt\n",
      "(19, 300)\n",
      "(19, 300) (19, 300)\n",
      "['OntoBiotope OBT:001481' 'NCBI_Taxonomy 1255614' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 56810' 'OntoBiotope OBT:000061'\n",
      " 'NCBI_Taxonomy 1255591' 'OntoBiotope OBT:000061' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:002338' 'OntoBiotope OBT:000061'\n",
      " 'OntoBiotope OBT:000006' 'NCBI_Taxonomy 287' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:002806'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:001481'\n",
      " 'OntoBiotope OBT:000113']\n",
      "modulation fibronectin  3\n",
      "virulence factors  4\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "teicoplanin  5\n",
      "resistant derivative methicillin  6\n",
      "resistant staphylococcus aureus  7\n",
      "impact glycopeptide resistance  8\n",
      "molecular regulation staphylococcus  9\n",
      "aureus virulence attachment host tissues  10\n",
      "stable teicoplanin  11\n",
      "resistant methicillin  12\n",
      "resistant aureus mrsa strain teicoplanin  13\n",
      "susceptible mrsa parent strain mrgr exhibits  14\n",
      "high degree virulence rat model  15\n",
      "chronic foreign body  16\n",
      "mrsa infection levels  17\n",
      "fibronectin  18\n",
      "adhesion surface display fibronectin  19\n",
      "proteins  20\n",
      "teicoplanin  21\n",
      "resistant strain teicoplanin  22\n",
      "susceptible parent teicoplanin  23\n",
      "susceptible revertant strain rev  24\n",
      "tissue cage infection  25\n",
      "quantitative reverse transcription pcr qrt pcr  26\n",
      "twofold  27\n",
      "higher steady state levels  28\n",
      "fnba  29\n",
      "fnbb transcripts  30\n",
      "strain  31\n",
      "teicoplanin susceptible counterparts analysis  32\n",
      "global regulatory activities  33\n",
      "qrt  34\n",
      "pcr  35\n",
      "strong reduction  36\n",
      "steady state levels  37\n",
      "rnaiii  38\n",
      "rnaii teicoplanin  39\n",
      "resistant strain  40\n",
      "teicoplanin  41\n",
      "susceptible counterparts contrast sara mrna levels  42\n",
      "fivefold  43\n",
      "higher strain mrgr  44\n",
      "rev  45\n",
      "alternative transcription factor sigma b  46\n",
      "higher level functional activity teicoplanin resistant strain teicoplanin  47\n",
      "susceptible counterparts  48\n",
      "significant increases  49\n",
      "sigma  50\n",
      "b dependent asp mrna levels  51\n",
      "sara  52\n",
      "p promoter  53\n",
      "transcript levels  54\n",
      "pcr northern  55\n",
      "data  56\n",
      "provide  57\n",
      "evidence emergence glycopeptide resistance  58\n",
      "understood molecular pathways  59\n",
      "significant pleiotropic changes  60\n",
      "expression  61\n",
      "regulation  62\n",
      "major virulence genes  63\n",
      "molecular phenotypic changes  64\n",
      "bacterial adhesion colonization properties  65\n",
      "multiresistant  66\n",
      "organisms  67\n",
      "['Habitat' 'Phenotype' 'Habitat' 'Phenotype' 'Microorganism' 'Phenotype'\n",
      " 'Microorganism' 'Phenotype' 'Habitat' 'Phenotype' 'Microorganism'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Phenotype' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Phenotype' 'Habitat' 'Habitat' 'Microorganism' 'Habitat'\n",
      " 'Phenotype' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Phenotype' 'Phenotype' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Phenotype' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Phenotype' 'Habitat' 'Habitat' 'Phenotype' 'Habitat']\n",
      "(65, 300) (65,)\n",
      "  text                              words         entity norm\n",
      "0   T3            modulation fibronectin         Habitat  NaN\n",
      "1   T4                 virulence factors       Phenotype  NaN\n",
      "2   T5                       teicoplanin         Habitat  NaN\n",
      "3   T6  resistant derivative methicillin       Phenotype  NaN\n",
      "4   T7   resistant staphylococcus aureus   Microorganism  NaN\n",
      "BB-norm+ner-15273106.txt\n",
      "(65, 300)\n",
      "(65, 300) (65, 300)\n",
      "['NCBI_Taxonomy 35818' 'OntoBiotope OBT:002806' 'OntoBiotope OBT:001134'\n",
      " 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 1280' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:002732' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 246432'\n",
      " 'OntoBiotope OBT:002817' 'OntoBiotope OBT:002817' 'NCBI_Taxonomy 1582'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:003269' 'NCBI_Taxonomy 511' 'OntoBiotope OBT:000103'\n",
      " 'OntoBiotope OBT:000061' 'OntoBiotope OBT:001134'\n",
      " 'OntoBiotope OBT:000152' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000360' 'NCBI_Taxonomy 1255655' 'OntoBiotope OBT:002806'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1582'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000850' 'OntoBiotope OBT:001444'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000029' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:002817' 'OntoBiotope OBT:001134'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000113'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 783' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 29412' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000521' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 36656'\n",
      " 'OntoBiotope OBT:000029' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 56810'\n",
      " 'NCBI_Taxonomy 1359' 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:002674'\n",
      " 'OntoBiotope OBT:000114']\n",
      "mycoplasma otitis california calves  3\n",
      "retrospective study mycoplasma otitis california calves  4\n",
      "characterize  5\n",
      "demographic features "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " 6\n",
      "disease  7\n",
      "pathologic findings  8\n",
      "infection  9\n",
      "confirmed cases  10\n",
      "mycoplasma  11\n",
      "necropsied cattle  12\n",
      "animals calves  13\n",
      "age weeks months  14\n",
      "median age months  15\n",
      "percent cases  16\n",
      "dairy  17\n",
      "higher percent  18\n",
      "calves  19\n",
      "mycoplasma  20\n",
      "otitis males  21\n",
      "proportion cases  22\n",
      "mycoplasma  23\n",
      "otitis  24\n",
      "significant p seasonal distribution  25\n",
      "highest proportion spring  26\n",
      "lowest summer months infections  27\n",
      "middle inner ear  28\n",
      "suppurative inflammatory response  29\n",
      "extensive bony involvement  30\n",
      "species  31\n",
      "isolated ears  32\n",
      "bovis  33\n",
      "bovirhinis alkalescens  34\n",
      "concurrent pneumonia  35\n",
      "cases  36\n",
      "isolated lungs cases  37\n",
      "proportion mycoplasma otitis cases  38\n",
      "years  39\n",
      "importance  40\n",
      "risk factors  41\n",
      "lower incidence disease calves  42\n",
      "['Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Habitat'\n",
      " 'Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat']\n",
      "(40, 300) (40,)\n",
      "  text                                              words   entity norm\n",
      "0   T3               mycoplasma otitis california calves   Habitat  NaN\n",
      "1   T4  retrospective study mycoplasma otitis californ...  Habitat  NaN\n",
      "2   T5                                      characterize   Habitat  NaN\n",
      "3   T6                              demographic features   Habitat  NaN\n",
      "4   T7                                           disease   Habitat  NaN\n",
      "BB-norm+ner-15460321.txt\n",
      "(40, 300)\n",
      "(40, 300) (40, 300)\n",
      "['OntoBiotope OBT:003269' 'OntoBiotope OBT:003220' 'NCBI_Taxonomy 120652'\n",
      " 'NCBI_Taxonomy 29412' 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000375' 'NCBI_Taxonomy 1585' 'NCBI_Taxonomy 2093'\n",
      " 'OntoBiotope OBT:003572' 'OntoBiotope OBT:003572'\n",
      " 'OntoBiotope OBT:003188' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:001988'\n",
      " 'OntoBiotope OBT:003269' 'NCBI_Taxonomy 1255655' 'NCBI_Taxonomy 2093'\n",
      " 'NCBI_Taxonomy 260680' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 2093'\n",
      " 'NCBI_Taxonomy 10699' 'OntoBiotope OBT:001480' 'OntoBiotope OBT:000360'\n",
      " 'OntoBiotope OBT:000113' 'OntoBiotope OBT:001205' 'NCBI_Taxonomy 12721'\n",
      " 'OntoBiotope OBT:001821' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000061' 'NCBI_Taxonomy 1246' 'OntoBiotope OBT:000019'\n",
      " 'NCBI_Taxonomy 33892' 'OntoBiotope OBT:003269' 'OntoBiotope OBT:002785'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:001341'\n",
      " 'OntoBiotope OBT:003220' 'NCBI_Taxonomy 1236' 'OntoBiotope OBT:003220']\n",
      "diagnosis  6\n",
      "atrophic body gastritis  7\n",
      "chinese patients  8\n",
      "serum pepsinogen  9\n",
      "atrophic body gastritis  10\n",
      "abg common china  11\n",
      "histology  12\n",
      "endoscopy efficient  13\n",
      "reliable  14\n",
      "invasive procedure  15\n",
      "present study serum pepsinogen pg  16\n",
      "biomarker  17\n",
      "novel noninvasive test  18\n",
      "first option  19\n",
      "abg certain groups  20\n",
      "chinese  21\n",
      "study population  22\n",
      "dyspeptic patients  23\n",
      "mean age years  24\n",
      "f  25\n",
      "underwent diagnostic gastroscopy  26\n",
      "least  27\n",
      "biopsy specimens  28\n",
      "corpus stomach  29\n",
      "specimens site  30\n",
      "histological diagnosis blood samples  31\n",
      "elisa  32\n",
      "assays  33\n",
      "serum  34\n",
      "pepsinogen pgi pepsinogen ii pgii igg antibodies  35\n",
      "hp igg drawn endoscopy cut points  36\n",
      "receiver  37\n",
      "curves  38\n",
      "roc  39\n",
      "correlation serum pg atrophy  40\n",
      "antral mucosa mean serum concentration pgi  41\n",
      "p patients  42\n",
      "abg  43\n",
      "microg l  44\n",
      "normal mucosa nm non abg microg l microg l  45\n",
      "mean  46\n",
      "ratio pgi pgii  47\n",
      "lower p patients  48\n",
      "abg  49\n",
      "nm non abg difference serum pgi pgi pgii ratio patients  50\n",
      "h pylori infection  51\n",
      "abg area roc pgi pgi pgii ratio ci ci  52\n",
      "maximum youden index yi pgi pgi pgii ratio  53\n",
      "best cut point pgi microg l sensitivity specificity  54\n",
      "pgi  55\n",
      "pgii ratio microg l sensitivity specificity  56\n",
      "serum pgi pgii ratio  57\n",
      "sensitive specific  58\n",
      "assay  59\n",
      "corpus atrophy  60\n",
      "noninvasive indicative test diagnosis  61\n",
      "atrophic gastritis  62\n",
      "['Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Phenotype' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat']\n",
      "(57, 300) (57,)\n",
      "  text                     words   entity norm\n",
      "0   T6                diagnosis   Habitat  NaN\n",
      "1   T7  atrophic body gastritis   Habitat  NaN\n",
      "2   T8         chinese patients   Habitat  NaN\n",
      "3   T9         serum pepsinogen   Habitat  NaN\n",
      "4  T10  atrophic body gastritis   Habitat  NaN\n",
      "BB-norm+ner-15612668.txt\n",
      "(57, 300)\n",
      "(57, 300) (57, 300)\n",
      "['OntoBiotope OBT:001480' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000524'\n",
      " 'OntoBiotope OBT:003269' 'NCBI_Taxonomy 12721' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000614' 'OntoBiotope OBT:003220'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1243' 'NCBI_Taxonomy 1047'\n",
      " 'OntoBiotope OBT:003188' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:003188' 'OntoBiotope OBT:003220'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 35818' 'NCBI_Taxonomy 4952'\n",
      " 'OntoBiotope OBT:003334' 'OntoBiotope OBT:000641'\n",
      " 'OntoBiotope OBT:000970' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 114277'\n",
      " 'NCBI_Taxonomy 114277' 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000524'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 48409'\n",
      " 'OntoBiotope OBT:000006' 'NCBI_Taxonomy 1752' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000061' 'OntoBiotope OBT:003269' 'NCBI_Taxonomy 114277'\n",
      " 'OntoBiotope OBT:000524' 'OntoBiotope OBT:003269' 'NCBI_Taxonomy 246432'\n",
      " 'OntoBiotope OBT:002034' 'OntoBiotope OBT:000113' 'NCBI_Taxonomy 114277'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:003220'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:003269']\n",
      "haemophilus influenzae  3\n",
      "significant pathogen acute otitis media  4\n",
      "haemophilus influenzae  5\n",
      "major pathogen acute otitis media  6\n",
      "aom  7\n",
      "disease  8\n",
      "distinguishable caused streptococcus pneumoniae aom  9\n",
      "h influenzae  10\n",
      "older age recurrent disease antibiotics  11\n",
      "differ  12\n",
      "ability eradicate h influenzae  13\n",
      "middle ear space  14\n",
      "united states  15\n",
      "widespread pneumococcal vaccination  16\n",
      "importance h influenzae  17\n",
      "major therapeutic challenge treatment aom  18\n",
      "['Microorganism' 'Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Microorganism' 'Habitat' 'Habitat' 'Microorganism'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Habitat']\n",
      "(16, 300) (16,)\n",
      "  text                                     words         entity norm\n",
      "0   T3                   haemophilus influenzae   Microorganism  NaN\n",
      "1   T4  significant pathogen acute otitis media         Habitat  NaN\n",
      "2   T5                   haemophilus influenzae   Microorganism  NaN\n",
      "3   T6        major pathogen acute otitis media         Habitat  NaN\n",
      "4   T7                                      aom         Habitat  NaN\n",
      "BB-norm+ner-15626953.txt\n",
      "(16, 300)\n",
      "(16, 300) (16, 300)\n",
      "['NCBI_Taxonomy 727' 'NCBI_Taxonomy 246432' 'NCBI_Taxonomy 727'\n",
      " 'OntoBiotope OBT:003220' 'NCBI_Taxonomy 260680' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:003269' 'NCBI_Taxonomy 727' 'OntoBiotope OBT:003220'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:003473' 'NCBI_Taxonomy 260680' 'OntoBiotope OBT:002806'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:003269']\n",
      "use rifampin treatment infection  3\n",
      "due staphylococcus aureus  4\n",
      "infection  5\n",
      "due staphylococcus aureus  6\n",
      "source  7\n",
      "significant morbidity mortality "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " 8\n",
      "treatment  9\n",
      "prevalence resistance antibiotics  10\n",
      "modes  11\n",
      "staphylococcal resistance  12\n",
      "penicillinase production  13\n",
      "intrinsic resistance sabath associates  14\n",
      "third type resistance  15\n",
      "susceptibility growth inhibition tolerance  16\n",
      "lethal action bactericidal agents mechanism tolerance  17\n",
      "deficiency  18\n",
      "autolytic enzyme activity part bacteria  19\n",
      "secondary inhibition autolysins  20\n",
      "tolerant staphylococcal strains strains  21\n",
      "patients infections  22\n",
      "treatment cell  23\n",
      "active antibiotics  24\n",
      "vancomycin  25\n",
      "unique mechanism action  26\n",
      "pharmacokinetic properties  27\n",
      "rifampin  28\n",
      "active  29\n",
      "antistaphylococcal agents  30\n",
      "capacity kill  31\n",
      "intraleukocytic staphylococci  32\n",
      "present cases  33\n",
      "addition  34\n",
      "regimens case year  35\n",
      "old male fever cough dyspnea  36\n",
      "abscess  37\n",
      "leg swelling weeks  38\n",
      "prior admission culture purulent material elbow abscess  39\n",
      "staphylococcus aureus chest x ray  40\n",
      "bilateral septic embolism phleborheography  41\n",
      "partial deep vein occlusion  42\n",
      "ileofemoral vein case year  43\n",
      "old female fever chills  44\n",
      "cough weeks blood culture  45\n",
      "staphylococcus aureus chest x ray  46\n",
      "bilateral septic embolism pneumonia  47\n",
      "standard antibiotics  48\n",
      "adequate vitro sensitivity tests  49\n",
      "clinical cure  50\n",
      "rifampin  51\n",
      "regimens results  52\n",
      "suggest  53\n",
      "rifampin  54\n",
      "adjunct therapy  55\n",
      "staphylococcal infections  56\n",
      "['Habitat' 'Microorganism' 'Habitat' 'Microorganism' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Phenotype' 'Habitat' 'Phenotype' 'Phenotype' 'Habitat'\n",
      " 'Habitat' 'Phenotype' 'Phenotype' 'Habitat' 'Phenotype' 'Phenotype'\n",
      " 'Phenotype' 'Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism']\n",
      "(54, 300) (54,)\n",
      "  text                              words         entity norm\n",
      "0   T3  use rifampin treatment infection         Habitat  NaN\n",
      "1   T4         due staphylococcus aureus   Microorganism  NaN\n",
      "2   T5                         infection         Habitat  NaN\n",
      "3   T6         due staphylococcus aureus   Microorganism  NaN\n",
      "4   T7                            source         Habitat  NaN\n",
      "BB-norm+ner-1649682.txt\n",
      "(54, 300)\n",
      "(54, 300) (54, 300)\n",
      "['OntoBiotope OBT:003269' 'NCBI_Taxonomy 263' 'OntoBiotope OBT:000375'\n",
      " 'NCBI_Taxonomy 263' 'NCBI_Taxonomy 1256511' 'NCBI_Taxonomy 29412'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:002674'\n",
      " 'OntoBiotope OBT:002806' 'OntoBiotope OBT:002817' 'NCBI_Taxonomy 246432'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000152'\n",
      " 'OntoBiotope OBT:000029' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:003220' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:003220' 'OntoBiotope OBT:000061'\n",
      " 'OntoBiotope OBT:003269' 'NCBI_Taxonomy 1246' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000113' 'OntoBiotope OBT:002337'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:002844'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1582' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:003220' 'OntoBiotope OBT:001533'\n",
      " 'OntoBiotope OBT:003220' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:003245' 'OntoBiotope OBT:003269' 'NCBI_Taxonomy 1314'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000103'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:003245'\n",
      " 'OntoBiotope OBT:001533' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000521'\n",
      " 'OntoBiotope OBT:002337' 'NCBI_Taxonomy 1703' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:002337' 'OntoBiotope OBT:003269' 'NCBI_Taxonomy 727']\n",
      "results  3\n",
      "strains  4\n",
      "isolated patients  5\n",
      "ulcerative colitis colitis gravis  6\n",
      "adaptive colitis  7\n",
      "chronic catarrhal colitis  8\n",
      "e coli strains  9\n",
      "isolated patients  10\n",
      "ulcerative colitis patients  11\n",
      "chronic catarrhal colitis patients  12\n",
      "adaptive colitis patient e coli strains  13\n",
      "healthy persons control group  14\n",
      "reaction agglutination stage glass  15\n",
      "use  16\n",
      "commercial set  17\n",
      "h monovalent serum result  18\n",
      "investigation  19\n",
      "patients  20\n",
      "different intestinal inflammatory pathologies  21\n",
      "significant pathologic changes "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " 22\n",
      "intestinal microbiocenosis  23\n",
      "main role playing strains  24\n",
      "e  25\n",
      "coli violations  26\n",
      "intestinal microbiocenosis  27\n",
      "leading complications  28\n",
      "basic disease  29\n",
      "e coli strains  30\n",
      "formation  31\n",
      "pathologic microbiocenosis  32\n",
      "equal frequency  33\n",
      "part  34\n",
      "enteropathogenic enteroinvasive enterotoxic strains  35\n",
      "e  36\n",
      "coli  37\n",
      "definite  38\n",
      "serotypes  39\n",
      "e  40\n",
      "coli representatives  41\n",
      "normal intestinal microflora ability synthesis thermolabile enterotoxin  42\n",
      "['Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Microorganism' 'Microorganism' 'Habitat'\n",
      " 'Microorganism' 'Microorganism' 'Microorganism' 'Habitat']\n",
      "(40, 300) (40,)\n",
      "  text                               words         entity norm\n",
      "0   T3                            results         Habitat  NaN\n",
      "1   T4                            strains   Microorganism  NaN\n",
      "2   T5                  isolated patients         Habitat  NaN\n",
      "3   T6  ulcerative colitis colitis gravis         Habitat  NaN\n",
      "4   T7                   adaptive colitis         Habitat  NaN\n",
      "BB-norm+ner-16510930.txt\n",
      "(40, 300)\n",
      "(40, 300) (40, 300)\n",
      "['OntoBiotope OBT:000038' 'OntoBiotope OBT:000061'\n",
      " 'OntoBiotope OBT:003220' 'OntoBiotope OBT:002806'\n",
      " 'OntoBiotope OBT:002806' 'OntoBiotope OBT:003269' 'NCBI_Taxonomy 1010796'\n",
      " 'OntoBiotope OBT:003220' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000524'\n",
      " 'OntoBiotope OBT:002806' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:002480' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003220'\n",
      " 'OntoBiotope OBT:003285' 'OntoBiotope OBT:000476'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000110'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1311' 'OntoBiotope OBT:003308'\n",
      " 'OntoBiotope OBT:000110' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:003269' 'NCBI_Taxonomy 1010796' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000113' 'OntoBiotope OBT:001205' 'NCBI_Taxonomy 1255655'\n",
      " 'OntoBiotope OBT:003269' 'NCBI_Taxonomy 1311' 'OntoBiotope OBT:000061'\n",
      " 'OntoBiotope OBT:000061' 'NCBI_Taxonomy 813' 'NCBI_Taxonomy 1311'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019']\n",
      "differentiation vibrio vulnificus  3\n",
      "expression vvhba hemolysin gene  4\n",
      "luxs quorum sensing system  5\n",
      "good vitro model  6\n",
      "bacterial surface adherence colonization  7\n",
      "differentiation bacterium  8\n",
      "changes  9\n",
      "expression  10\n",
      "virulence factors  11\n",
      "invasiveness  12\n",
      "early stages infection study  13\n",
      "determine  14\n",
      "expression vvha  15\n",
      "hemolysin cytolysin vvha  16\n",
      "downregulated  17\n",
      "differentiation  18\n",
      "insertional inactivation vvha  19\n",
      "detectable effect expression v vulnificus  20\n",
      "motility  21\n",
      "vvha transcriptional reporter  22\n",
      "assay  23\n",
      "vvha expression  24\n",
      "v vulnificus  25\n",
      "planktonic v vulnificus  26\n",
      "expression vvha  27\n",
      "v vulnificus  28\n",
      "result  29\n",
      "deletional inactivation luxs gene  30\n",
      "results  31\n",
      "show  32\n",
      "vvha expression  33\n",
      "v vulnificus  34\n",
      "activity  35\n",
      "luxs  36\n",
      "quorum  37\n",
      "system  38\n",
      "vvha performs  39\n",
      "essential role invasiveness  40\n",
      "v vulnificus  41\n",
      "adherence colonization body surfaces  42\n",
      "early stages  43\n",
      "infection  44\n",
      "vvha  45\n",
      "significant role  46\n",
      "pathophysiological deterioration  47\n",
      "v vulnificus  48\n",
      "planktonic v vulnificus  49\n",
      "['Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Phenotype'\n",
      " 'Habitat' 'Habitat' 'Phenotype' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Microorganism' 'Habitat']\n",
      "(47, 300) (47,)\n",
      "  text                                      words         entity norm\n",
      "0   T3         differentiation vibrio vulnificus   Microorganism  NaN\n",
      "1   T4           expression vvhba hemolysin gene         Habitat  NaN\n",
      "2   T5                luxs quorum sensing system         Habitat  NaN\n",
      "3   T6                          good vitro model         Habitat  NaN\n",
      "4   T7  bacterial surface adherence colonization         Habitat  NaN\n",
      "BB-norm+ner-16728960.txt\n",
      "(47, 300)\n",
      "(47, 300) (47, 300)\n",
      "['OntoBiotope OBT:000524' 'OntoBiotope OBT:000031'\n",
      " 'OntoBiotope OBT:000031' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000029' 'OntoBiotope OBT:002806' 'NCBI_Taxonomy 119856'\n",
      " 'NCBI_Taxonomy 56810' 'OntoBiotope OBT:002806' 'OntoBiotope OBT:002806'\n",
      " 'NCBI_Taxonomy 1639' 'OntoBiotope OBT:000521' 'OntoBiotope OBT:002480'\n",
      " 'OntoBiotope OBT:003220' 'NCBI_Taxonomy 1313' 'NCBI_Taxonomy 1051'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 119856'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:002480' 'NCBI_Taxonomy 40269' 'NCBI_Taxonomy 234834'\n",
      " 'OntoBiotope OBT:002480' 'NCBI_Taxonomy 40269' 'NCBI_Taxonomy 1311'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000038'\n",
      " 'OntoBiotope OBT:000222' 'OntoBiotope OBT:002480' 'NCBI_Taxonomy 40269'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000143' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003269' 'NCBI_Taxonomy 40269'\n",
      " 'OntoBiotope OBT:002806' 'NCBI_Taxonomy 1977402' 'OntoBiotope OBT:000375'\n",
      " 'OntoBiotope OBT:003473' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:002806' 'NCBI_Taxonomy 40269' 'NCBI_Taxonomy 234834']\n",
      "use transposon promoter probe vectors  3\n",
      "metabolic  4\n",
      "engineering obligate methanotroph methylomonas  5\n",
      "sp  6\n",
      "strain  7\n",
      "c carotenoid synthesis  8\n",
      "recent expansion  9\n",
      "genetic genomic tools  10\n",
      "metabolic engineering  11\n",
      "development microorganisms  12\n",
      "industrial production  13\n",
      "compounds  14\n",
      "transposable elements  15\n",
      "chromosomal locations  16\n",
      "obligate  17\n",
      "methanotroph methylomonas  18\n",
      "sp  19\n",
      "strain support  20\n",
      "high level expression genes  21\n",
      "synthesis c carotenoids  22\n",
      "canthaxanthin  23\n",
      "astaxanthin  24\n",
      "promoterless carotenoid transposons  25\n",
      "chromosomal locations flics  26\n",
      "hsdm  27\n",
      "ccp cysh nirs regions  28\n",
      "total carotenoid synthesis  29\n",
      "fold carotenoid gene clusters  30\n",
      "chromosomal locations  31\n",
      "carotenoid gene clusters  32\n",
      "neutral locations  33\n",
      "control  34\n",
      "promoter gene  35\n",
      "resistance chloramphenicol  36\n",
      "chromosomal integration system  37\n",
      "sucrose lethality  38\n",
      "targeted gene deletions site  39\n",
      "specific integration  40\n",
      "carotenoid gene cluster  41\n",
      "methylomonas  42\n",
      "genome  43\n",
      "genetic scars  44\n",
      "chromosome  45\n",
      "antibiotic resistance genes  46\n",
      "present integration vector  47\n",
      "genetic approaches  48\n",
      "work demonstrate  49\n",
      "metabolic engineering microorganisms  50\n",
      "less studied environmental isolates  51\n",
      "integration sites  52\n",
      "chromosome host permit  53\n",
      "optimal expression target genes  54\n",
      "['Habitat' 'Habitat' 'Microorganism' 'Microorganism' 'Microorganism'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Phenotype' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Phenotype' 'Microorganism' 'Microorganism'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Phenotype'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Phenotype' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Phenotype' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat']\n",
      "(52, 300) (52,)\n",
      "  text                                            words         entity norm\n",
      "0   T3           use transposon promoter probe vectors         Habitat  NaN\n",
      "1   T4                                       metabolic         Habitat  NaN\n",
      "2   T5  engineering obligate methanotroph methylomonas   Microorganism  NaN\n",
      "3   T6                                              sp   Microorganism  NaN\n",
      "4   T7                                          strain   Microorganism  NaN\n",
      "BB-norm+ner-17261513.txt\n",
      "(52, 300)\n",
      "(52, 300) (52, 300)\n",
      "['OntoBiotope OBT:000031' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000011' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1582'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 260680'\n",
      " 'OntoBiotope OBT:002806' 'OntoBiotope OBT:000054'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000061' 'OntoBiotope OBT:003269' 'NCBI_Taxonomy 273384'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 114277'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:002711' 'OntoBiotope OBT:000061'\n",
      " 'OntoBiotope OBT:000113' 'NCBI_Taxonomy 83332' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000477'\n",
      " 'OntoBiotope OBT:000031' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:002772' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000106' 'OntoBiotope OBT:000648'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 47715'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:001142' 'NCBI_Taxonomy 1047'\n",
      " 'OntoBiotope OBT:002674' 'OntoBiotope OBT:002129'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000006' 'NCBI_Taxonomy 1182512'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000061']\n",
      "involvement cd cd l  3\n",
      "il expression cytotoxic  4\n",
      "bacterial viral "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " 5\n",
      "human cells  6\n",
      "regulation cytotoxic effector molecule expression  7\n",
      "human  8\n",
      "viral bacterial activation  9\n",
      "understood  10\n",
      "human autologous dendritic cells  11\n",
      "dcs prime lymphocytes  12\n",
      "virus hsv vaccinia virus  13\n",
      "intracellular bacteria  14\n",
      "listeria  15\n",
      "innocua listeria monocytogenes mycobacterium tuberculosis chlamydophila pneumoniae  16\n",
      "ctls contrast  17\n",
      "larger quantities  18\n",
      "ifn  19\n",
      "gamma tnf alpha  20\n",
      "listeria  21\n",
      "cultures  22\n",
      "granzyme  23\n",
      "b granulysin  24\n",
      "tested viruses  25\n",
      "intracellular bacteria  26\n",
      "hsv  27\n",
      "enhanced surface expression costimulatory  28\n",
      "molecule cd cd l  29\n",
      "dc  30\n",
      "enhanced secretion il  31\n",
      "cd  32\n",
      "il abs  33\n",
      "cell  34\n",
      "activation  35\n",
      "hsv dependent regulation perforin data  36\n",
      "indicate  37\n",
      "distinct ctl effector function response intracellular pathogens  38\n",
      "endogenous il production  39\n",
      "costimulation cd  40\n",
      "['Habitat' 'Habitat' 'Habitat' 'Habitat' 'Phenotype' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Phenotype' 'Microorganism'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism'\n",
      " 'Habitat' 'Habitat' 'Microorganism' 'Habitat' 'Phenotype' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Phenotype' 'Habitat' 'Habitat']\n",
      "(38, 300) (38,)\n",
      "  text                                              words     entity norm\n",
      "0   T3                               involvement cd cd l     Habitat  NaN\n",
      "1   T4                           il expression cytotoxic     Habitat  NaN\n",
      "2   T5                                   bacterial viral     Habitat  NaN\n",
      "3   T6                                       human cells     Habitat  NaN\n",
      "4   T7  regulation cytotoxic effector molecule express...  Phenotype  NaN\n",
      "BB-norm+ner-19494280.txt\n",
      "(38, 300)\n",
      "(38, 300) (38, 300)\n",
      "['OntoBiotope OBT:001623' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 666'\n",
      " 'OntoBiotope OBT:000061' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:002488' 'OntoBiotope OBT:002806'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000061'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:002669' 'NCBI_Taxonomy 1051'\n",
      " 'NCBI_Taxonomy 1637' 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1279' 'OntoBiotope OBT:001623'\n",
      " 'NCBI_Taxonomy 1637' 'OntoBiotope OBT:001030' 'NCBI_Taxonomy 1270'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 12721' 'NCBI_Taxonomy 1051'\n",
      " 'NCBI_Taxonomy 210' 'OntoBiotope OBT:000019' 'OntoBiotope OBT:001623'\n",
      " 'OntoBiotope OBT:002785' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:001623' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000706' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 120652' 'NCBI_Taxonomy 37763'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:001623']\n",
      "protection salmonella ampicillin  3\n",
      "resistant escherichia coli presence  4\n",
      "lethal drug concentrations  5\n",
      "microbial systems  6\n",
      "preferred  7\n",
      "grounds  8\n",
      "experimental work evolution traits  9\n",
      "benefit  10\n",
      "group members  11\n",
      "work  12\n",
      "conceptual theoretical models frequency  13\n",
      "dependent selection  14\n",
      "populations  15\n",
      "fruitful terms  16\n",
      "dynamics group  17\n",
      "beneficial public goods traits  18\n",
      "species  19\n",
      "expand  20\n",
      "scope  21\n",
      "microbial work evolution group  22\n",
      "beneficial traits case  23\n",
      "multi species communities  24\n",
      "affect  25\n",
      "human health  26\n",
      "beta lactamase  27\n",
      "escherichia coli  28\n",
      "sensitive cohorts species  29\n",
      "species  30\n",
      "human disease beta lactamase  31\n",
      "e coli "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " 32\n",
      "retain  33\n",
      "survival large number ampicillin  34\n",
      "sensitive cohorts salmonella  35\n",
      "enterica  36\n",
      "serovar typhimurium  37\n",
      "laboratory clinical isolates  38\n",
      "salmonella  39\n",
      "survivors  40\n",
      "sensitive ampicillin  41\n",
      "solid medium evidence gene transfer salmonella  42\n",
      "direct physical contact  43\n",
      "resistant e coli  44\n",
      "phenomenon  45\n",
      "involve  46\n",
      "release beta lactamase e  47\n",
      "coli present enterica  48\n",
      "imply  49\n",
      "resistant e coli pathogenic  50\n",
      "selfish respect e coli  51\n",
      "salmonella  52\n",
      "gain protection antibiotics  53\n",
      "e  54\n",
      "coli  55\n",
      "gene transfer  56\n",
      "consequence  57\n",
      "antibiotic resistant e coli  58\n",
      "play  59\n",
      "decisive role survival  60\n",
      "disease  61\n",
      "successful treatment  62\n",
      "['Phenotype' 'Phenotype' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Phenotype' 'Habitat' 'Microorganism' 'Microorganism' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Microorganism' 'Habitat' 'Microorganism' 'Habitat'\n",
      " 'Phenotype' 'Habitat' 'Habitat' 'Phenotype' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Microorganism' 'Microorganism' 'Habitat' 'Microorganism'\n",
      " 'Habitat' 'Microorganism' 'Microorganism' 'Habitat' 'Habitat' 'Phenotype'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat']\n",
      "(60, 300) (60,)\n",
      "  text                                 words     entity norm\n",
      "0   T3     protection salmonella ampicillin   Phenotype  NaN\n",
      "1   T4  resistant escherichia coli presence   Phenotype  NaN\n",
      "2   T5           lethal drug concentrations     Habitat  NaN\n",
      "3   T6                    microbial systems     Habitat  NaN\n",
      "4   T7                            preferred     Habitat  NaN\n",
      "BB-norm+ner-19656787.txt\n",
      "(60, 300)\n",
      "(60, 300) (60, 300)\n",
      "['NCBI_Taxonomy 727' 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000598'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:001200' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 260680'\n",
      " 'OntoBiotope OBT:001828' 'OntoBiotope OBT:000006'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000113'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 260680' 'OntoBiotope OBT:003269'\n",
      " 'NCBI_Taxonomy 1535326' 'OntoBiotope OBT:003188' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:001430'\n",
      " 'OntoBiotope OBT:000113' 'OntoBiotope OBT:002488'\n",
      " 'OntoBiotope OBT:003269' 'NCBI_Taxonomy 1045010' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1010796'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000061'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 28901' 'NCBI_Taxonomy 90371'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 599' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:003188' 'OntoBiotope OBT:002806' 'NCBI_Taxonomy 37763'\n",
      " 'NCBI_Taxonomy 1766' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 28901'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:002806'\n",
      " 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 599' 'OntoBiotope OBT:000007'\n",
      " 'NCBI_Taxonomy 1311' 'OntoBiotope OBT:000061' 'OntoBiotope OBT:001367'\n",
      " 'OntoBiotope OBT:002480' 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 63186'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003269' 'NCBI_Taxonomy 1703']\n",
      "microbiota  6\n",
      "pediatric inflammatory bowel disease  7\n",
      "test hypothesis  8\n",
      "controls children  9\n",
      "inflammatory bowel disease  10\n",
      "ibd exhibit differences  11\n",
      "relationships  12\n",
      "gut microbiota disease activity  13\n",
      "children adolescents  14\n",
      "n median age years  15\n",
      "ibd  16\n",
      "healthy controls  17\n",
      "median age years  18\n",
      "study disease activity  19\n",
      "pediatric ulcerative colitis activity index  20\n",
      "pediatric crohn disease activity index cell  21\n",
      "bacterial groups species  22\n",
      "fecal microbiota  23\n",
      "real time polymerase chain reaction analysis  24\n",
      "major changes  25\n",
      "patients  26\n",
      "ulcerative colitis  27\n",
      "decrease bifidobacteria  28\n",
      "active state ibd children  29\n",
      "active inactive crohn disease cd  30\n",
      "lower numbers  31\n",
      "faecalibacterium  32\n",
      "prausnitzii bifidobacteria  33\n",
      "p  34\n",
      "patients  35\n",
      "active cd  36\n",
      "higher numbers  37\n",
      "escherichia  38\n",
      "coli  39\n",
      "microbiota children cd  40\n",
      "decreased numbers  41\n",
      "f  42\n",
      "praunsitzii  43\n",
      "numbers  44\n",
      "e  45\n",
      "coli  46\n",
      "['Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism'\n",
      " 'Microorganism' 'Microorganism' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Microorganism' 'Habitat' 'Habitat' 'Microorganism'\n",
      " 'Microorganism' 'Habitat' 'Microorganism' 'Microorganism']\n",
      "(41, 300) (41,)\n",
      "  text                                  words   entity norm\n",
      "0   T6                            microbiota   Habitat  NaN\n",
      "1   T7  pediatric inflammatory bowel disease   Habitat  NaN\n",
      "2   T8                       test hypothesis   Habitat  NaN\n",
      "3   T9                     controls children   Habitat  NaN\n",
      "4  T10            inflammatory bowel disease   Habitat  NaN\n",
      "BB-norm+ner-20400104.txt\n",
      "(41, 300)\n",
      "(41, 300) (41, 300)\n",
      "['OntoBiotope OBT:000113' 'OntoBiotope OBT:003269' 'NCBI_Taxonomy 263'\n",
      " 'OntoBiotope OBT:003188' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000029'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:003188'\n",
      " 'OntoBiotope OBT:000113' 'NCBI_Taxonomy 54388' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:001169'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:003220'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000630'\n",
      " 'OntoBiotope OBT:003285' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:001623' 'OntoBiotope OBT:000031' 'NCBI_Taxonomy 1308'\n",
      " 'OntoBiotope OBT:002711' 'NCBI_Taxonomy 585' 'OntoBiotope OBT:003285'\n",
      " 'OntoBiotope OBT:001623' 'NCBI_Taxonomy 260680' 'NCBI_Taxonomy 90371'\n",
      " 'OntoBiotope OBT:000061' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:001386' 'NCBI_Taxonomy 35818' 'NCBI_Taxonomy 10699'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1311' 'OntoBiotope OBT:000061']\n",
      "month study safety efficacy  3\n",
      "courses  4\n",
      "lysine cystic fibrosis  5\n",
      "chronic airway infection pseudomonas  6\n",
      "aeruginosa  7\n",
      "pa causes morbidity mortality patients  8\n",
      "cystic fibrosis  9\n",
      "additional anti pa therapies  10\n",
      "improve  11\n",
      "health status health  12\n",
      "quality life air  13\n",
      "cf international month  14\n",
      "open label study  15\n",
      "safety efficacy  16\n",
      "courses  17\n",
      "aztreonam inhalation solution  18\n",
      "cayston patients  19\n",
      "years  20\n",
      "cf  21\n",
      "pa infection  22\n",
      "phase studies  23\n",
      "air  24\n",
      "cf air cf patients  25\n",
      "courses days days  26\n",
      "mg  27\n",
      "azli  28\n",
      "bid  29\n",
      "times  30\n",
      "tid  31\n",
      "randomization  32\n",
      "previous trials patients  33\n",
      "mean age years  34\n",
      "range  35\n",
      "years  36\n",
      "mean treatment  37\n",
      "high bid group tid group hospitalization rates  38\n",
      "low adverse events  39\n",
      "consistent cf course  40\n",
      "azli fev scores  41\n",
      "cystic fibrosis questionnaire  42\n",
      "respiratory symptom scale  43\n",
      "bacterial density sputum  44\n",
      "benefits  45\n",
      "days  46\n",
      "weight gain  47\n",
      "months  48\n",
      "decreases  49\n",
      "pa  50\n",
      "susceptibility  51\n",
      "dose  52\n",
      "response  53\n",
      "azli tid "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " 54\n",
      "patients  55\n",
      "greater improvements  56\n",
      "lung function respiratory symptoms months  57\n",
      "intermittent day  58\n",
      "azli treatment  59\n",
      "clinical benefits  60\n",
      "pulmonary function health  61\n",
      "quality life  62\n",
      "observed course therapy azli  63\n",
      "safe effective new therapy patients  64\n",
      "cf  65\n",
      "pa airway infection  66\n",
      "['Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Phenotype' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat']\n",
      "(64, 300) (64,)\n",
      "  text                                  words         entity norm\n",
      "0   T3           month study safety efficacy         Habitat  NaN\n",
      "1   T4                               courses         Habitat  NaN\n",
      "2   T5                lysine cystic fibrosis         Habitat  NaN\n",
      "3   T6  chronic airway infection pseudomonas         Habitat  NaN\n",
      "4   T7                            aeruginosa   Microorganism  NaN\n",
      "BB-norm+ner-20672296.txt\n",
      "(64, 300)\n",
      "(64, 300) (64, 300)\n",
      "['OntoBiotope OBT:002806' 'OntoBiotope OBT:000649'\n",
      " 'OntoBiotope OBT:000097' 'OntoBiotope OBT:002669' 'NCBI_Taxonomy 287'\n",
      " 'OntoBiotope OBT:003220' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 813'\n",
      " 'NCBI_Taxonomy 1582' 'NCBI_Taxonomy 63186' 'OntoBiotope OBT:000328'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:002806' 'OntoBiotope OBT:000649'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:001341' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000113' 'NCBI_Taxonomy 1703' 'OntoBiotope OBT:001341'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:003188'\n",
      " 'OntoBiotope OBT:000106' 'NCBI_Taxonomy 1047' 'NCBI_Taxonomy 12814'\n",
      " 'NCBI_Taxonomy 1766' 'NCBI_Taxonomy 1255580' 'NCBI_Taxonomy 1047'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1246'\n",
      " 'OntoBiotope OBT:001341' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000097' 'NCBI_Taxonomy 813' 'OntoBiotope OBT:000097'\n",
      " 'OntoBiotope OBT:001288' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:003269' 'NCBI_Taxonomy 1111734' 'NCBI_Taxonomy 615'\n",
      " 'OntoBiotope OBT:003184' 'OntoBiotope OBT:000110'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:002675' 'NCBI_Taxonomy 1428'\n",
      " 'NCBI_Taxonomy 12721' 'OntoBiotope OBT:003188' 'OntoBiotope OBT:003285'\n",
      " 'NCBI_Taxonomy 1350' 'OntoBiotope OBT:003220' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:003188' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1697' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:002806']\n",
      "tlr dependent  3\n",
      "heterologous modulation cc chemokine receptors  4\n",
      "human blood monocytes  5\n",
      "innate immune responses  6\n",
      "inflammatory cc chemokine receptors  7\n",
      "ccr  8\n",
      "ccr ccr mediate recruitment blood monocytes  9\n",
      "tissues  10\n",
      "cell migration response chemokines  11\n",
      "ccl  12\n",
      "toll  13\n",
      "receptors  14\n",
      "play  15\n",
      "essential role  16\n",
      "pathogen recognition  17\n",
      "monocytes  18\n",
      "demonstrate  19\n",
      "toll  20\n",
      "receptor tlr stimulation lipoteichoic acid  21\n",
      "lta staphylococcus aureus  22\n",
      "gradual modulation ccr ccr ccr plasma membrane  23\n",
      "human blood  24\n",
      "inhibits  25\n",
      "chemotaxis  26\n",
      "lta promote  27\n",
      "rapid desensitization chemokine  28\n",
      "calcium responses  29\n",
      "tlr crosstalk chemokine receptors  30\n",
      "dependent toll interleukin receptor domain  31\n",
      "adaptor protein  32\n",
      "phospholipase  33\n",
      "small g  34\n",
      "protein  35\n",
      "rac phorbol  36\n",
      "sensitive activation  37\n",
      "lta lead arrestin  38\n",
      "endocytosis ser  39\n",
      "ccr  40\n",
      "endosomes  41\n",
      "ccl  42\n",
      "treatment  43\n",
      "induced internalization ccr  44\n",
      "slower process  45\n",
      "phospholipase c  46\n",
      "phorbol  47\n",
      "sensitive phosphorylation  48\n",
      "overall data  49\n",
      "indicate  50\n",
      "tlr  51\n",
      "ccr ccr ccr  52\n",
      "human blood monocytes  53\n",
      "machinery  54\n",
      "support chemokine  55\n",
      "dependent modulation  56\n",
      "molecular mechanism  57\n",
      "monocyte migration pathogen recognition  58\n",
      "['Habitat' 'Phenotype' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Phenotype'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism'\n",
      " 'Habitat' 'Habitat' 'Phenotype' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Phenotype' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Microorganism' 'Habitat' 'Phenotype' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat']\n",
      "(56, 300) (56,)\n",
      "  text                                            words     entity norm\n",
      "0   T3                                   tlr dependent     Habitat  NaN\n",
      "1   T4  heterologous modulation cc chemokine receptors   Phenotype  NaN\n",
      "2   T5                           human blood monocytes     Habitat  NaN\n",
      "3   T6                         innate immune responses     Habitat  NaN\n",
      "4   T7             inflammatory cc chemokine receptors     Habitat  NaN\n",
      "BB-norm+ner-21148810.txt\n",
      "(56, 300)\n",
      "(56, 300) (56, 300)\n",
      "['OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:003001' 'OntoBiotope OBT:000029'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:001623'\n",
      " 'OntoBiotope OBT:000062' 'OntoBiotope OBT:000196' 'NCBI_Taxonomy 614'\n",
      " 'OntoBiotope OBT:000827' 'OntoBiotope OBT:001122'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 63186' 'NCBI_Taxonomy 1752'\n",
      " 'OntoBiotope OBT:002732' 'OntoBiotope OBT:003001'\n",
      " 'OntoBiotope OBT:000328' 'OntoBiotope OBT:001122'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000521'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:002806'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000375'\n",
      " 'OntoBiotope OBT:000061' 'OntoBiotope OBT:000062'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1182512' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:001623' 'OntoBiotope OBT:000463'\n",
      " 'OntoBiotope OBT:000827' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000061' 'OntoBiotope OBT:000029' 'NCBI_Taxonomy 813'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 120652' 'OntoBiotope OBT:002785'\n",
      " 'OntoBiotope OBT:001623' 'OntoBiotope OBT:003001'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:001623'\n",
      " 'OntoBiotope OBT:000061' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019']\n",
      "absorption  3\n",
      "linear dichroism  4\n",
      "single light  5\n",
      "system role disorder chlorosomes  6\n",
      "green photosynthetic bacteria  7\n",
      "chlorosomes  8\n",
      "light  9\n",
      "antennae photosynthetic bacteria  10\n",
      "large numbers  11\n",
      "self  12\n",
      "bacteriochlorophyll bchl molecules  13\n",
      "unique photophysical properties  14\n",
      "enable absorb light transfer excitation energy  15\n",
      "high efficiency  16\n",
      "molecular level organization  17\n",
      "photophysical properties  18\n",
      "bchl  19\n",
      "molecules aggregates  20\n",
      "understood  21\n",
      "reasons heterogeneity  22\n",
      "chlorosome  23\n",
      "structure  24\n",
      "rise  25\n",
      "hierarchy structural energy disorder report  26\n",
      "first time  27\n",
      "measure absorption  28\n",
      "linear dichroism  29\n",
      "ld individual  30\n",
      "chlorosomes  31\n",
      "fluorescence  32\n",
      "dimensional ld experiments  33\n",
      "reveal  34\n",
      "large amount disorder  35\n",
      "single chlorosome level form distributions  36\n",
      "ld  37\n",
      "observables chlorosomes  38\n",
      "wild  39\n",
      "type bacterium chlorobaculum tepidum fluorescence  40\n",
      "spectral parameters  41\n",
      "peak  42\n",
      "wavelength bandwidth measures  43\n",
      "aggregate  44\n",
      "excitonic properties parameters  45\n",
      "individual chlorosomes  46\n",
      "uncorrelated  47\n",
      "ld distributions  48\n",
      "indicate  49\n",
      "disorder  50\n",
      "due inner structural disorder  51\n",
      "chlorosome  52\n",
      "excitonic disorder  53\n",
      "present  54\n",
      "ld distributions  55\n",
      "values  56\n",
      "ld  57\n",
      "parameter distributions  58\n",
      "free effect  59\n",
      "structural disorder define range "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " 60\n",
      "excitonic dipole moment  61\n",
      "respect surface  62\n",
      "dimensional aggregate bchl  63\n",
      "experiments chlorosomes  64\n",
      "triple mutant chlorobaculum tepidum show  65\n",
      "mutant  66\n",
      "less inner structural disorder  67\n",
      "higher symmetry  68\n",
      "compatible model  69\n",
      "concentric cylinders  70\n",
      "different values transition  71\n",
      "dipole moment orientations  72\n",
      "consistent different molecular level organization bchl  73\n",
      "mutant wild type chlorosomes  74\n",
      "['Habitat' 'Habitat' 'Habitat' 'Habitat' 'Phenotype' 'Habitat' 'Habitat'\n",
      " 'Phenotype' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Phenotype' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat']\n",
      "(72, 300) (72,)\n",
      "  text                              words     entity norm\n",
      "0   T3                        absorption     Habitat  NaN\n",
      "1   T4                  linear dichroism     Habitat  NaN\n",
      "2   T5                      single light     Habitat  NaN\n",
      "3   T6  system role disorder chlorosomes     Habitat  NaN\n",
      "4   T7     green photosynthetic bacteria   Phenotype  NaN\n",
      "BB-norm+ner-21476570.txt\n",
      "(72, 300)\n",
      "(72, 300) (72, 300)\n",
      "['NCBI_Taxonomy 1334627' 'OntoBiotope OBT:000107' 'OntoBiotope OBT:000113'\n",
      " 'OntoBiotope OBT:002806' 'OntoBiotope OBT:000113'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000598' 'NCBI_Taxonomy 1046'\n",
      " 'OntoBiotope OBT:000103' 'OntoBiotope OBT:003451'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000061' 'NCBI_Taxonomy 1047'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:001231' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000006'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000107'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 777' 'OntoBiotope OBT:000598'\n",
      " 'OntoBiotope OBT:000103' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1255580'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:001288'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 781' 'OntoBiotope OBT:000019'\n",
      " 'NCBI_Taxonomy 1697' 'NCBI_Taxonomy 1697' 'OntoBiotope OBT:003188'\n",
      " 'NCBI_Taxonomy 12721' 'NCBI_Taxonomy 120652' 'NCBI_Taxonomy 260680'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000408' 'NCBI_Taxonomy 12721' 'NCBI_Taxonomy 12721'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1255580' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000118'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 1752'\n",
      " 'NCBI_Taxonomy 1255580' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 781'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000019']\n",
      "molecular epidemiology pasteurella multocida dairy beef calves  3\n",
      "molecular epidemiology pasteurella multocida  4\n",
      "farm level cattle  5\n",
      "aim  6\n",
      "study determine  7\n",
      "single multiple strains  8\n",
      "p  9\n",
      "multocida  10\n",
      "tend  11\n",
      "exist  12\n",
      "farms  13\n",
      "molecular characterisation  14\n",
      "isolates  15\n",
      "nasal swabs calves  16\n",
      "beef dairy farms  17\n",
      "scotland calves  18\n",
      "farms  19\n",
      "mayenne region france  20\n",
      "response respiratory disease  21\n",
      "p multocida isolates  22\n",
      "amplified polymorphic dna rapd  23\n",
      "pulsed field gel electrophoresis pfge  24\n",
      "restriction  25\n",
      "apai addition isolates  26\n",
      "representative  27\n",
      "farm rapd profile combination  28\n",
      "multilocus sequence  29\n",
      "mlst  30\n",
      "scottish isolates  31\n",
      "rapd  32\n",
      "profiles  33\n",
      "majority farms  34\n",
      "indistinguishable profiles  35\n",
      "positive animals  36\n",
      "farms  37\n",
      "profiles  38\n",
      "french isolates  39\n",
      "rapd  40\n",
      "profiles  41\n",
      "farm heterogeneity  42\n",
      "farms  43\n",
      "profile e  44\n",
      "calves  45\n",
      "profile e  46\n",
      "french isolates  47\n",
      "pfge  48\n",
      "discriminatory rapd  49\n",
      "results respect  50\n",
      "farm homogeneity heterogeneity  51\n",
      "whereas mlst discriminatory  52\n",
      "enough farm level epidemiology host species  53\n",
      "several strains  54\n",
      "dominant strain p multocida  55\n",
      "farms evidence role management factors movements  56\n",
      "farm number strains  57\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat']\n",
      "(55, 300) (55,)\n",
      "  text                                              words         entity norm\n",
      "0   T3  molecular epidemiology pasteurella multocida d...        Habitat  NaN\n",
      "1   T4      molecular epidemiology pasteurella multocida   Microorganism  NaN\n",
      "2   T5                                 farm level cattle         Habitat  NaN\n",
      "3   T6                                               aim         Habitat  NaN\n",
      "4   T7                                   study determine         Habitat  NaN\n",
      "BB-norm+ner-21511409.txt\n",
      "(55, 300)\n",
      "(55, 300) (55, 300)\n",
      "['OntoBiotope OBT:000019' 'OntoBiotope OBT:002806'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 386585' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 585' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000029' 'NCBI_Taxonomy 2750' 'OntoBiotope OBT:000630'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000097' 'NCBI_Taxonomy 1051'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 781' 'OntoBiotope OBT:000630'\n",
      " 'NCBI_Taxonomy 1111734' 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1173061'\n",
      " 'NCBI_Taxonomy 1766' 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:002338' 'NCBI_Taxonomy 1663' 'NCBI_Taxonomy 12333'\n",
      " 'NCBI_Taxonomy 511' 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000630'\n",
      " 'OntoBiotope OBT:002674' 'OntoBiotope OBT:000193'\n",
      " 'OntoBiotope OBT:000630' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:001341' 'NCBI_Taxonomy 511' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000630'\n",
      " 'OntoBiotope OBT:000061' 'NCBI_Taxonomy 1255655' 'OntoBiotope OBT:000061'\n",
      " 'OntoBiotope OBT:001341' 'OntoBiotope OBT:001058'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003269' 'NCBI_Taxonomy 1697'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000029' 'OntoBiotope OBT:000364']\n",
      "insights acinetobacter  3\n",
      "baumannii  4\n",
      "pathogenicity  5\n",
      "acinetobacter spp  6\n",
      "significant attention public  7\n",
      "scientific medical communities  8\n",
      "recent years  9\n",
      "acinetobacter baumannii become  10\n",
      "red alert human pathogen  11\n",
      "exceptional ability  12\n",
      "resistance  13\n",
      "available antibiotics  14\n",
      "characteristic  15\n",
      "unique abilities  16\n",
      "survive  17\n",
      "diverse range environments  18\n",
      "healthcare institutions  19\n",
      "problematic outbreaks  20\n",
      "virulence  21\n",
      "organism  22\n",
      "recent clinical reports  23\n",
      "suggest  24\n",
      "cause  25\n",
      "serious life  26\n",
      "infections  27\n",
      "metabolic adaptability  28\n",
      "selective advantage  29\n",
      "harsh hospital environments  30\n",
      "review  31\n",
      "focuses  32\n",
      "current understanding baumannii pathogenesis model systems  33\n",
      "study  34\n",
      "organism  35\n",
      "['Microorganism' 'Microorganism' 'Phenotype' 'Microorganism' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Phenotype'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Phenotype' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Phenotype' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat']\n",
      "(33, 300) (33,)\n",
      "  text                          words         entity norm\n",
      "0   T3        insights acinetobacter   Microorganism  NaN\n",
      "1   T4                     baumannii   Microorganism  NaN\n",
      "2   T5                 pathogenicity       Phenotype  NaN\n",
      "3   T6             acinetobacter spp   Microorganism  NaN\n",
      "4   T7  significant attention public         Habitat  NaN\n",
      "BB-norm+ner-21989983.txt\n",
      "(33, 300)\n",
      "(33, 300) (33, 300)\n",
      "['OntoBiotope OBT:000019' 'OntoBiotope OBT:000152'\n",
      " 'OntoBiotope OBT:000375' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000029' 'NCBI_Taxonomy 12721'\n",
      " 'NCBI_Taxonomy 127906' 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003245'\n",
      " 'OntoBiotope OBT:002869' 'NCBI_Taxonomy 10508' 'OntoBiotope OBT:002480'\n",
      " 'NCBI_Taxonomy 114277' 'NCBI_Taxonomy 386585' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000113'\n",
      " 'OntoBiotope OBT:000375' 'OntoBiotope OBT:000113'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:002817' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000189' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000113' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000113']\n",
      "characterization screening plant  3\n",
      "probiotic traits bacteria  4\n",
      "rice seeds  5\n",
      "argentina  6\n",
      "many seeds  7\n",
      "carry  8\n",
      "good chances  9\n",
      "colonization work  10\n",
      "seed  11\n",
      "borne bacterial flora rice varieties  12\n",
      "northeast argentina surface "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " 13\n",
      "seeds rice cultivars  14\n",
      "ct  15\n",
      "el paso camba irga  16\n",
      "average cfu g  17\n",
      "mesophilic copiotrophic bacteria  18\n",
      "microbiological physiological molecular characterization  19\n",
      "isolates  20\n",
      "ct  21\n",
      "seeds  22\n",
      "important diversity seed  23\n",
      "borne mesophiles  24\n",
      "potential plant  25\n",
      "probiotic activities  26\n",
      "diazotrophy antagonism  27\n",
      "fungal pathogens fact seed  28\n",
      "borne bacterial flora  29\n",
      "rice seedlings  30\n",
      "curvularia  31\n",
      "sp infection root colonization pattern pantoea  32\n",
      "seeds  33\n",
      "fluorescence microscopy  34\n",
      "axenic rice seedlings  35\n",
      "site emergence  36\n",
      "lateral roots lenticels  37\n",
      "entry sites  38\n",
      "endophytic  39\n",
      "findings  40\n",
      "suggest  41\n",
      "rice plants  42\n",
      "allow  43\n",
      "grain colonization  44\n",
      "bacterial species  45\n",
      "natural biofertilizers  46\n",
      "early seed germination  47\n",
      "['Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Phenotype' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Phenotype' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat']\n",
      "(45, 300) (45,)\n",
      "  text                              words   entity norm\n",
      "0   T3  characterization screening plant   Habitat  NaN\n",
      "1   T4         probiotic traits bacteria   Habitat  NaN\n",
      "2   T5                        rice seeds   Habitat  NaN\n",
      "3   T6                         argentina   Habitat  NaN\n",
      "4   T7                        many seeds   Habitat  NaN\n",
      "BB-norm+ner-22203552.txt\n",
      "(45, 300)\n",
      "(45, 300) (45, 300)\n",
      "['OntoBiotope OBT:000006' 'NCBI_Taxonomy 12333' 'OntoBiotope OBT:002806'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:003269' 'NCBI_Taxonomy 1582' 'NCBI_Taxonomy 1703'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000113'\n",
      " 'OntoBiotope OBT:002337' 'OntoBiotope OBT:000113'\n",
      " 'OntoBiotope OBT:000642' 'NCBI_Taxonomy 1255580' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000097' 'OntoBiotope OBT:000642' 'NCBI_Taxonomy 1585'\n",
      " 'NCBI_Taxonomy 1977402' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1173061'\n",
      " 'OntoBiotope OBT:000113' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:002806' 'OntoBiotope OBT:000113'\n",
      " 'OntoBiotope OBT:000152' 'NCBI_Taxonomy 47845' 'NCBI_Taxonomy 12721'\n",
      " 'NCBI_Taxonomy 1585' 'NCBI_Taxonomy 294' 'OntoBiotope OBT:000598'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:003220'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000598' 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 781'\n",
      " 'NCBI_Taxonomy 1697' 'OntoBiotope OBT:000360' 'NCBI_Taxonomy 256701'\n",
      " 'NCBI_Taxonomy 33892' 'OntoBiotope OBT:000019']\n",
      "mutational analysis  3\n",
      "n terminal domain  4\n",
      "urer positive transcriptional regulator  5\n",
      "urease gene expression  6\n",
      "escherichia coli plasmid  7\n",
      "urease virulence factor  8\n",
      "human animal infections  9\n",
      "urinary gastroduodenal tracts  10\n",
      "substrate urea present growth medium  11\n",
      "urea dependent urease expression  12\n",
      "transcriptional level arac  13\n",
      "activator  14\n",
      "urer  15\n",
      "previous work  16\n",
      "n terminal amino acid residues  17\n",
      "urer binds  18\n",
      "urea single site  19\n",
      "full length urer forms  20\n",
      "oligomer  21\n",
      "oligomerization motif  22\n",
      "n terminal portion molecule  23\n",
      "c  24\n",
      "terminal domain  25\n",
      "urer  26\n",
      "helix  27\n",
      "turn  28\n",
      "helix motifs  29\n",
      "necessary dna  30\n",
      "study  31\n",
      "mutational analyses  32\n",
      "n  33\n",
      "terminal domain  34\n",
      "urer determine domain  35\n",
      "similar arac family members  36\n",
      "urer mutants  37\n",
      "ability activate transcription lacz  38\n",
      "uredp lacz transcriptional fusion construct  39\n",
      "n terminal amino acids  40\n",
      "oligomer gel filtration  41\n",
      "dominant negative phenotype  42\n",
      "wild type urer allele  43\n",
      "hypothesize  44\n",
      "dominant negative phenotype results  45\n",
      "formation  46\n",
      "inactive heterodimers  47\n",
      "wild  48\n",
      "type  49\n",
      "urer dominant negative analysis cross  50\n",
      "assays  51\n",
      "coli urer active dimer dimerization  52\n",
      "first residues  53\n",
      "['Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Phenotype'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Phenotype' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Phenotype' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Phenotype'\n",
      " 'Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Phenotype'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Phenotype' 'Habitat' 'Phenotype']\n",
      "(51, 300) (51,)\n",
      "  text                                     words         entity norm\n",
      "0   T3                      mutational analysis         Habitat  NaN\n",
      "1   T4                        n terminal domain         Habitat  NaN\n",
      "2   T5  urer positive transcriptional regulator         Habitat  NaN\n",
      "3   T6                   urease gene expression         Habitat  NaN\n",
      "4   T7                 escherichia coli plasmid   Microorganism  NaN\n",
      "BB-norm+ner-22537874.txt\n",
      "(51, 300)\n",
      "(51, 300) (51, 300)\n",
      "['OntoBiotope OBT:000019' 'OntoBiotope OBT:003269' 'NCBI_Taxonomy 35818'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:002806'\n",
      " 'OntoBiotope OBT:000407' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:001738' 'OntoBiotope OBT:002480' 'NCBI_Taxonomy 120652'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000103' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:001623'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 263' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:002480' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 2369' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:003269' 'NCBI_Taxonomy 294' 'NCBI_Taxonomy 1752'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 263' 'NCBI_Taxonomy 260680'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 263' 'NCBI_Taxonomy 263'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:001480'\n",
      " 'OntoBiotope OBT:003338' 'OntoBiotope OBT:000641'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:001288' 'NCBI_Taxonomy 33889' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000062'\n",
      " 'OntoBiotope OBT:000019']\n",
      "taibaiella smilacinae gen  3\n",
      "nov sp  4\n",
      "nov endophytic member family chitinophagaceae  5\n",
      "isolated stem  6\n",
      "japonica  7\n",
      "description flavihumibacter petaseus  8\n",
      "light yellow  9\n",
      "bacterium  10\n",
      "strain ptjt  11\n",
      "stem smilacina japonica gray  12\n",
      "taibai mountain  13\n",
      "shaanxi province  14\n",
      "north west china  15\n",
      "taxonomic study  16\n",
      "polyphasic approach novel isolate  17\n",
      "c ph flexirubin type pigments  18\n",
      "cells  19\n",
      "gram reaction  20\n",
      "negative  21\n",
      "aerobic rod  22\n",
      "motile phylogenetic analysis  23\n",
      "rrna gene sequences  24\n",
      "strain ptjt member phylum  25\n",
      "highest sequence similarity lacibacter cauensis  26\n",
      "major cellular fatty acids  27\n",
      "iso  28\n",
      "c iso c g  29\n",
      "iso c iso  30\n",
      "c  31\n",
      "polyamine homospermidine  32\n",
      "major polar lipid phosphatidylethanolamine respiratory quinone mk dna g c content mol  33\n",
      "phylogenetic genotypic data strain ptjt  34\n",
      "represent novel species  35\n",
      "new genus family chitinophagaceae name taibaiella smilacinae gen  36\n",
      "nov sp nov  37\n",
      "type strain taibaiella smilacinae ptjt cctcc ab kctc  38\n",
      "description flavihumibacter petaseus  39\n",
      "['Microorganism' 'Microorganism' 'Microorganism' 'Habitat' 'Microorganism'\n",
      " 'Microorganism' 'Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Phenotype'\n",
      " 'Phenotype' 'Phenotype' 'Habitat' 'Habitat' 'Habitat' 'Microorganism'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Phenotype'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Microorganism'\n",
      " 'Microorganism' 'Microorganism']\n",
      "(37, 300) (37,)\n",
      "  text                                           words         entity norm\n",
      "0   T3                      taibaiella smilacinae gen   Microorganism  NaN\n",
      "1   T4                                         nov sp   Microorganism  NaN\n",
      "2   T5  nov endophytic member family chitinophagaceae   Microorganism  NaN\n",
      "3   T6                                  isolated stem         Habitat  NaN\n",
      "4   T7                                       japonica   Microorganism  NaN\n",
      "BB-norm+ner-23645023.txt\n",
      "(37, 300)\n",
      "(37, 300) (37, 300)\n",
      "['NCBI_Taxonomy 1288' 'NCBI_Taxonomy 191770' 'OntoBiotope OBT:000007'\n",
      " 'NCBI_Taxonomy 12333' 'OntoBiotope OBT:002034' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:001341' 'OntoBiotope OBT:001471'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000360'\n",
      " 'OntoBiotope OBT:000598' 'NCBI_Taxonomy 620' 'OntoBiotope OBT:002806'\n",
      " 'OntoBiotope OBT:000113' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000801'\n",
      " 'OntoBiotope OBT:000649' 'NCBI_Taxonomy 120652' 'NCBI_Taxonomy 256702'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000031'\n",
      " 'OntoBiotope OBT:000011' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:001341'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000742' 'NCBI_Taxonomy 263'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:003269' 'NCBI_Taxonomy 114277' 'OntoBiotope OBT:003269'\n",
      " 'NCBI_Taxonomy 1255614' 'OntoBiotope OBT:002806' 'OntoBiotope OBT:000007']\n",
      "molecular determinants  3\n",
      "cardiovascular collapse anthrax  4\n",
      "bacillus  5\n",
      "anthracis  6\n",
      "bipartite proteins  7\n",
      "lethal toxin edema factor "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " 8\n",
      "contribute  9\n",
      "progression anthrax  10\n",
      "shock  11\n",
      "anthrax  12\n",
      "prevents disease toxins  13\n",
      "main virulence factors  14\n",
      "bacterium  15\n",
      "anthrax bacterium anthrax  16\n",
      "trigger multi organ failure  17\n",
      "vascular permeability hemorrhage  18\n",
      "cardiac dysfunction animal challenge models  19\n",
      "recent study  20\n",
      "mice  21\n",
      "anthrax toxin receptor  22\n",
      "specific cells  23\n",
      "corresponding mice  24\n",
      "receptor  25\n",
      "specific cell types  26\n",
      "cardiovascular cells  27\n",
      "critical disease  28\n",
      "anthrax lethal toxin studies  29\n",
      "consistent involvement cardiovascular system increase  30\n",
      "cardiac failure markers  31\n",
      "human anthrax animal models  32\n",
      "b anthracis anthrax  33\n",
      "review  34\n",
      "discusses  35\n",
      "current state knowledge  36\n",
      "pathophysiology anthrax tries  37\n",
      "provide  38\n",
      "mechanistic model  39\n",
      "molecular determinants circulatory shock anthrax  40\n",
      "['Habitat' 'Habitat' 'Microorganism' 'Microorganism' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat']\n",
      "(38, 300) (38,)\n",
      "  text                             words         entity norm\n",
      "0   T3           molecular determinants         Habitat  NaN\n",
      "1   T4  cardiovascular collapse anthrax         Habitat  NaN\n",
      "2   T5                         bacillus   Microorganism  NaN\n",
      "3   T6                        anthracis   Microorganism  NaN\n",
      "4   T7               bipartite proteins         Habitat  NaN\n",
      "BB-norm+ner-24389148.txt\n",
      "(38, 300)\n",
      "(38, 300) (38, 300)\n",
      "['OntoBiotope OBT:000019' 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 1386'\n",
      " 'NCBI_Taxonomy 1279' 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000064' 'NCBI_Taxonomy 1255655' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:002806' 'OntoBiotope OBT:002806'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:001471'\n",
      " 'OntoBiotope OBT:002806' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:003211'\n",
      " 'OntoBiotope OBT:002669' 'OntoBiotope OBT:000061'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1111734' 'NCBI_Taxonomy 263'\n",
      " 'OntoBiotope OBT:001623' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:002806' 'OntoBiotope OBT:000189' 'NCBI_Taxonomy 1046'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003220'\n",
      " 'OntoBiotope OBT:000029' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:003269']\n",
      "destabilization fenna  3\n",
      "olson  4\n",
      "complex chlorobaculum tepidum  5\n",
      "fenna matthews  6\n",
      "olson  7\n",
      "fmo complex green sulfur bacterium chlorobaculum tepidum  8\n",
      "respect stability  9\n",
      "provide  10\n",
      "critical assessment  11\n",
      "optical spectra fmo complexes  12\n",
      "destabilize time  13\n",
      "spectral shifts  14\n",
      "samples  15\n",
      "higher hole  16\n",
      "efficiencies  17\n",
      "protein energy landscape  18\n",
      "correlated peak shifts  19\n",
      "nm  20\n",
      "suggest  21\n",
      "possible correlated protein fluctuations  22\n",
      "value cm  23\n",
      "reorganization energy e  24\n",
      "important implications contributions coherence rate  25\n",
      "kramer  26\n",
      "various  27\n",
      "studies  28\n",
      "dimensional electronic spectra  29\n",
      "demonstrate value  30\n",
      "likely cm  31\n",
      "suggest  32\n",
      "spectra  33\n",
      "literature  34\n",
      "different fmo samples  35\n",
      "exhibit  36\n",
      "varied peak positions  37\n",
      "due different purification isolation procedures destabilization effects  38\n",
      "['Habitat' 'Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Microorganism'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat']"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "(36, 300) (36,)\n",
      "  text                           words         entity norm\n",
      "0   T3          destabilization fenna         Habitat  NaN\n",
      "1   T4                          olson         Habitat  NaN\n",
      "2   T5  complex chlorobaculum tepidum   Microorganism  NaN\n",
      "3   T6                 fenna matthews         Habitat  NaN\n",
      "4   T7                          olson         Habitat  NaN\n",
      "BB-norm+ner-24584903.txt\n",
      "(36, 300)\n",
      "(36, 300) (36, 300)\n",
      "['OntoBiotope OBT:000019' 'NCBI_Taxonomy 1697' 'NCBI_Taxonomy 10699'\n",
      " 'OntoBiotope OBT:000031' 'NCBI_Taxonomy 1697' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:001480' 'OntoBiotope OBT:000029'\n",
      " 'OntoBiotope OBT:000113' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000113' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1400'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1255580' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000103' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 668' 'NCBI_Taxonomy 260680'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000113' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000118'\n",
      " 'OntoBiotope OBT:003269']\n",
      "comparative transcriptomics  3\n",
      "relevant cyanobacteria reveals  4\n",
      "transcriptome diversity  5\n",
      "prochlorococcus genus  6\n",
      "abundant  7\n",
      "important marine cyanobacteria  8\n",
      "present comprehensive comparison structure composition  9\n",
      "prochlorococcus strains  10\n",
      "similarities  11\n",
      "gene pool  12\n",
      "specific environmental constraints  13\n",
      "present genome wide maps  14\n",
      "transcriptional start  15\n",
      "tss organisms representatives  16\n",
      "diverse clades  17\n",
      "major ecotypes  18\n",
      "high low light conditions  19\n",
      "data  20\n",
      "suggest antisense transcription  21\n",
      "quarters genes  22\n",
      "bacteria  23\n",
      "hundreds tss  24\n",
      "genes  25\n",
      "prochlorosin genes  26\n",
      "strain  27\n",
      "mit  28\n",
      "direct comparison  29\n",
      "little conservation location  30\n",
      "tss nature non  31\n",
      "transcripts strains  32\n",
      "short untranslated regions  33\n",
      "median length nt  34\n",
      "mit  35\n",
      "protein  36\n",
      "genes  37\n",
      "median distance start codon  38\n",
      "shorter findings  39\n",
      "obvious shine dalgarno motif  40\n",
      "suggest  41\n",
      "leaderless translation  42\n",
      "ribosomal protein  43\n",
      "dependent translation constitute  44\n",
      "alternative mechanisms translation initiation prochlorococcus  45\n",
      "conclude  46\n",
      "genome wide antisense transcription  47\n",
      "major component transcriptional output  48\n",
      "small genomes  49\n",
      "hitherto  50\n",
      "unrecognized high degree complexity variability gene expression  51\n",
      "transcriptional architecture  52\n",
      "['Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Phenotype' 'Habitat' 'Microorganism' 'Phenotype'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat']\n",
      "(50, 300) (50,)\n",
      "  text                            words         entity norm\n",
      "0   T3     comparative transcriptomics         Habitat  NaN\n",
      "1   T4  relevant cyanobacteria reveals         Habitat  NaN\n",
      "2   T5         transcriptome diversity         Habitat  NaN\n",
      "3   T6           prochlorococcus genus   Microorganism  NaN\n",
      "4   T7                        abundant         Habitat  NaN\n",
      "BB-norm+ner-24739626.txt\n",
      "(50, 300)\n",
      "(50, 300) (50, 300)\n",
      "['OntoBiotope OBT:000103' 'NCBI_Taxonomy 1697' 'OntoBiotope OBT:000742'\n",
      " 'OntoBiotope OBT:002034' 'OntoBiotope OBT:000648'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:002806' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 83332'\n",
      " 'OntoBiotope OBT:000031' 'OntoBiotope OBT:002806'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 36656' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000113'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000113' 'NCBI_Taxonomy 1582' 'OntoBiotope OBT:000524'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 294'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003188' 'NCBI_Taxonomy 781'\n",
      " 'OntoBiotope OBT:000524' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1359' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:003220' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:003220' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000007']\n",
      "warm temperature acclimation protein  3\n",
      "important role  4\n",
      "inflammatory response turbot scophthalmus maximus  5\n",
      "wap molecule  6\n",
      "similar mammalian hemopexin serum glycoprotein  7\n",
      "high affinity "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " 8\n",
      "heme  9\n",
      "primary role  10\n",
      "iron metabolism  11\n",
      "heme  12\n",
      "plasma  13\n",
      "important role hemopexin inflammation  14\n",
      "acute phase protein production  15\n",
      "pro inflammatory cytokines  16\n",
      "immune induction fish wap genes  17\n",
      "species presents  18\n",
      "wap  19\n",
      "genes  20\n",
      "physiological functions  21\n",
      "fact  22\n",
      "transcriptional patterns  23\n",
      "wap  24\n",
      "genes  25\n",
      "stimulatory treatments  26\n",
      "variable contradictory present study  27\n",
      "wap genes  28\n",
      "wap  29\n",
      "wap  30\n",
      "first time turbot scophthalmus  31\n",
      "constitutive expression  32\n",
      "differential modulation  33\n",
      "thermal treatments  34\n",
      "immune challenges  35\n",
      "bacterial viral  36\n",
      "iron supplementation  37\n",
      "genes  38\n",
      "tested tissues  39\n",
      "whereas  40\n",
      "wap wap  41\n",
      "temperature rise  42\n",
      "bacterial challenge vhsv infection  43\n",
      "expression genes  44\n",
      "iron  45\n",
      "dextran administration  46\n",
      "overexpression wap  47\n",
      "induction  48\n",
      "head kidney buy  49\n",
      "effect wap protein  50\n",
      "turbot serum hemin  51\n",
      "agarose affinity chromatography  52\n",
      "demonstrate possible anti inflammatory role  53\n",
      "inhibitory effect  54\n",
      "migration  55\n",
      "zymosan injection peritoneal cavity  56\n",
      "['Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Phenotype'\n",
      " 'Habitat' 'Habitat']\n",
      "(54, 300) (54,)\n",
      "  text                                              words   entity norm\n",
      "0   T3              warm temperature acclimation protein   Habitat  NaN\n",
      "1   T4                                    important role   Habitat  NaN\n",
      "2   T5  inflammatory response turbot scophthalmus maxi...  Habitat  NaN\n",
      "3   T6                                      wap molecule   Habitat  NaN\n",
      "4   T7    similar mammalian hemopexin serum glycoprotein   Habitat  NaN\n",
      "BB-norm+ner-24794581.txt\n",
      "(54, 300)\n",
      "(54, 300) (54, 300)\n",
      "['OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:002714' 'OntoBiotope OBT:000062'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003220'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 246432' 'OntoBiotope OBT:000019'\n",
      " 'NCBI_Taxonomy 28875' 'NCBI_Taxonomy 1255670' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 1255670' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000061' 'NCBI_Taxonomy 1255670' 'NCBI_Taxonomy 1255670'\n",
      " 'NCBI_Taxonomy 210' 'OntoBiotope OBT:003269' 'OntoBiotope OBT:002674'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 666'\n",
      " 'NCBI_Taxonomy 1752' 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:003269' 'NCBI_Taxonomy 1255670' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:002806' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:002480' 'OntoBiotope OBT:002806'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1334627' 'OntoBiotope OBT:002806'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1582' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:001028' 'NCBI_Taxonomy 1255655']\n",
      "mycobacterium tuberculosis  3\n",
      "single outpatient  4\n",
      "clinic panama city exhibit  5\n",
      "wide genetic diversity  6\n",
      "mycobacterium tuberculosis biodiversity transmission  7\n",
      "significant tuberculosis control  8\n",
      "short report  9\n",
      "determine genetic diversity tuberculosis  10\n",
      "outpatient clinic panama city  11\n",
      "total tuberculosis isolates  12\n",
      "loci mycobacterial  13\n",
      "repetitive units  14\n",
      "variable number tandem  15\n",
      "miru  16\n",
      "vntr  17\n",
      "forty  18\n",
      "isolates  19\n",
      "unique miru vntr genotypes  20\n",
      "clusters  21\n",
      "isolates  22\n",
      "polyclonal miru vntr genotypes  23\n",
      "miru  24\n",
      "vntr hunter gaston discriminatory index  25\n",
      "analysis  26\n",
      "tuberculosis families  27\n",
      "latin american mediterranean harlem beijing findings  28\n",
      "suggest  29\n",
      "wide genetic diversity tuberculosis  30\n",
      "outpatient clinic  31\n",
      "molecular epidemiology survey  32\n",
      "second massive immigration  33\n",
      "local panama  34\n",
      "canal expansion activities  35\n",
      "['Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat']\n",
      "(33, 300) (33,)\n",
      "  text                                              words         entity norm\n",
      "0   T3                        mycobacterium tuberculosis   Microorganism  NaN\n",
      "1   T4                                 single outpatient         Habitat  NaN\n",
      "2   T5                        clinic panama city exhibit         Habitat  NaN\n",
      "3   T6                            wide genetic diversity         Habitat  NaN\n",
      "4   T7  mycobacterium tuberculosis biodiversity transm...        Habitat  NaN\n",
      "BB-norm+ner-24865686.txt\n",
      "(33, 300)\n",
      "(33, 300) (33, 300)\n",
      "['NCBI_Taxonomy 83332' 'OntoBiotope OBT:003220' 'OntoBiotope OBT:002694'\n",
      " 'OntoBiotope OBT:000113' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:002806' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:002694'\n",
      " 'OntoBiotope OBT:003188' 'OntoBiotope OBT:000061'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 1400'\n",
      " 'NCBI_Taxonomy 471872' 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000097'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 12721' 'OntoBiotope OBT:000097'\n",
      " 'OntoBiotope OBT:003269' 'NCBI_Taxonomy 1400' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000097'\n",
      " 'OntoBiotope OBT:000006' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:002694'\n",
      " 'OntoBiotope OBT:000011' 'NCBI_Taxonomy 1766' 'OntoBiotope OBT:003269'\n",
      " 'NCBI_Taxonomy 1582']\n",
      "wolbachia endosymbionts  3\n",
      "human disease control  4\n",
      "human filarial nematode parasites  5\n",
      "arthropods  6\n",
      "hosts  7\n",
      "bacterial endosymbiont wolbachia filaria  8\n",
      "wolbachia  9\n",
      "normal development fertility survival  10\n",
      "arthropods  11\n",
      "parasitic influence development reproduction  12\n",
      "host survival  13\n",
      "due obligate nature  14\n",
      "filarial parasites  15\n",
      "wolbachia  16\n",
      "drug discovery  17\n",
      "several approaches  18\n",
      "diversity  19\n",
      "library  20\n",
      "genomic sequence analysis vitro vivo anti  21\n",
      "antibiotic treatments  22\n",
      "adulticidal activity  23\n",
      "goal  24\n",
      "filarial parasite drug discovery mosquitoes  25\n",
      "presence wolbachia inhibit transmission  26\n",
      "certain viruses  27\n",
      "dengue  28\n",
      "yellow fever  29\n",
      "west  30\n",
      "infectivity malaria  31\n",
      "protozoan plasmodium  32\n",
      "filarial nematodes  33\n",
      "wolbachia  34\n",
      "cause form  35\n",
      "conditional sterility  36\n",
      "suppress populations  37\n",
      "mosquitoes  38\n",
      "additional  39\n",
      "important insects  40\n",
      "wolbachia  41\n",
      "pandemic endosymbiont  42\n",
      "great potential elimination  43\n",
      "wide variety  44\n",
      "human diseases  45\n",
      "['Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Phenotype' 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Phenotype' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Habitat'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat']\n",
      "(43, 300) (43,)\n",
      "  text                               words   entity norm\n",
      "0   T3            wolbachia endosymbionts   Habitat  NaN\n",
      "1   T4              human disease control   Habitat  NaN\n",
      "2   T5  human filarial nematode parasites   Habitat  NaN\n",
      "3   T6                         arthropods   Habitat  NaN\n",
      "4   T7                              hosts   Habitat  NaN\n",
      "BB-norm+ner-25046729.txt\n",
      "(43, 300)\n",
      "(43, 300) (43, 300)\n",
      "['OntoBiotope OBT:000061' 'OntoBiotope OBT:002806'\n",
      " 'OntoBiotope OBT:003206' 'OntoBiotope OBT:000029' 'NCBI_Taxonomy 29340'\n",
      " 'NCBI_Taxonomy 813' 'NCBI_Taxonomy 633' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000029' 'NCBI_Taxonomy 1173061' 'NCBI_Taxonomy 263'\n",
      " 'OntoBiotope OBT:000061' 'NCBI_Taxonomy 1288' 'NCBI_Taxonomy 633'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000113' 'OntoBiotope OBT:000029'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 246432' 'OntoBiotope OBT:000007'\n",
      " 'NCBI_Taxonomy 1977402' 'OntoBiotope OBT:000031' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:003451'\n",
      " 'OntoBiotope OBT:000524' 'NCBI_Taxonomy 1255655' 'OntoBiotope OBT:000641'\n",
      " 'NCBI_Taxonomy 780' 'NCBI_Taxonomy 1051' 'NCBI_Taxonomy 633'\n",
      " 'OntoBiotope OBT:002480' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 12814' 'OntoBiotope OBT:000563'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 633' 'NCBI_Taxonomy 36656'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000113'\n",
      " 'OntoBiotope OBT:001625']\n",
      "emergence prevention measures  7\n",
      "multidrug  8\n",
      "resistant pseudomonas "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " 9\n",
      "aeruginosa  10\n",
      "catheter  11\n",
      "urinary tract infection  12\n",
      "spinal cord injury patients  13\n",
      "evaluate measures  14\n",
      "multidrug  15\n",
      "resistant pseudomonas aeruginosa mdrp catheter  16\n",
      "urinary tract infection  17\n",
      "spinal cord injury patients  18\n",
      "spinal cord injury unit  19\n",
      "hyogo prefectural hyogo  20\n",
      "prefectural rehabilitation center kobe japan  21\n",
      "mdrp resistance amikacin imipenem levofloxacin  22\n",
      "cases  23\n",
      "mdrp  24\n",
      "cauti  25\n",
      "neurogenic bladder patients  26\n",
      "spinal cord injury months  27\n",
      "pulse field gel electrophoresis pfge  28\n",
      "epidemiological studies  29\n",
      "prevention measures  30\n",
      "mdrp  31\n",
      "emergence  32\n",
      "nd month surveillance  33\n",
      "cauti infection control  34\n",
      "outcomes measures  35\n",
      "total months  36\n",
      "pfge results  37\n",
      "mdrp isolates  38\n",
      "related strains  39\n",
      "outbreak  40\n",
      "cauti  41\n",
      "ratio mdrp  42\n",
      "decrease months surveillance  43\n",
      "th quarter  44\n",
      "th th months  45\n",
      "st quarter  46\n",
      "st nd months  47\n",
      "urinary tract device usage  48\n",
      "period p  49\n",
      "experienced outbreak emergent mdrp  50\n",
      "cauti  51\n",
      "neurogenic bladder patients  52\n",
      "spinal cord injury  53\n",
      "preventive measures  54\n",
      "outbreak  55\n",
      "surveillance  56\n",
      "decrease  57\n",
      "ratio mdrp  58\n",
      "['Habitat' 'Phenotype' 'Phenotype' 'Microorganism' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Phenotype' 'Phenotype' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Phenotype' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat']\n",
      "(52, 300) (52,)\n",
      "  text                           words         entity norm\n",
      "0   T7  emergence prevention measures         Habitat  NaN\n",
      "1   T8                      multidrug       Phenotype  NaN\n",
      "2   T9          resistant pseudomonas       Phenotype  NaN\n",
      "3  T10                     aeruginosa   Microorganism  NaN\n",
      "4  T11                       catheter         Habitat  NaN\n",
      "BB-norm+ner-25179651.txt\n",
      "(52, 300)\n",
      "(52, 300) (52, 300)\n",
      "['OntoBiotope OBT:002806' 'OntoBiotope OBT:002806'\n",
      " 'OntoBiotope OBT:002669' 'NCBI_Taxonomy 287' 'NCBI_Taxonomy 1069533'\n",
      " 'OntoBiotope OBT:000360' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:002806'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000360'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1255621'\n",
      " 'NCBI_Taxonomy 1255621' 'OntoBiotope OBT:003269' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003146'\n",
      " 'OntoBiotope OBT:000716' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:002337' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:001231' 'NCBI_Taxonomy 114277'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:003263'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 781'\n",
      " 'OntoBiotope OBT:002806' 'OntoBiotope OBT:003146'\n",
      " 'OntoBiotope OBT:000061' 'OntoBiotope OBT:001386'\n",
      " 'OntoBiotope OBT:000524' 'NCBI_Taxonomy 12721' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:001984' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:003146' 'OntoBiotope OBT:000716'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:002806' 'NCBI_Taxonomy 114277' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000061']\n",
      "biocontainment  3\n",
      "organisms synthetic protein design  4\n",
      "organisms gmos  5\n",
      "large scales  6\n",
      "open environments  7\n",
      "genetic biocontainment strategies  8\n",
      "prevent  9\n",
      "unintended proliferation gmos  10\n",
      "natural ecosystems  11\n",
      "biocontainment methods  12\n",
      "insufficient impose evolutionary pressure organism  13\n",
      "eject  14\n",
      "spontaneous mutagenesis  15\n",
      "horizontal gene transfer  16\n",
      "available compounds  17\n",
      "redesign "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " 18\n",
      "essential enzymes  19\n",
      "first organism  20\n",
      "genetic code escherichia coli strain c  21\n",
      "confer  22\n",
      "metabolic dependence  23\n",
      "non standard amino acids  24\n",
      "gmos  25\n",
      "bypass biocontainment mechanisms  26\n",
      "environmental compounds  27\n",
      "unprecedented resistance  28\n",
      "evolutionary escape mutagenesis  29\n",
      "horizontal gene transfer work  30\n",
      "foundation safer gmos  31\n",
      "natural ecosystems reliance  32\n",
      "synthetic metabolites  33\n",
      "['Habitat' 'Habitat' 'Phenotype' 'Habitat' 'Habitat' 'Phenotype' 'Habitat'\n",
      " 'Phenotype' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Phenotype' 'Habitat' 'Microorganism' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Phenotype' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat']\n",
      "(31, 300) (31,)\n",
      "  text                                words     entity norm\n",
      "0   T3                      biocontainment     Habitat  NaN\n",
      "1   T4  organisms synthetic protein design     Habitat  NaN\n",
      "2   T5                      organisms gmos   Phenotype  NaN\n",
      "3   T6                        large scales     Habitat  NaN\n",
      "4   T7                   open environments     Habitat  NaN\n",
      "BB-norm+ner-25607366.txt\n",
      "(31, 300)\n",
      "(31, 300) (31, 300)\n",
      "['OntoBiotope OBT:000524' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000061' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 260680' 'OntoBiotope OBT:000038'\n",
      " 'OntoBiotope OBT:002806' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:001231' 'NCBI_Taxonomy 1697' 'NCBI_Taxonomy 1752'\n",
      " 'OntoBiotope OBT:000524' 'OntoBiotope OBT:000113'\n",
      " 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 1255655' 'OntoBiotope OBT:000019'\n",
      " 'NCBI_Taxonomy 263' 'NCBI_Taxonomy 37763' 'OntoBiotope OBT:000029'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:002712' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000113'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019']\n",
      "comparative genome  3\n",
      "rickettsia rickettsii strains  4\n",
      "differ  5\n",
      "virulence  6\n",
      "rickettsia rickettsii  7\n",
      "obligate  8\n",
      "intracellular pathogen  9\n",
      "causative agent  10\n",
      "rocky mountain  11\n",
      "fever strains  12\n",
      "r  13\n",
      "rickettsii differ  14\n",
      "virulence guinea pig model infection severity disease  15\n",
      "fever response varies  16\n",
      "virulent  17\n",
      "sheila smith  18\n",
      "iowa  19\n",
      "identify potential determinants virulence r rickettsii genomes  20\n",
      "additional strains  21\n",
      "comparison  22\n",
      "sequences  23\n",
      "comparative genome  24\n",
      "cgs r rickettsii  25\n",
      "morgan r strains  26\n",
      "avirulent r rickettsii iowa virulent r rickettsii sheila smith strains  27\n",
      "montana  28\n",
      "strains  29\n",
      "sheila  30\n",
      "smith r  31\n",
      "similar eastern strains  32\n",
      "iowa  33\n",
      "similar major surface antigen  34\n",
      "rickettsial outer membrane protein  35\n",
      "truncated iowa strain region  36\n",
      "tandem repeats  37\n",
      "snps  38\n",
      "nonsynonymous r morgan strains  39\n",
      "smith additional snps  40\n",
      "major surface antigen autotransporter rompb exhibits  41\n",
      "defect  42\n",
      "iowa strain beta fragment  43\n",
      "sequence analysis ompb reveals  44\n",
      "identical sequences  45\n",
      "iowa  46\n",
      "morgan strains  47\n",
      "r  48\n",
      "sheila smith strains number  49\n",
      "snps insertions deletions sequences  50\n",
      "montana  51\n",
      "strains  52\n",
      "eastern strains  53\n",
      "low  54\n",
      "field  55\n",
      "possible virulence factors  56\n",
      "['Habitat' 'Microorganism' 'Habitat' 'Phenotype' 'Microorganism'\n",
      " 'Phenotype' 'Phenotype' 'Phenotype' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Phenotype' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism'\n",
      " 'Microorganism' 'Microorganism' 'Habitat' 'Microorganism' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Phenotype' 'Habitat' 'Habitat'\n",
      " 'Phenotype' 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Phenotype']\n",
      "(54, 300) (54,)\n",
      "  text                           words         entity norm\n",
      "0   T3             comparative genome         Habitat  NaN\n",
      "1   T4  rickettsia rickettsii strains   Microorganism  NaN\n",
      "2   T5                         differ         Habitat  NaN\n",
      "3   T6                      virulence       Phenotype  NaN\n",
      "4   T7          rickettsia rickettsii   Microorganism  NaN\n",
      "BB-norm+ner-25644009.txt\n",
      "(54, 300)\n",
      "(54, 300) (54, 300)\n",
      "['OntoBiotope OBT:000360' 'NCBI_Taxonomy 780' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000375' 'NCBI_Taxonomy 780' 'NCBI_Taxonomy 273384'\n",
      " 'OntoBiotope OBT:000375' 'OntoBiotope OBT:002732'\n",
      " 'OntoBiotope OBT:002829' 'OntoBiotope OBT:001821'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:002669' 'NCBI_Taxonomy 12721'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000375' 'NCBI_Taxonomy 1255670'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000031'\n",
      " 'OntoBiotope OBT:003188' 'OntoBiotope OBT:002806'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000360' 'NCBI_Taxonomy 36656'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 4952'\n",
      " 'OntoBiotope OBT:000061' 'OntoBiotope OBT:003451'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000064'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:002806'\n",
      " 'OntoBiotope OBT:000103' 'NCBI_Taxonomy 569' 'OntoBiotope OBT:003188'\n",
      " 'OntoBiotope OBT:000106' 'NCBI_Taxonomy 781' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 263' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000062'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 260680' 'NCBI_Taxonomy 11216'\n",
      " 'NCBI_Taxonomy 4952' 'OntoBiotope OBT:000061' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:002034'\n",
      " 'OntoBiotope OBT:002806']\n",
      "production cellulase  3\n",
      "free alkaline xylanase bacillus pumilus mtcc  4\n",
      "fermentation application biobleaching  5\n",
      "production cellulase  6\n",
      "free alkaline xylanase bacillus pumilus mtcc  7\n",
      "fermentation application  8\n",
      "various process parameters  9\n",
      "xylanase production b pumilus  10\n",
      "plackett burman design  11\n",
      "response surface methodology  12\n",
      "statistical methods aid  13\n",
      "enzyme yield  14\n",
      "individual crucial components  15\n",
      "medium maximum production  16\n",
      "extract magnesium sulphate h  17\n",
      "inoculum age incubation temperature  18\n",
      "c ph  19\n",
      "conditions  20\n",
      "xylanase  21\n",
      "activity iu ml media engineering  22\n",
      "fold increase enzyme production  23\n",
      "electron microscopy sem  24\n",
      "significant changes  25\n",
      "surface  26\n",
      "xylanase  27\n",
      "pulps result  28\n",
      "xylan hydrolysis  29\n",
      "roughness paper carton  30\n",
      "apparent  31\n",
      "electron micrograph  32\n",
      "due  33\n",
      "micro fibrils  34\n",
      "present surface xylanase action  35\n",
      "pulp show change results  36\n",
      "b pumilus mtcc xylanase  37\n",
      "effective bio  38\n",
      "paper carton  39\n",
      "['Phenotype' 'Microorganism' 'Habitat' 'Phenotype' 'Microorganism'\n",
      " 'Habitat' 'Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Phenotype' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Habitat' 'Phenotype' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Habitat'\n",
      " 'Habitat']\n",
      "(37, 300) (37,)\n",
      "  text                                          words         entity norm\n",
      "0   T3                          production cellulase       Phenotype  NaN\n",
      "1   T4  free alkaline xylanase bacillus pumilus mtcc   Microorganism  NaN\n",
      "2   T5         fermentation application biobleaching         Habitat  NaN\n",
      "3   T6                          production cellulase       Phenotype  NaN\n",
      "4   T7  free alkaline xylanase bacillus pumilus mtcc   Microorganism  NaN\n",
      "BB-norm+ner-26155675.txt\n",
      "(37, 300)\n",
      "(37, 300) (37, 300)\n",
      "['OntoBiotope OBT:000019' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1255600'\n",
      " 'OntoBiotope OBT:000029' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:001288' 'NCBI_Taxonomy 813' 'NCBI_Taxonomy 1255580'\n",
      " 'NCBI_Taxonomy 12814' 'NCBI_Taxonomy 260680' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1977402' 'OntoBiotope OBT:000007'\n",
      " 'NCBI_Taxonomy 12814' 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000019'\n",
      " 'NCBI_Taxonomy 260680' 'NCBI_Taxonomy 29412' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:001341' 'OntoBiotope OBT:003004'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1585'\n",
      " 'NCBI_Taxonomy 1697' 'OntoBiotope OBT:000019']\n",
      "carriage  3\n",
      "influenzae healthy gambian children  4\n",
      "throat samples  5\n",
      "different seasons  6\n",
      "different communities  7\n",
      "gambia  8\n",
      "west africa carriage rate haemophilus influenzae type b  9\n",
      "higher  10\n",
      "open communities duration carriage  11\n",
      "short less months  12\n",
      "h  13\n",
      "influenzae b  14\n",
      "carriers  15\n",
      "previous  16\n",
      "survey children  17\n",
      "years  18\n",
      "old "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " 19\n",
      "h influenzae b throat  20\n",
      "children  21\n",
      "older years  22\n",
      "high carriage rate correlate wet  23\n",
      "dry season carriage rate children  24\n",
      "rural areas  25\n",
      "similar children  26\n",
      "urban areas children day care  27\n",
      "nurseries  28\n",
      "low carriage rate carriage rate h influenzae b  29\n",
      "presence h  30\n",
      "influenzae subspecies  31\n",
      "random  32\n",
      "sample  33\n",
      "h influenzae subspecies  34\n",
      "children years  35\n",
      "old  36\n",
      "strains  37\n",
      "h influenzae  38\n",
      "thirds type b capsule types  39\n",
      "meningitis cases  40\n",
      "survey populations  41\n",
      "conclude  42\n",
      "prevalence h influenzae b  43\n",
      "open gambian communities  44\n",
      "similar  45\n",
      "communities  46\n",
      "kinetics  47\n",
      "different  48\n",
      "communities persistence infection  49\n",
      "gambian children  50\n",
      "short  51\n",
      "['Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Microorganism' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Microorganism'\n",
      " 'Habitat' 'Habitat' 'Microorganism' 'Microorganism' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat']\n",
      "(49, 300) (49,)\n",
      "  text                                 words   entity norm\n",
      "0   T3                             carriage   Habitat  NaN\n",
      "1   T4  influenzae healthy gambian children   Habitat  NaN\n",
      "2   T5                       throat samples   Habitat  NaN\n",
      "3   T6                    different seasons   Habitat  NaN\n",
      "4   T7                different communities   Habitat  NaN\n",
      "BB-norm+ner-2617654.txt\n",
      "(49, 300)\n",
      "(49, 300) (49, 300)\n",
      "['OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000113' 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 12721'\n",
      " 'NCBI_Taxonomy 1047' 'OntoBiotope OBT:000113' 'OntoBiotope OBT:002806'\n",
      " 'OntoBiotope OBT:002712' 'NCBI_Taxonomy 36656' 'OntoBiotope OBT:002806'\n",
      " 'NCBI_Taxonomy 1047' 'OntoBiotope OBT:002806' 'OntoBiotope OBT:001341'\n",
      " 'OntoBiotope OBT:000708' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:003188' 'OntoBiotope OBT:002480' 'NCBI_Taxonomy 781'\n",
      " 'OntoBiotope OBT:002806' 'NCBI_Taxonomy 1051' 'OntoBiotope OBT:003188'\n",
      " 'OntoBiotope OBT:003188' 'OntoBiotope OBT:001480'\n",
      " 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 63186' 'NCBI_Taxonomy 549'\n",
      " 'OntoBiotope OBT:000113' 'NCBI_Taxonomy 1279' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:003188' 'OntoBiotope OBT:000708'\n",
      " 'OntoBiotope OBT:000061' 'NCBI_Taxonomy 727' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:002034'\n",
      " 'OntoBiotope OBT:003220' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000113' 'NCBI_Taxonomy 585' 'OntoBiotope OBT:000113'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019']\n",
      "construction host vector system  3\n",
      "candida maltosa  4\n",
      "ars site  5\n",
      "genome  6\n",
      "construct host vector system  7\n",
      "n alkane  8\n",
      "candida maltosa isolation ars site  9\n",
      "genome  10\n",
      "c maltosa  11\n",
      "leu mutants  12\n",
      "c  13\n",
      "maltosa  14\n",
      "gene library  15\n",
      "yep leu vector leu transformants  16\n",
      "high frequency plasmid  17\n",
      "isolated recipient cells pcs  18\n",
      "kilobase kb fragment c  19\n",
      "genome kb  20\n",
      "fragment ars activity  21\n",
      "tra transformation ability region vectors  22\n",
      "ptra  23\n",
      "ptra c  24\n",
      "maltosa  25\n",
      "fragment pbr leu gene saccharomyces  26\n",
      "cerevisiae  27\n",
      "transformation c maltosa j plasmids  28\n",
      "successful spheroplast lithium acetate methods  29\n",
      "southern blot analysis  30\n",
      "copy number  31\n",
      "ptra c maltosa  32\n",
      "growth  33\n",
      "selective pressure medium  34\n",
      "vectors  35\n",
      "cerevisiae leu leu  36\n",
      "tra region  37\n",
      "ars site  38\n",
      "specific c maltosa  39\n",
      "['Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Habitat' 'Microorganism' 'Habitat' 'Microorganism'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Microorganism'\n",
      " 'Habitat' 'Microorganism' 'Microorganism' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Microorganism']"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "(37, 300) (37,)\n",
      "  text                             words         entity norm\n",
      "0   T3  construction host vector system         Habitat  NaN\n",
      "1   T4                  candida maltosa   Microorganism  NaN\n",
      "2   T5                         ars site         Habitat  NaN\n",
      "3   T6                           genome         Habitat  NaN\n",
      "4   T7     construct host vector system         Habitat  NaN\n",
      "BB-norm+ner-3015879.txt\n",
      "(37, 300)\n",
      "(37, 300) (37, 300)\n",
      "['OntoBiotope OBT:001030' 'NCBI_Taxonomy 1535326' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:003188'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:003211' 'NCBI_Taxonomy 263' 'OntoBiotope OBT:002711'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000061' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000113' 'OntoBiotope OBT:000031' 'NCBI_Taxonomy 1255655'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:002806' 'NCBI_Taxonomy 1280'\n",
      " 'OntoBiotope OBT:002711' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000113' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 263'\n",
      " 'NCBI_Taxonomy 263' 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:003220' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007']\n",
      "identification  3\n",
      "nonfimbrial adhesive factor  4\n",
      "enterotoxigenic escherichia coli strain  5\n",
      "enterotoxigenic escherichia coli strain strain  6\n",
      "patient acute infantile diarrhea  7\n",
      "brush border  8\n",
      "human intestinal epithelial cells  9\n",
      "strain  10\n",
      "hemagglutinate human bovine chicken guinea pig  11\n",
      "adhesion e coli  12\n",
      "nonfimbrial bacterial surface protein daltons  13\n",
      "bacteria degrees  14\n",
      "c  15\n",
      "min surface protein  16\n",
      "adhesive factor pretreatment  17\n",
      "protein extract  18\n",
      "inhibits adhesion e coli  19\n",
      "adhesive factor  20\n",
      "distinct adhesive factors  21\n",
      "enterotoxigenic e coli  22\n",
      "plasmid dna megadaltons  23\n",
      "synthesis nonfimbrial adhesive factor  24\n",
      "['Habitat' 'Habitat' 'Microorganism' 'Microorganism' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Microorganism' 'Habitat' 'Habitat']\n",
      "(22, 300) (22,)\n",
      "  text                                            words         entity norm\n",
      "0   T3                                  identification         Habitat  NaN\n",
      "1   T4                     nonfimbrial adhesive factor         Habitat  NaN\n",
      "2   T5         enterotoxigenic escherichia coli strain   Microorganism  NaN\n",
      "3   T6  enterotoxigenic escherichia coli strain strain   Microorganism  NaN\n",
      "4   T7                patient acute infantile diarrhea         Habitat  NaN\n",
      "BB-norm+ner-3516874.txt\n",
      "(22, 300)\n",
      "(22, 300) (22, 300)\n",
      "['NCBI_Taxonomy 12333' 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 1010796'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003269' 'NCBI_Taxonomy 1535326'\n",
      " 'OntoBiotope OBT:000061' 'NCBI_Taxonomy 1582' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:002806' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000648' 'NCBI_Taxonomy 263' 'NCBI_Taxonomy 48409'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:001623'\n",
      " 'OntoBiotope OBT:000029' 'OntoBiotope OBT:002806'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019']\n",
      "sequence regulation expression  3\n",
      "gene aeromonas  4\n",
      "hydrophila  5\n",
      "lexa gene aeromonas  6\n",
      "hydrophila  7\n",
      "specific  8\n",
      "step  9\n",
      "system  10\n",
      "ah lexa repressor  11\n",
      "able block escherichia coli ec sos gene expression  12\n",
      "activated reca protein  13\n",
      "bacterial species  14\n",
      "dna  15\n",
      "damage ah lexa  16\n",
      "protein  17\n",
      "amino acids  18\n",
      "identical lexa repressor  19\n",
      "ec  20\n",
      "ec  21\n",
      "sos boxes  22\n",
      "upstream ah lexa distance bp  23\n",
      "whereas distance ec lexa bp structure sequence dna  24\n",
      "domain lexa repressor  25\n",
      "region hydrolysis  26\n",
      "ah lexa  27\n",
      "residue region  28\n",
      "specific cleavage reaction  29\n",
      "present  30\n",
      "reca  31\n",
      "cleavable repressors  32\n",
      "ah lexa expression ah lexa dna damage "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " 33\n",
      "inducible ah ec  34\n",
      "genetic backgrounds  35\n",
      "extent contrast ec  36\n",
      "dna  37\n",
      "ah cells  38\n",
      "['Habitat' 'Microorganism' 'Microorganism' 'Microorganism' 'Microorganism'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Phenotype' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat']\n",
      "(36, 300) (36,)\n",
      "  text                            words         entity norm\n",
      "0   T3  sequence regulation expression         Habitat  NaN\n",
      "1   T4                  gene aeromonas   Microorganism  NaN\n",
      "2   T5                      hydrophila   Microorganism  NaN\n",
      "3   T6             lexa gene aeromonas   Microorganism  NaN\n",
      "4   T7                      hydrophila   Microorganism  NaN\n",
      "BB-norm+ner-7867953.txt\n",
      "(36, 300)\n",
      "(36, 300) (36, 300)\n",
      "['OntoBiotope OBT:003269' 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 644'\n",
      " 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 644' 'NCBI_Taxonomy 263'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:001623' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 256701' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000477' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:001134'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 83560' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 12721' 'OntoBiotope OBT:000524'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:002806' 'OntoBiotope OBT:000113'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:001623']\n",
      "spread vibrio cholerae india  3\n",
      "['Microorganism']\n",
      "(1, 300) (1,)\n",
      "  text                          words         entity norm\n",
      "0   T3  spread vibrio cholerae india   Microorganism  NaN\n",
      "BB-norm+ner-7876637.txt\n",
      "(1, 300)\n",
      "(1, 300) (1, 300)\n",
      "['OntoBiotope OBT:003269']\n",
      "tissue dispersion cell  3\n",
      "harvest fluid suspension culture  4\n",
      "use  5\n",
      "bacterial neutral protease  6\n",
      "bacterial neutral protease bacillus polymyxa  7\n",
      "disperse mammalian tissues cells  8\n",
      "primary cell cultures  9\n",
      "several tissues treatments  10\n",
      "kunitz  11\n",
      "unit  12\n",
      "ml protease  13\n",
      "phosphate buffer solution  14\n",
      "salt solution tissue culture medium  15\n",
      "serum monolayer  16\n",
      "wither early passage levels  17\n",
      "strains  18\n",
      "treatment protease  19\n",
      "monolayer removal  20\n",
      "culture strain l  21\n",
      "monodisperse suspension presence protease contrast trypsin protease  22\n",
      "active presence serum  23\n",
      "stable incubation  24\n",
      "injured cells  25\n",
      "['Habitat' 'Habitat' 'Habitat' 'Phenotype' 'Microorganism' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Phenotype' 'Habitat' 'Habitat' 'Habitat']\n",
      "(23, 300) (23,)\n",
      "  text                                          words         entity norm\n",
      "0   T3                        tissue dispersion cell         Habitat  NaN\n",
      "1   T4              harvest fluid suspension culture         Habitat  NaN\n",
      "2   T5                                           use         Habitat  NaN\n",
      "3   T6                    bacterial neutral protease       Phenotype  NaN\n",
      "4   T7  bacterial neutral protease bacillus polymyxa   Microorganism  NaN\n",
      "BB-norm+ner-817056.txt\n",
      "(23, 300)\n",
      "(23, 300) (23, 300)\n",
      "['OntoBiotope OBT:000062' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:002480' 'OntoBiotope OBT:000375'\n",
      " 'OntoBiotope OBT:002806' 'OntoBiotope OBT:000061'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003188'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000328'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000061'\n",
      " 'OntoBiotope OBT:001174' 'OntoBiotope OBT:000061'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000019']\n",
      "virulence determinants  3\n",
      "nontoxinogenic  4\n",
      "escherichia  5\n",
      "coli strains  6\n",
      "cause  7\n",
      "infantile diarrhea  8\n",
      "ten sorbitol  9\n",
      "escherichia coli strains  10\n",
      "cause  11\n",
      "infantile diarrhea  12\n",
      "positive fluorescence actin  13\n",
      "test  14\n",
      "negative shiga  15\n",
      "toxin slt genes  16\n",
      "complete eae genes  17\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "slt  18\n",
      "e coli restriction fragment length polymorphism analysis  19\n",
      "nucleotide sequence analysis bp  20\n",
      "stretch end eae data  21\n",
      "presence  22\n",
      "eae genotypes  23\n",
      "serogroup  24\n",
      "eae  25\n",
      "enteropathogenic e coli  26\n",
      "epec  27\n",
      "strain e  28\n",
      "nontoxinogenic e coli  29\n",
      "eae ehec strain edl  30\n",
      "toxinogenic e coli  31\n",
      "epec  32\n",
      "specific virulence determinant  33\n",
      "large plasmids  34\n",
      "epec adherence factor sequences  35\n",
      "slt  36\n",
      "negative e coli  37\n",
      "described fall  38\n",
      "epec serves  39\n",
      "explanation  40\n",
      "virulence infants  41\n",
      "represent  42\n",
      "third pathogroup  43\n",
      "serogroup  44\n",
      "['Phenotype' 'Habitat' 'Microorganism' 'Microorganism' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Phenotype' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Habitat' 'Microorganism' 'Microorganism' 'Microorganism'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Microorganism' 'Microorganism'\n",
      " 'Microorganism' 'Habitat' 'Phenotype' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Microorganism']\n",
      "(42, 300) (42,)\n",
      "  text                    words         entity norm\n",
      "0   T3  virulence determinants       Phenotype  NaN\n",
      "1   T4          nontoxinogenic         Habitat  NaN\n",
      "2   T5             escherichia   Microorganism  NaN\n",
      "3   T6            coli strains   Microorganism  NaN\n",
      "4   T7                   cause         Habitat  NaN\n",
      "BB-norm+ner-8406892.txt\n",
      "(42, 300)\n",
      "(42, 300) (42, 300)\n",
      "['OntoBiotope OBT:000019' 'NCBI_Taxonomy 246432' 'NCBI_Taxonomy 90371'\n",
      " 'NCBI_Taxonomy 263' 'OntoBiotope OBT:003269' 'OntoBiotope OBT:002806'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000152'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:002806'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 263' 'NCBI_Taxonomy 12814'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:002829' 'NCBI_Taxonomy 1582' 'OntoBiotope OBT:003269'\n",
      " 'NCBI_Taxonomy 1255621' 'OntoBiotope OBT:000375' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:002806' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:002806' 'NCBI_Taxonomy 813'\n",
      " 'OntoBiotope OBT:002806' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000524' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000113' 'NCBI_Taxonomy 1173061' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1173061' 'OntoBiotope OBT:000375']\n",
      "macrophages  3\n",
      "borrelia  4\n",
      "burgdorferi  5\n",
      "lyme arthritis hamsters  6\n",
      "mechanism  7\n",
      "study  8\n",
      "macrophages  9\n",
      "direct effector role pathogenesis lyme arthritis  10\n",
      "severe destructive arthritis  11\n",
      "recipients macrophages  12\n",
      "borrelia  13\n",
      "nonvaccinated hamsters  14\n",
      "formalin  15\n",
      "b burgdorferi vitro  16\n",
      "lyme spirochete  17\n",
      "hind paws  18\n",
      "h infection  19\n",
      "peaked h  20\n",
      "initial  21\n",
      "decreased day  22\n",
      "severe  23\n",
      "hind paws  24\n",
      "days infection  25\n",
      "peak  26\n",
      "day  27\n",
      "histopathologic examination  28\n",
      "macrophages  29\n",
      "inactivated spirochetes  30\n",
      "severe destructive lyme arthritis  31\n",
      "severity  32\n",
      "severe destructive arthritis  33\n",
      "dependent number macrophages  34\n",
      "contrast macrophages  35\n",
      "b burgdorferi  36\n",
      "induce  37\n",
      "severe destructive arthritis recipients  38\n",
      "challenge  39\n",
      "b burgdorferi  40\n",
      "severe destructive arthritis  41\n",
      "recipients macrophages  42\n",
      "spirochetal growth medium results  43\n",
      "macrophages  44\n",
      "hamsters infection b burgdorferi  45\n",
      "tissues  46\n",
      "findings demonstrate macrophages  47\n",
      "b burgdorferi  48\n",
      "induction lyme arthritis  49\n",
      "['Habitat' 'Microorganism' 'Microorganism' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Microorganism'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Microorganism' 'Phenotype' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Microorganism' 'Habitat']\n",
      "(47, 300) (47,)\n",
      "  text                     words         entity norm\n",
      "0   T3              macrophages         Habitat  NaN\n",
      "1   T4                 borrelia   Microorganism  NaN\n",
      "2   T5              burgdorferi   Microorganism  NaN\n",
      "3   T6  lyme arthritis hamsters         Habitat  NaN\n",
      "4   T7                mechanism         Habitat  NaN\n",
      "BB-norm+ner-8698477.txt\n",
      "(47, 300)\n",
      "(47, 300) (47, 300)\n",
      "['OntoBiotope OBT:002995' 'OntoBiotope OBT:000524'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:002995' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000648' 'OntoBiotope OBT:000799'\n",
      " 'OntoBiotope OBT:000524' 'OntoBiotope OBT:002674'\n",
      " 'OntoBiotope OBT:001623' 'NCBI_Taxonomy 1597' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:003066' 'NCBI_Taxonomy 615' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000064'\n",
      " 'OntoBiotope OBT:002711' 'OntoBiotope OBT:003066'\n",
      " 'OntoBiotope OBT:001737' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000031' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:002995' 'OntoBiotope OBT:000375'\n",
      " 'OntoBiotope OBT:000061' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000648' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:002806' 'NCBI_Taxonomy 263' 'OntoBiotope OBT:001169'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:003494' 'NCBI_Taxonomy 263'\n",
      " 'OntoBiotope OBT:000648' 'OntoBiotope OBT:000799'\n",
      " 'OntoBiotope OBT:002806' 'OntoBiotope OBT:002995'\n",
      " 'OntoBiotope OBT:002034' 'OntoBiotope OBT:000196'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 263' 'OntoBiotope OBT:002806']\n",
      "neuraminidase sialidase activity haemophilus parasuis  3\n",
      "neuraminidase sialidase  4\n",
      "potential virulence factor bacteria  5\n",
      "haemophilus parasuis  6\n",
      "invasive swine pathogen  7\n",
      "pathogens  8\n",
      "pasteurellaceae  9\n",
      "family h influenzae h somnus h paragallinarum  10\n",
      "pleuropneumoniae h  11\n",
      "parasuis neuraminidase  12\n",
      "acidic ph  13\n",
      "optimum specificity  14\n",
      "several substrates  15\n",
      "bacterial neuraminidases  16\n",
      "similar neuraminidase pasteurella multocida h  17\n",
      "parasuis neuraminidase cell  18\n",
      "require  19\n",
      "divalent cations activity  20\n",
      "exogenous sialic acid  21\n",
      "growth medium h parasuis  22\n",
      "lag h cultures  23\n",
      "greater final density cultures  24\n",
      "sialic acid  25\n",
      "exogenous sialic acid  26\n",
      "role sialidase  27\n",
      "nutrients  28\n",
      "h parasuis  29\n",
      "important factor obligate parasitism  30\n",
      "['Phenotype' 'Microorganism' 'Phenotype' 'Microorganism' 'Habitat'\n",
      " 'Phenotype' 'Microorganism' 'Microorganism' 'Microorganism' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Phenotype' 'Microorganism' 'Habitat'\n",
      " 'Habitat' 'Phenotype' 'Phenotype' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Phenotype' 'Phenotype' 'Habitat' 'Microorganism' 'Phenotype']\n",
      "(28, 300) (28,)\n",
      "  text                                              words         entity norm\n",
      "0   T3  neuraminidase sialidase activity haemophilus p...      Phenotype  NaN\n",
      "1   T4                           neuraminidase sialidase   Microorganism  NaN\n",
      "2   T5               potential virulence factor bacteria       Phenotype  NaN\n",
      "3   T6                              haemophilus parasuis   Microorganism  NaN\n",
      "4   T7                           invasive swine pathogen         Habitat  NaN\n",
      "BB-norm+ner-9231420.txt\n",
      "(28, 300)\n",
      "(28, 300) (28, 300)\n",
      "['OntoBiotope OBT:000019' 'NCBI_Taxonomy 12333' 'OntoBiotope OBT:000029'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:002806'\n",
      " 'OntoBiotope OBT:000375' 'OntoBiotope OBT:000524'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000061'\n",
      " 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 263' 'OntoBiotope OBT:000113'\n",
      " 'OntoBiotope OBT:002806' 'OntoBiotope OBT:000061' 'NCBI_Taxonomy 260680'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 1639' 'NCBI_Taxonomy 186765'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 83334'\n",
      " 'OntoBiotope OBT:000019']\n",
      "enterobacteriaceae  3\n",
      "high numbers  4\n",
      "fish  5\n",
      "meat  6\n",
      "milk cream presence toxin  7\n",
      "genes  8\n",
      "enterobacteriaceae  9\n",
      "high numbers  10\n",
      "storage  11\n",
      "degrees  12\n",
      "c  13\n",
      "consumers packs  14\n",
      "milk cream  15\n",
      "fish retail packs  16\n",
      "meat seventy  17\n",
      "fresh water fishes packs  18\n",
      "meat milk packs  19\n",
      "isolates "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " 20\n",
      "tryptone glucose  21\n",
      "extract tge agar degrees  22\n",
      "c  23\n",
      "violet  24\n",
      "red bile glucose vrbg agar degrees  25\n",
      "c  26\n",
      "meat serratia  27\n",
      "hafnia alvei rahnella  28\n",
      "fish  29\n",
      "species  30\n",
      "r  31\n",
      "aquatilis milk  32\n",
      "species  33\n",
      "h alvei r  34\n",
      "isolates  35\n",
      "citrobacter  36\n",
      "freundii  37\n",
      "food categories  38\n",
      "polymerase chain reaction pcr technique gene  39\n",
      "escherichia  40\n",
      "coli heat  41\n",
      "labile toxin lt  42\n",
      "fish isolate r aquatilis  43\n",
      "heat  44\n",
      "stable toxin genes  45\n",
      "h alvei  46\n",
      "fish  47\n",
      "meat  48\n",
      "positive pcr reaction vero cytotoxin genes  49\n",
      "h alvei strain  50\n",
      "fish vt  51\n",
      "liquefaciens strains  52\n",
      "meat vt c freundii reference  53\n",
      "strain  54\n",
      "st  55\n",
      "positive h alvei  56\n",
      "meat  57\n",
      "eaea gene  58\n",
      "phenotype  59\n",
      "enteropathogenic e coli  60\n",
      "['Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Microorganism' 'Habitat'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Habitat'\n",
      " 'Microorganism' 'Habitat' 'Microorganism' 'Microorganism' 'Habitat'\n",
      " 'Phenotype' 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Phenotype' 'Microorganism'\n",
      " 'Habitat' 'Microorganism' 'Habitat' 'Microorganism' 'Microorganism'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Microorganism']\n",
      "(58, 300) (58,)\n",
      "  text                       words         entity norm\n",
      "0   T3         enterobacteriaceae   Microorganism  NaN\n",
      "1   T4               high numbers         Habitat  NaN\n",
      "2   T5                       fish         Habitat  NaN\n",
      "3   T6                       meat         Habitat  NaN\n",
      "4   T7  milk cream presence toxin         Habitat  NaN\n",
      "BB-norm+ner-9562874.txt\n",
      "(58, 300)\n",
      "(58, 300) (58, 300)\n",
      "['NCBI_Taxonomy 543' 'OntoBiotope OBT:000019' 'OntoBiotope OBT:001549'\n",
      " 'OntoBiotope OBT:001200' 'OntoBiotope OBT:000152'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 543' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 263'\n",
      " 'OntoBiotope OBT:000902' 'OntoBiotope OBT:000152' 'NCBI_Taxonomy 263'\n",
      " 'NCBI_Taxonomy 244584' 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000097' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 263' 'NCBI_Taxonomy 260680'\n",
      " 'NCBI_Taxonomy 1313' 'NCBI_Taxonomy 263' 'OntoBiotope OBT:003269'\n",
      " 'NCBI_Taxonomy 569' 'OntoBiotope OBT:001549' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1584' 'OntoBiotope OBT:003269'\n",
      " 'NCBI_Taxonomy 569' 'OntoBiotope OBT:000097' 'NCBI_Taxonomy 1334627'\n",
      " 'OntoBiotope OBT:003269' 'NCBI_Taxonomy 1334627' 'OntoBiotope OBT:000019'\n",
      " 'NCBI_Taxonomy 90371' 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000360' 'OntoBiotope OBT:000152'\n",
      " 'OntoBiotope OBT:002840' 'NCBI_Taxonomy 569' 'OntoBiotope OBT:001549'\n",
      " 'OntoBiotope OBT:001200' 'OntoBiotope OBT:001623' 'NCBI_Taxonomy 569'\n",
      " 'NCBI_Taxonomy 12721' 'NCBI_Taxonomy 1535326' 'NCBI_Taxonomy 1496'\n",
      " 'NCBI_Taxonomy 1582' 'NCBI_Taxonomy 12333' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:001200' 'OntoBiotope OBT:000371'\n",
      " 'OntoBiotope OBT:001288' 'OntoBiotope OBT:002806']\n",
      "comparative studies activities  3\n",
      "antimicrobial agents  4\n",
      "causative organisms  5\n",
      "patients  6\n",
      "urinary tract infections susceptibility distribution  7\n",
      "frequencies isolation  8\n",
      "antimicrobial agents  9\n",
      "bacterial strains  10\n",
      "isolated patients  11\n",
      "urinary tract infections  12\n",
      "utis hospitals period june  13\n",
      "isolates  14\n",
      "gram  15\n",
      "positive bacteria  16\n",
      "majority enterococcus  17\n",
      "faecalis gram  18\n",
      "negative bacteria  19\n",
      "escherichia coli susceptabilities  20\n",
      "several isolated bacteria  21\n",
      "antimicrobial agents  22\n",
      "enterococcus faecalis  23\n",
      "ampicillin  24\n",
      "abpc  25\n",
      "highest activity e  26\n",
      "isolated patients  27\n",
      "utis mic microgram ml imipenem ipm vancomycin vcm  28\n",
      "active mic micrograms  29\n",
      "ml  30\n",
      "others "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " 31\n",
      "low activities  32\n",
      "mic micrograms  33\n",
      "ml  34\n",
      "staphylococcus  35\n",
      "mrsa arbekacin abk vcm  36\n",
      "highest activities  37\n",
      "aureus  38\n",
      "isolated patients  39\n",
      "utis mic micrograms  40\n",
      "ml  41\n",
      "others  42\n",
      "minocycline mino  43\n",
      "low activities  44\n",
      "mic micrograms  45\n",
      "ml  46\n",
      "staphylococcus epidermidis abk vcm  47\n",
      "strongest activities  48\n",
      "patients  49\n",
      "utis mics strains  50\n",
      "equal lower micrograms  51\n",
      "cefazolin cez cefotiam ctm cefozopran czop  52\n",
      "active mic micrograms  53\n",
      "ml  54\n",
      "antimicrobial activities  55\n",
      "cephems  56\n",
      "state  57\n",
      "micrograms  58\n",
      "ml micrograms  59\n",
      "ml  60\n",
      "streptococcus agalactiae drugs  61\n",
      "mino  62\n",
      "active agalactiae abpc czop ipm clarithromycin cam  63\n",
      "highest activities mics strains  64\n",
      "equal lower micromilligrams tosufloxacin tflx vcm  65\n",
      "active mic micromilligrams  66\n",
      "citrobacter  67\n",
      "freundii gentamicin gm  68\n",
      "highest activity c freundii  69\n",
      "patients  70\n",
      "utis mic micrograms  71\n",
      "ml  72\n",
      "ipm amikacin amk  73\n",
      "active mic microgram ml micrograms  74\n",
      "ml  75\n",
      "cefpirome cpr czop  76\n",
      "active mic micrograms  77\n",
      "ml  78\n",
      "mic others  79\n",
      "micrograms  80\n",
      "ml enterobacter cloacae ipm  81\n",
      "highest activity e cloacae mics  82\n",
      "equal lower microgram ml  83\n",
      "mic ciprofloxacin cpfx tflx microgram ml  84\n",
      "mic amk micrograms  85\n",
      "ml  86\n",
      "mic czop gm  87\n",
      "oflx micrograms  88\n",
      "ml  89\n",
      "mic cephems  90\n",
      "cez cefmetazole cmz cefaclor ccl  91\n",
      "state  92\n",
      "escherichia coli drugs  93\n",
      "penicillins  94\n",
      "mino  95\n",
      "active e coli  96\n",
      "cpr  97\n",
      "czop  98\n",
      "ipm  99\n",
      "highest activities  100\n",
      "e  101\n",
      "coli mic microgram ml  102\n",
      "e coli strains  103\n",
      "low susceptibilities  104\n",
      "cez cefoperazone cpz latamoxef  105\n",
      "lmox ccl  106\n",
      "klebsiella pneumoniae k pneumoniae  107\n",
      "susceptible drugs  108\n",
      "penicillins  109\n",
      "mic micrograms  110\n",
      "ml  111\n",
      "cpr strongest activity mics  112\n",
      "equal lower microgram ml flomoxef fmox cefixime cfix czop carumonam crmn  113\n",
      "active mic microgram ml pseudomonas  114\n",
      "aeruginosa  115\n",
      "drugs  116\n",
      "quinolones  117\n",
      "active p aeruginosa  118\n",
      "mic micrograms  119\n",
      "ml  120\n",
      "quinolones  121\n",
      "active mic micrograms  122\n",
      "ml  123\n",
      "micrograms  124\n",
      "ml  125\n",
      "mic microgram ml micrograms  126\n",
      "ml  127\n",
      "serratia marcescens  128\n",
      "gm  129\n",
      "highest activity marcescens  130\n",
      "mic  131\n",
      "micro  132\n",
      "['Habitat' 'Phenotype' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Phenotype'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Phenotype' 'Microorganism' 'Microorganism' 'Phenotype' 'Microorganism'\n",
      " 'Phenotype' 'Phenotype' 'Microorganism' 'Microorganism' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Microorganism'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Microorganism' 'Phenotype' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Phenotype' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Microorganism' 'Microorganism' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Phenotype' 'Habitat' 'Microorganism' 'Microorganism' 'Habitat'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism'\n",
      " 'Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Microorganism'\n",
      " 'Phenotype' 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Microorganism' 'Habitat' 'Phenotype' 'Microorganism' 'Habitat'\n",
      " 'Habitat' 'Phenotype' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat']\n",
      "(130, 300) (130,)\n",
      "  text                                              words     entity norm\n",
      "0   T3                    comparative studies activities     Habitat  NaN\n",
      "1   T4                              antimicrobial agents   Phenotype  NaN\n",
      "2   T5                               causative organisms     Habitat  NaN\n",
      "3   T6                                          patients     Habitat  NaN\n",
      "4   T7  urinary tract infections susceptibility distri...    Habitat  NaN\n",
      "BB-norm+ner-9575437.txt\n",
      "(130, 300)\n",
      "(130, 300) (130, 300)\n",
      "['NCBI_Taxonomy 1703' 'OntoBiotope OBT:002337' 'OntoBiotope OBT:002806'\n",
      " 'OntoBiotope OBT:003285' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:002337'\n",
      " 'OntoBiotope OBT:002732' 'OntoBiotope OBT:003220' 'NCBI_Taxonomy 487'\n",
      " 'OntoBiotope OBT:002714' 'OntoBiotope OBT:000097'\n",
      " 'OntoBiotope OBT:000103' 'OntoBiotope OBT:000061'\n",
      " 'OntoBiotope OBT:000113' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000113' 'NCBI_Taxonomy 83334' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:002337' 'NCBI_Taxonomy 1351' 'OntoBiotope OBT:002762'\n",
      " 'OntoBiotope OBT:000716' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:003220' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000902' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1255621'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1279' 'NCBI_Taxonomy 10912'\n",
      " 'OntoBiotope OBT:002806' 'NCBI_Taxonomy 1280' 'OntoBiotope OBT:003220'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000902' 'NCBI_Taxonomy 1752' 'OntoBiotope OBT:000019'\n",
      " 'NCBI_Taxonomy 1255621' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1255621'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003285'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 1752'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000113' 'NCBI_Taxonomy 585' 'NCBI_Taxonomy 191769'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:002806' 'NCBI_Taxonomy 1173061'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1334627'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:003285' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:002034' 'NCBI_Taxonomy 1111734'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1428' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 2093' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 727' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 2369' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 90371' 'OntoBiotope OBT:000019'\n",
      " 'NCBI_Taxonomy 1334627' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 191769'\n",
      " 'OntoBiotope OBT:000113' 'OntoBiotope OBT:001134' 'NCBI_Taxonomy 1173061'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 53345' 'OntoBiotope OBT:000304'\n",
      " 'NCBI_Taxonomy 1334627' 'OntoBiotope OBT:002806' 'NCBI_Taxonomy 1311'\n",
      " 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 1010796' 'OntoBiotope OBT:002993'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1111734' 'NCBI_Taxonomy 1313'\n",
      " 'OntoBiotope OBT:000152' 'OntoBiotope OBT:001134' 'NCBI_Taxonomy 1255621'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 287'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:002840'\n",
      " 'OntoBiotope OBT:003269' 'NCBI_Taxonomy 1255621' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:002840' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 615' 'OntoBiotope OBT:001480'\n",
      " 'NCBI_Taxonomy 727' 'NCBI_Taxonomy 1069533' 'OntoBiotope OBT:000019']\n",
      "effect cytokine modulation  7\n",
      "granulomatous response murine tuberculosis  8\n",
      "experimental murine tuberculosis  9\n",
      "evaluate effect cytokine modulation  10\n",
      "progression lung  11\n",
      "granulomatous response  12\n",
      "aerosol tuberculosis infection mice  13\n",
      "infected respiratory route  14\n",
      "viable mycobacterium tuberculosis erdman  15\n",
      "subcutaneous injections  16\n",
      "mg  17\n",
      "saline weeks  18\n",
      "bacillary  19\n",
      "granulomatous response cytokine production lungs  20\n",
      "aerosol tuberculosis infection  21\n",
      "progressive granulomatous response lungs days  22\n",
      "infection  23\n",
      "large granulomata central necrosis apoptosis  24\n",
      "infection  25\n",
      "high serum lung  26\n",
      "cytokine  27\n",
      "mrna levels  28\n",
      "thalidomide  29\n",
      "treatment  30\n",
      "significant reduction tumor necrosis factor  31\n",
      "interleukin il il protein levels blood mrna expression lungs  32\n",
      "il  33\n",
      "interferon gamma  34\n",
      "unaffected lungs  35\n",
      "mice smaller granulomata  36\n",
      "apoptotic cells "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " 37\n",
      "necrosis thalidomide treatment change  38\n",
      "bacillary load  39\n",
      "thalidomide immunomodulation reduces  40\n",
      "inflammatory cytokines  41\n",
      "concomitant lung pathology  42\n",
      "acute aerosol tuberculosis infection  43\n",
      "bacillary load  44\n",
      "['Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Phenotype' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat']\n",
      "(38, 300) (38,)\n",
      "  text                                        words   entity norm\n",
      "0   T7                  effect cytokine modulation   Habitat  NaN\n",
      "1   T8  granulomatous response murine tuberculosis   Habitat  NaN\n",
      "2   T9            experimental murine tuberculosis   Habitat  NaN\n",
      "3  T10         evaluate effect cytokine modulation   Habitat  NaN\n",
      "4  T11                            progression lung   Habitat  NaN\n",
      "BB-norm+ner-9666962.txt\n",
      "(38, 300)\n",
      "(38, 300) (38, 300)\n",
      "['OntoBiotope OBT:000019' 'OntoBiotope OBT:000332'\n",
      " 'OntoBiotope OBT:003451' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000117'\n",
      " 'OntoBiotope OBT:000152' 'NCBI_Taxonomy 12721' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000524' 'OntoBiotope OBT:000106' 'NCBI_Taxonomy 66861'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000332'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000375' 'OntoBiotope OBT:000298'\n",
      " 'OntoBiotope OBT:000375' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:002995' 'NCBI_Taxonomy 1069533' 'OntoBiotope OBT:002711'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:002806'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 83554' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 1173061' 'OntoBiotope OBT:000061'\n",
      " 'OntoBiotope OBT:003220' 'OntoBiotope OBT:000152'\n",
      " 'OntoBiotope OBT:000061' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000152']\n",
      "strains  3\n",
      "species  4\n",
      "lactic acid bacteria  5\n",
      "isolated traditional italian cheeses  6\n",
      "reduction activity enterococcus faecalis enterococcus faecium enterococcus durans  7\n",
      "streptococcus  8\n",
      "thermophilus lactococcus lactis ssp lactis  9\n",
      "paracasei ssp paracasei  10\n",
      "plantarum  11\n",
      "delbrueckii ssp bulgaricus  12\n",
      "helveticus pediococcus pentosaceus  13\n",
      "minimum redox potential lactobacilli reduction rate enterococcus spp l lactis ssp lactis  14\n",
      "higher streptococci  15\n",
      "spp p pentosaceus  16\n",
      "poor reduction activity  17\n",
      "species  18\n",
      "['Microorganism' 'Habitat' 'Phenotype' 'Habitat' 'Microorganism'\n",
      " 'Microorganism' 'Microorganism' 'Microorganism' 'Microorganism'\n",
      " 'Microorganism' 'Microorganism' 'Microorganism' 'Habitat' 'Microorganism'\n",
      " 'Habitat' 'Habitat']\n",
      "(16, 300) (16,)\n",
      "  text                                              words         entity norm\n",
      "0   T3                                           strains   Microorganism  NaN\n",
      "1   T4                                           species         Habitat  NaN\n",
      "2   T5                              lactic acid bacteria       Phenotype  NaN\n",
      "3   T6              isolated traditional italian cheeses         Habitat  NaN\n",
      "4   T7  reduction activity enterococcus faecalis enter...  Microorganism  NaN\n",
      "BB-norm+ner-F-17953562-000.txt\n",
      "(16, 300)\n",
      "(16, 300) (16, 300)\n",
      "['OntoBiotope OBT:000061' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:002874' 'NCBI_Taxonomy 1301' 'NCBI_Taxonomy 1358'\n",
      " 'NCBI_Taxonomy 1173061' 'NCBI_Taxonomy 1590' 'NCBI_Taxonomy 1585'\n",
      " 'NCBI_Taxonomy 1173061' 'NCBI_Taxonomy 1359' 'OntoBiotope OBT:003188'\n",
      " 'NCBI_Taxonomy 1752' 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003269']\n",
      "e faecalis strains  34\n",
      "relevant reduction power fact species  35\n",
      "lowest ehmin mv  36\n",
      "largest max mv  37\n",
      "fastest reach max h  38\n",
      "table behaviour strains "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " 39\n",
      "vs  40\n",
      "rapid e faecalis  41\n",
      "strain  42\n",
      "h time  43\n",
      "showing  44\n",
      "highest ehmin mv  45\n",
      "c mean curves  46\n",
      "e  47\n",
      "faecium e durans  48\n",
      "l  49\n",
      "lactis ssp lactis  50\n",
      "similar eh evolution  51\n",
      "sharp decrease  52\n",
      "second hour  53\n",
      "culture  54\n",
      "minimum values  55\n",
      "different strains  56\n",
      "l  57\n",
      "lactis ssp lactis  58\n",
      "strains  59\n",
      "species  60\n",
      "ehmin  61\n",
      "mv vs  62\n",
      "fig strains  63\n",
      "e  64\n",
      "faecium  65\n",
      "similar reduction power  66\n",
      "h incubation  67\n",
      "strains  68\n",
      "fk  69\n",
      "fk fk  70\n",
      "scimudin cheese  71\n",
      "higher  72\n",
      "activity  73\n",
      "slowest least  74\n",
      "strain  75\n",
      "caciocavallo ragusano cheese  76\n",
      "e durans  77\n",
      "similar eh evolution  78\n",
      "h  79\n",
      "vs  80\n",
      "formag la  81\n",
      "cheese  82\n",
      "vs  83\n",
      "fastest  84\n",
      "achieve  85\n",
      "ehmin max tmin h h  86\n",
      "higher reduction power ehmin mv others  87\n",
      "l  88\n",
      "lactis ssp lactis  89\n",
      "highest reduction activity e faecalis  90\n",
      "ehmin values  91\n",
      "mv  92\n",
      "rapid decrease tmin h  93\n",
      "h atypical behaviour  94\n",
      "detected strains  95\n",
      "formag  96\n",
      "vs  97\n",
      "vs  98\n",
      "lower reduction power  99\n",
      "lactococci  100\n",
      "species  101\n",
      "streptococcus  102\n",
      "thermophilus  103\n",
      "major species  104\n",
      "starter culture  105\n",
      "cheese manufacture  106\n",
      "lab species reduction activity  107\n",
      "smaller max strains  108\n",
      "ehmin  109\n",
      "end h incubation period exception  110\n",
      "isolated fontina  111\n",
      "ehmin h  112\n",
      "lod slowest thermophilus strain h  113\n",
      "highest ehmin mv  114\n",
      "lactobacilli highest  115\n",
      "species  116\n",
      "l  117\n",
      "paracasei ssp paracasei strains  118\n",
      "lowest ehmin mv  119\n",
      "fastest  120\n",
      "achieve  121\n",
      "max h b b strain  122\n",
      "similar behaviour enterococci ehmin h  123\n",
      "table contrary bb strain  124\n",
      "formai  125\n",
      "cheese slow  126\n",
      "h reach ehmin redox curve l paracasei ssp paracasei  127\n",
      "comparable e faecalis  128\n",
      "similar final values  129\n",
      "gradual slope  130\n",
      "max  131\n",
      "longer times  132\n",
      "regard  133\n",
      "strains  134\n",
      "l  135\n",
      "plantarum  136\n",
      "redox potential course  137\n",
      "initial stabilizing  138\n",
      "period  139\n",
      "rapid consistent decrease  140\n",
      "major max mv  141\n",
      "lactobacilli species  142\n",
      "good reduction power  143\n",
      "max l paracasei ssp paracasei  144\n",
      "lowest  145\n",
      "lactobacilli  146\n",
      "l delbrueckii ssp bulgaricus l helveticus  147\n",
      "ehmin  148\n",
      "mv mv  149\n",
      "thermophilic lactobacilli species  150\n",
      "eh  151\n",
      "l delbrueckii ssp bulgaricus h l helveticus h  152\n",
      "mean curves  153\n",
      "similar evolution h period  154\n",
      "p pentosaceus  155\n",
      "poor reduction activity fact  156\n",
      "highest ehmin mv  157\n",
      "smallest max mv  158\n",
      "slowest achieve max  159\n",
      "h  160\n",
      "isolates  161\n",
      "investigation  162\n",
      "cheeses  163\n",
      "mountains  164\n",
      "interesting note majority  165\n",
      "scimudin  166\n",
      "cheese fastest  167\n",
      "achieve  168\n",
      "ehmin  169\n",
      "strains  170\n",
      "formai  171\n",
      "time reach minimum  172\n",
      "eh value  173\n",
      "formag  174\n",
      "strains  175\n",
      "l  176\n",
      "lactis ssp lactis e durans  177\n",
      "strains  178\n",
      "vs  179\n",
      "lower reduction power  180\n",
      "strains species  181\n",
      "less time  182\n",
      "['Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Microorganism' 'Microorganism' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Microorganism' 'Microorganism' 'Microorganism' 'Microorganism'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Microorganism' 'Microorganism' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Microorganism' 'Microorganism' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Microorganism'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Microorganism' 'Microorganism' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Habitat' 'Microorganism' 'Microorganism' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Microorganism' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism'\n",
      " 'Microorganism' 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Microorganism' 'Habitat' 'Microorganism' 'Habitat'\n",
      " 'Microorganism' 'Microorganism' 'Habitat' 'Habitat' 'Microorganism'\n",
      " 'Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Microorganism' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism'\n",
      " 'Microorganism' 'Microorganism' 'Microorganism' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Habitat']\n",
      "(149, 300) (149,)\n",
      "  text                                   words         entity norm\n",
      "0  T34                     e faecalis strains   Microorganism  NaN\n",
      "1  T35  relevant reduction power fact species         Habitat  NaN\n",
      "2  T36                        lowest ehmin mv         Habitat  NaN\n",
      "3  T37                         largest max mv         Habitat  NaN\n",
      "4  T38                    fastest reach max h         Habitat  NaN\n",
      "BB-norm+ner-F-17953562-001.txt\n",
      "(149, 300)\n",
      "(149, 300) (149, 300)\n",
      "['OntoBiotope OBT:002806' 'NCBI_Taxonomy 781' 'NCBI_Taxonomy 1047'\n",
      " 'OntoBiotope OBT:002201' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 28875'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000152' 'NCBI_Taxonomy 1582'\n",
      " 'OntoBiotope OBT:000038' 'OntoBiotope OBT:000113' 'NCBI_Taxonomy 1182512'\n",
      " 'OntoBiotope OBT:002674' 'NCBI_Taxonomy 1311' 'OntoBiotope OBT:001341'\n",
      " 'NCBI_Taxonomy 29412' 'NCBI_Taxonomy 1360' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003188'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 29412' 'NCBI_Taxonomy 1360'\n",
      " 'OntoBiotope OBT:000061' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:002823'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1311' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000061' 'OntoBiotope OBT:003522'\n",
      " 'OntoBiotope OBT:003522' 'OntoBiotope OBT:003522' 'NCBI_Taxonomy 1047'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1582'\n",
      " 'OntoBiotope OBT:003510' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:002712'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1359' 'OntoBiotope OBT:001480'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1496' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 29412'\n",
      " 'NCBI_Taxonomy 1360' 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'NCBI_Taxonomy 1047' 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:003459'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:003220' 'NCBI_Taxonomy 1357' 'OntoBiotope OBT:003269'\n",
      " 'NCBI_Taxonomy 1301' 'NCBI_Taxonomy 1308' 'OntoBiotope OBT:000062'\n",
      " 'OntoBiotope OBT:001030' 'OntoBiotope OBT:001480'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1582' 'OntoBiotope OBT:000648'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1182512'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003269' 'NCBI_Taxonomy 29412'\n",
      " 'NCBI_Taxonomy 160386' 'NCBI_Taxonomy 1047' 'NCBI_Taxonomy 1496'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000029' 'OntoBiotope OBT:000113' 'NCBI_Taxonomy 1173061'\n",
      " 'OntoBiotope OBT:001480' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 119856'\n",
      " 'OntoBiotope OBT:003269' 'NCBI_Taxonomy 1047' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:001480' 'NCBI_Taxonomy 294' 'OntoBiotope OBT:000061'\n",
      " 'NCBI_Taxonomy 29412' 'NCBI_Taxonomy 1590' 'NCBI_Taxonomy 1582'\n",
      " 'OntoBiotope OBT:000113' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:002712' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:002714' 'NCBI_Taxonomy 1334627'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1047'\n",
      " 'NCBI_Taxonomy 1578' 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000007'\n",
      " 'NCBI_Taxonomy 28985' 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:001231' 'NCBI_Taxonomy 1182512' 'OntoBiotope OBT:000598'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:002712'\n",
      " 'OntoBiotope OBT:000097' 'OntoBiotope OBT:003220'\n",
      " 'OntoBiotope OBT:001480' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003522'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000061' 'NCBI_Taxonomy 1173061'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:002806'\n",
      " 'OntoBiotope OBT:003459' 'OntoBiotope OBT:000061' 'NCBI_Taxonomy 29412'\n",
      " 'NCBI_Taxonomy 456327' 'OntoBiotope OBT:000061' 'OntoBiotope OBT:000019'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " 'OntoBiotope OBT:003220' 'NCBI_Taxonomy 1359' 'OntoBiotope OBT:000019']\n",
      "second component  6\n",
      "heavy  7\n",
      "distinct groups strains  8\n",
      "enterococcus spp l lactis ssp lactis l paracasei ssp paracasei l plantarum  9\n",
      "thermophilic lactobacilli l delbrueckii ssp bulgaricus l helveticus  10\n",
      "noted important differences  11\n",
      "strains species  12\n",
      "fig e faecalis  13\n",
      "e  14\n",
      "faecium e durans  15\n",
      "l  16\n",
      "lactis ssp lactis  17\n",
      "quadrant l paracasei ssp paracasei l plantarum  18\n",
      "quadrant fact groups  19\n",
      "high  20\n",
      "reduction activity species  21\n",
      "tend  22\n",
      "fall  23\n",
      "side  24\n",
      "plot  25\n",
      "variable  26\n",
      "max  27\n",
      "['Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Microorganism' 'Habitat'\n",
      " 'Microorganism' 'Microorganism' 'Microorganism' 'Microorganism'\n",
      " 'Microorganism' 'Microorganism' 'Microorganism' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat']\n",
      "(22, 300) (22,)\n",
      "  text                                              words         entity norm\n",
      "0   T6                                  second component         Habitat  NaN\n",
      "1   T7                                             heavy         Habitat  NaN\n",
      "2   T8                           distinct groups strains         Habitat  NaN\n",
      "3   T9  enterococcus spp l lactis ssp lactis l paracas...  Microorganism  NaN\n",
      "4  T10  thermophilic lactobacilli l delbrueckii ssp bu...  Microorganism  NaN\n",
      "BB-norm+ner-F-17953562-002.txt\n",
      "(22, 300)\n",
      "(22, 300) (22, 300)\n",
      "['OntoBiotope OBT:002995' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 29340' 'NCBI_Taxonomy 1359'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1311' 'OntoBiotope OBT:001341'\n",
      " 'NCBI_Taxonomy 29412' 'NCBI_Taxonomy 1360' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000029'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:001386' 'NCBI_Taxonomy 260680'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000007']\n",
      "considerable  6\n",
      "variation  7\n",
      "different species  8\n",
      "ability change  9\n",
      "redox  10\n",
      "potential species  11\n",
      "paper  12\n",
      "present raw milk cheeses  13\n",
      "enterococcus  14\n",
      "spp pediococcus spp l lactis ssp lactis  15\n",
      "make  16\n",
      "starter cultures  17\n",
      "thermophilus  18\n",
      "l delbrueckii ssp bulgaricus l helveticus species  19\n",
      "probiotic bactera l paracasei subsp paracasei l plantarum  20\n",
      "['Habitat' 'Habitat' 'Habitat' 'Habitat' 'Phenotype' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Microorganism' 'Microorganism' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Microorganism' 'Microorganism']\n",
      "(15, 300) (15,)\n",
      "  text               words     entity norm\n",
      "0   T6       considerable     Habitat  NaN\n",
      "1   T7          variation     Habitat  NaN\n",
      "2   T8  different species     Habitat  NaN\n",
      "3   T9     ability change     Habitat  NaN\n",
      "4  T10              redox   Phenotype  NaN\n",
      "BB-norm+ner-F-17953562-003.txt\n",
      "(15, 300)\n",
      "(15, 300) (15, 300)\n",
      "['OntoBiotope OBT:000019' 'OntoBiotope OBT:001288'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:003220' 'NCBI_Taxonomy 1051'\n",
      " 'NCBI_Taxonomy 43987' 'OntoBiotope OBT:000019' 'OntoBiotope OBT:001480'\n",
      " 'NCBI_Taxonomy 1350' 'NCBI_Taxonomy 1360' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:001030' 'NCBI_Taxonomy 1308' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000007']\n",
      "cocci  4\n",
      "species  5\n",
      "e "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " 6\n",
      "faecalis  7\n",
      "rods  8\n",
      "l  9\n",
      "paracasei ssp paracasei nonpathogen enterococci l lactis ssp lactis  10\n",
      "reach  11\n",
      "eh values  12\n",
      "short time  13\n",
      "h thermophilus l delbrueckii ssp bulgaricus species  14\n",
      "starter cultures  15\n",
      "lower  16\n",
      "power  17\n",
      "probiotics  18\n",
      "ssp paracase l plantarum  19\n",
      "good  20\n",
      "power  21\n",
      "['Phenotype' 'Habitat' 'Microorganism' 'Microorganism' 'Habitat'\n",
      " 'Microorganism' 'Microorganism' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism'\n",
      " 'Habitat' 'Habitat']\n",
      "(18, 300) (18,)\n",
      "  text      words         entity norm\n",
      "0   T4     cocci       Phenotype  NaN\n",
      "1   T5   species         Habitat  NaN\n",
      "2   T6         e   Microorganism  NaN\n",
      "3   T7  faecalis   Microorganism  NaN\n",
      "4   T8      rods         Habitat  NaN\n",
      "BB-norm+ner-F-17953562-004.txt\n",
      "(18, 300)\n",
      "(18, 300) (18, 300)\n",
      "['OntoBiotope OBT:000850' 'OntoBiotope OBT:003269' 'NCBI_Taxonomy 1311'\n",
      " 'NCBI_Taxonomy 511' 'OntoBiotope OBT:003264' 'NCBI_Taxonomy 29412'\n",
      " 'NCBI_Taxonomy 1173061' 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:001030' 'OntoBiotope OBT:000936'\n",
      " 'OntoBiotope OBT:000598' 'OntoBiotope OBT:001269'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000598']\n",
      "carnobacterium maltaromaticum  3\n",
      "atypical soft cheese presence cheese  4\n",
      "decrease concentration psychrobacter  5\n",
      "cellular concentration c maltaromaticum  6\n",
      "main factor  7\n",
      "inhibition  8\n",
      "malty chocolate  9\n",
      "aroma  10\n",
      "due methylbutanal milk species  11\n",
      "major role cheese  12\n",
      "flora  13\n",
      "['Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Phenotype'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat']\n",
      "(11, 300) (11,)\n",
      "  text                                     words         entity norm\n",
      "0   T3            carnobacterium maltaromaticum   Microorganism  NaN\n",
      "1   T4     atypical soft cheese presence cheese         Habitat  NaN\n",
      "2   T5     decrease concentration psychrobacter         Habitat  NaN\n",
      "3   T6  cellular concentration c maltaromaticum         Habitat  NaN\n",
      "4   T7                              main factor         Habitat  NaN\n",
      "BB-norm+ner-F-24010601-000.txt\n",
      "(11, 300)\n",
      "(11, 300) (11, 300)\n",
      "['OntoBiotope OBT:000061' 'OntoBiotope OBT:001480'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000152' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:001480'\n",
      " 'OntoBiotope OBT:000113']\n",
      "carnobacterium maltaromaticum  12\n",
      "lactic acid bacterium  13\n",
      "soft cheese  14\n",
      "objective  15\n",
      "work study  16\n",
      "potential positive impact  17\n",
      "cheese  18\n",
      "technology phenotypic genotypic characterization  19\n",
      "strains  20\n",
      "c  21\n",
      "maltaromaticum  22\n",
      "different  23\n",
      "phylogenetic groups  24\n",
      "strains  25\n",
      "ability  26\n",
      "coagulate milk  27\n",
      "acidotolerant affect  28\n",
      "coagulation capacity  29\n",
      "lactic acid bacteria  30\n",
      "lactococcus  31\n",
      "lactis streptococcus thermophilus  32\n",
      "dairy industry impact  33\n",
      "c "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " 34\n",
      "maltaromaticum  35\n",
      "lma bacterial flora  36\n",
      "cheese  37\n",
      "significant decrease psychrobacter  38\n",
      "sp concentration  39\n",
      "responsible cheese  40\n",
      "phenomena  41\n",
      "['Microorganism' 'Phenotype' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Microorganism' 'Microorganism' 'Microorganism'\n",
      " 'Habitat' 'Habitat' 'Microorganism' 'Phenotype' 'Habitat' 'Phenotype'\n",
      " 'Habitat' 'Phenotype' 'Microorganism' 'Microorganism' 'Habitat'\n",
      " 'Microorganism' 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat']\n",
      "(30, 300) (30,)\n",
      "  text                           words         entity norm\n",
      "0  T12  carnobacterium maltaromaticum   Microorganism  NaN\n",
      "1  T13          lactic acid bacterium       Phenotype  NaN\n",
      "2  T14                    soft cheese         Habitat  NaN\n",
      "3  T15                      objective         Habitat  NaN\n",
      "4  T16                     work study         Habitat  NaN\n",
      "BB-norm+ner-F-24010601-001.txt\n",
      "(30, 300)\n",
      "(30, 300) (30, 300)\n",
      "['OntoBiotope OBT:000061' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:001480' 'OntoBiotope OBT:000648'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000029'\n",
      " 'OntoBiotope OBT:001480' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000061' 'NCBI_Taxonomy 263' 'NCBI_Taxonomy 244584'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003245'\n",
      " 'OntoBiotope OBT:000061' 'OntoBiotope OBT:000360'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1255670' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1357' 'NCBI_Taxonomy 1358'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 263' 'NCBI_Taxonomy 244584'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:001480'\n",
      " 'OntoBiotope OBT:000113' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:001480' 'OntoBiotope OBT:000019']\n",
      "cellular concentration c maltaromaticum lma  3\n",
      "main factor  4\n",
      "inhibition psychrobacter sp l monocytogenes  5\n",
      "['Habitat' 'Habitat' 'Microorganism']\n",
      "(3, 300) (3,)\n",
      "  text                                         words         entity norm\n",
      "0   T3  cellular concentration c maltaromaticum lma         Habitat  NaN\n",
      "1   T4                                  main factor         Habitat  NaN\n",
      "2   T5  inhibition psychrobacter sp l monocytogenes   Microorganism  NaN\n",
      "BB-norm+ner-F-24010601-002.txt\n",
      "(3, 300)\n",
      "(3, 300) (3, 300)\n",
      "['OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007']\n",
      "presence  7\n",
      "opportunistic psychrotrophic lab  8\n",
      "carnobacterium maltaromaticum  9\n",
      "numerous french cheeses  10\n",
      "appellation  11\n",
      "origine  12\n",
      "prot g e  13\n",
      "designation origin pdo  14\n",
      "end  15\n",
      "cold storage  16\n",
      "final product  17\n",
      "quality cailliez  18\n",
      "grimal et al milli et al  19\n",
      "['Habitat' 'Phenotype' 'Microorganism' 'Habitat' 'Habitat' 'Microorganism'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Microorganism']\n",
      "(13, 300) (13,)\n",
      "  text                              words         entity norm\n",
      "0   T7                          presence         Habitat  NaN\n",
      "1   T8  opportunistic psychrotrophic lab       Phenotype  NaN\n",
      "2   T9     carnobacterium maltaromaticum   Microorganism  NaN\n",
      "3  T10           numerous french cheeses         Habitat  NaN\n",
      "4  T11                       appellation         Habitat  NaN\n",
      "BB-norm+ner-F-24010601-003.txt\n",
      "(13, 300)\n",
      "(13, 300) (13, 300)\n",
      "['OntoBiotope OBT:003269' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000061' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:002480'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000232']\n",
      "many carnobacterium strains  3\n",
      "protective cultures  4\n",
      "pathogenic bacteria instance listeria monocytogenes  5\n",
      "fish meat products  6\n",
      "brillet  7\n",
      "al duffes et al matamoros  8\n",
      "et  9\n",
      "al vescovo et al bacteriocin  10\n",
      "ability carnobacterium  11\n",
      "key role  12\n",
      "spoilage  13\n",
      "pathogenic bacteria food  14\n",
      "vitro model systems  15\n",
      "dos  16\n",
      "reis  17\n",
      "al martin visscher et al bernardi  18\n",
      "al external factors  19\n",
      "ph salt storage temperatures  20\n",
      "significant impact hwang leroi et al  21\n",
      "['Microorganism' 'Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Microorganism' 'Microorganism' 'Habitat' 'Habitat'\n",
      " 'Phenotype' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat']\n",
      "(19, 300) (19,)\n",
      "  text                                              words         entity norm\n",
      "0   T3                       many carnobacterium strains   Microorganism  NaN\n",
      "1   T4                               protective cultures         Habitat  NaN\n",
      "2   T5  pathogenic bacteria instance listeria monocyto...  Microorganism  NaN\n",
      "3   T6                                fish meat products         Habitat  NaN\n",
      "4   T7                                           brillet         Habitat  NaN"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "BB-norm+ner-F-24010601-004.txt\n",
      "(19, 300)\n",
      "(19, 300) (19, 300)\n",
      "['OntoBiotope OBT:000304' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000061' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000583' 'NCBI_Taxonomy 260680' 'OntoBiotope OBT:000007'\n",
      " 'NCBI_Taxonomy 234834' 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 29340'\n",
      " 'NCBI_Taxonomy 10508' 'OntoBiotope OBT:002732' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:001480' 'NCBI_Taxonomy 1255670' 'NCBI_Taxonomy 1334627'\n",
      " 'NCBI_Taxonomy 260680' 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 260680']\n",
      "differentiation strains  6\n",
      "c  7\n",
      "maltaromaticum  8\n",
      "isolated soft cheeses  9\n",
      "biochemical  10\n",
      "genotypic tests  11\n",
      "comparison  12\n",
      "biochemical characteristics  13\n",
      "weak differences fermentation pattern data  14\n",
      "['Habitat' 'Microorganism' 'Microorganism' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Habitat' 'Phenotype' 'Habitat']\n",
      "(9, 300) (9,)\n",
      "  text                     words         entity norm\n",
      "0   T6  differentiation strains         Habitat  NaN\n",
      "1   T7                        c   Microorganism  NaN\n",
      "2   T8           maltaromaticum   Microorganism  NaN\n",
      "3   T9    isolated soft cheeses         Habitat  NaN\n",
      "4  T10              biochemical         Habitat  NaN\n",
      "BB-norm+ner-F-24010601-005.txt\n",
      "(9, 300)\n",
      "(9, 300) (9, 300)\n",
      "['NCBI_Taxonomy 263' 'NCBI_Taxonomy 263' 'NCBI_Taxonomy 244584'\n",
      " 'OntoBiotope OBT:001480' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000011' 'OntoBiotope OBT:002806'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019']\n",
      "Warning: parsing empty text\n",
      "conditions  6\n",
      "strains  7\n",
      "able produce  8\n",
      "methylbutanal milk  9\n",
      "c c  10\n",
      "h incubation table milk bottles  11\n",
      "p carnobacterium species  12\n",
      "h  13\n",
      "taste panel  14\n",
      "milk malty flavor bottles  15\n",
      "c maltaromaticum  16\n",
      "lactic aromatic flavor lma strains  17\n",
      "exhibit  18\n",
      "particular flavors  19\n",
      "strain  20\n",
      "c maltaromaticum lma  21\n",
      "cheese assay  22\n",
      "['Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Microorganism' 'Habitat'\n",
      " 'Microorganism' 'Microorganism' 'Habitat' 'Habitat' 'Microorganism'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Microorganism' 'Habitat']\n",
      "(17, 300) (17,)\n",
      "  text                words         entity norm\n",
      "0   T6          conditions         Habitat  NaN\n",
      "1   T7             strains   Microorganism  NaN\n",
      "2   T8        able produce         Habitat  NaN\n",
      "3   T9  methylbutanal milk         Habitat  NaN\n",
      "4  T10                 c c   Microorganism  NaN\n",
      "BB-norm+ner-F-24010601-006.txt\n",
      "(17, 300)\n",
      "(17, 300) (17, 300)\n",
      "['NCBI_Taxonomy 1255580' 'OntoBiotope OBT:000061' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 263' 'OntoBiotope OBT:001480'\n",
      " 'NCBI_Taxonomy 228' 'OntoBiotope OBT:002712' 'NCBI_Taxonomy 781'\n",
      " 'OntoBiotope OBT:000152' 'OntoBiotope OBT:000113'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:002806' 'NCBI_Taxonomy 1582' 'NCBI_Taxonomy 1720'\n",
      " 'OntoBiotope OBT:000019']\n",
      "days  3\n",
      "cheeses  4\n",
      "p reference  5\n",
      "c maltaromaticum  6\n",
      "lma taste panel  7\n",
      "malty butyric flavor cheeses  8\n",
      "c maltaromaticum lma  9\n",
      "['Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Habitat' 'Habitat'\n",
      " 'Microorganism']\n",
      "(7, 300) (7,)\n",
      "  text              words         entity norm\n",
      "0   T3              days         Habitat  NaN\n",
      "1   T4           cheeses         Habitat  NaN\n",
      "2   T5       p reference         Habitat  NaN\n",
      "3   T6  c maltaromaticum   Microorganism  NaN\n",
      "4   T7   lma taste panel         Habitat  NaN\n",
      "BB-norm+ner-F-24010601-007.txt\n",
      "(7, 300)\n",
      "(7, 300) (7, 300)\n",
      "['NCBI_Taxonomy 1111734' 'OntoBiotope OBT:001480' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000113' 'NCBI_Taxonomy 114277' 'NCBI_Taxonomy 28038'\n",
      " 'NCBI_Taxonomy 1720']\n",
      "increase  4\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "cellular concentration c maltaromaticum lma  5\n",
      "inhibition psychrobacter sp lma l monocytogenes  6\n",
      "cip  7\n",
      "milk  8\n",
      "['Habitat' 'Habitat' 'Microorganism' 'Microorganism' 'Habitat']\n",
      "(5, 300) (5,)\n",
      "  text                                             words         entity norm\n",
      "0   T4                                         increase         Habitat  NaN\n",
      "1   T5      cellular concentration c maltaromaticum lma         Habitat  NaN\n",
      "2   T6  inhibition psychrobacter sp lma l monocytogenes   Microorganism  NaN\n",
      "3   T7                                              cip   Microorganism  NaN\n",
      "4   T8                                             milk         Habitat  NaN\n",
      "BB-norm+ner-F-24010601-008.txt\n",
      "(5, 300)\n",
      "(5, 300) (5, 300)\n",
      "['OntoBiotope OBT:001481' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1357'\n",
      " 'NCBI_Taxonomy 1752' 'OntoBiotope OBT:001645']\n",
      "strains  6\n",
      "c  7\n",
      "least  8\n",
      "gene  9\n",
      "bacteriocin none target bacteria  10\n",
      "inhibited c maltaromaticum  11\n",
      "lma lma  12\n",
      "gene  13\n",
      "culture conditions  14\n",
      "pfge analysis  15\n",
      "strains  16\n",
      "different  17\n",
      "criteria  18\n",
      "et  19\n",
      "al  20\n",
      "diversity species  21\n",
      "biotope  22\n",
      "cheese industry  23\n",
      "species  24\n",
      "major role milk transformation  25\n",
      "cheese e l lactis  26\n",
      "coagulate milk  27\n",
      "['Microorganism' 'Microorganism' 'Habitat' 'Habitat' 'Phenotype'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism'\n",
      " 'Habitat' 'Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat']\n",
      "(22, 300) (22,)\n",
      "  text                              words         entity norm\n",
      "0   T6                           strains   Microorganism  NaN\n",
      "1   T7                                 c   Microorganism  NaN\n",
      "2   T8                             least         Habitat  NaN\n",
      "3   T9                              gene         Habitat  NaN\n",
      "4  T10  bacteriocin none target bacteria       Phenotype  NaN\n",
      "BB-norm+ner-F-24010601-009.txt\n",
      "(22, 300)\n",
      "(22, 300) (22, 300)\n",
      "['OntoBiotope OBT:000061' 'NCBI_Taxonomy 263' 'OntoBiotope OBT:003334'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 1269' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000061' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:002806' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:003146' 'OntoBiotope OBT:000113' 'NCBI_Taxonomy 1590'\n",
      " 'OntoBiotope OBT:001480' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000019']\n",
      "different species psychrobacter  12\n",
      "various ecosystems  13\n",
      "cold hot  14\n",
      "frozen sea fish  15\n",
      "meat  16\n",
      "clinical  17\n",
      "samples  18\n",
      "bowman  19\n",
      "case  20\n",
      "cheese bacterium  21\n",
      "milk  22\n",
      "environmental contamination belongs order pseudomonadales  23\n",
      "food microbiology spoilage  24\n",
      "dairy products  25\n",
      "exhibit  26\n",
      "high proteolytic activities  27\n",
      "role  28\n",
      "process  29\n",
      "cheese  30\n",
      "process c maltaromaticum  31\n",
      "flora  32\n",
      "inhibition  33\n",
      "target strains  34\n",
      "psychrobacter  35\n",
      "sp lma l monocytogenes  36\n",
      "cip  37\n",
      "milk  38\n",
      "cellular concentration c maltaromaticum  39\n",
      "factor  40\n",
      "high inhibitory growth response psychrobacter sp l monocytogenes  41\n",
      "milk  42\n",
      "combination variables  43\n",
      "impact  44\n",
      "exact mechanism inhibition  45\n",
      "unknown deserves  46\n",
      "investigations  47\n",
      "['Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Phenotype' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Habitat' 'Phenotype' 'Habitat' 'Microorganism'\n",
      " 'Microorganism' 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Phenotype'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat']\n",
      "(36, 300) (36,)\n",
      "  text                             words         entity norm\n",
      "0  T12  different species psychrobacter   Microorganism  NaN\n",
      "1  T13               various ecosystems         Habitat  NaN\n",
      "2  T14                         cold hot         Habitat  NaN\n",
      "3  T15                  frozen sea fish         Habitat  NaN\n",
      "4  T16                             meat         Habitat  NaN\n",
      "BB-norm+ner-F-24010601-010.txt\n",
      "(36, 300)\n",
      "(36, 300) (36, 300)\n",
      "['OntoBiotope OBT:000113' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:002982' 'OntoBiotope OBT:000029'\n",
      " 'OntoBiotope OBT:001200' 'OntoBiotope OBT:003220' 'NCBI_Taxonomy 1400'\n",
      " 'OntoBiotope OBT:000491' 'OntoBiotope OBT:000113'\n",
      " 'OntoBiotope OBT:000113' 'OntoBiotope OBT:001645'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1111734' 'OntoBiotope OBT:001208'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000038'\n",
      " 'OntoBiotope OBT:001480' 'NCBI_Taxonomy 1288' 'OntoBiotope OBT:000113'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 497'\n",
      " 'OntoBiotope OBT:001623' 'NCBI_Taxonomy 1752' 'OntoBiotope OBT:001645'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:001645'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000006'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019']\n",
      "use strain c maltaromaticum  17\n",
      "lma cheese technology impact microflora  18\n",
      "soft cheese  19\n",
      "study presence  20\n",
      "decrease concentration psychrobacter sp  21\n",
      "responsible  22\n",
      "phenomena soft cheese  23\n",
      "cellular concentration c maltaromaticum lma  24\n",
      "main factor  25\n",
      "inhibition psychrobacter sp l monocytogenes  26\n",
      "cip experimental conditions  27\n",
      "exact  28\n",
      "mechanism  29\n",
      "inhibition  30\n",
      "unknown deserves investigations  31\n",
      "addition c  32\n",
      "maltaromaticum  33\n",
      "lma  34\n",
      "malty chocolate  35\n",
      "aroma  36\n",
      "due methylbutanal catabolism leucine afzal et al  37\n",
      "essential study impact proteolysis  38\n",
      "lipolysis cheese  39\n",
      "psychrotrophic  40\n",
      "alkalinophilic antibacterial activity ability increase  41\n",
      "flavor species  42\n",
      "major role cheese  43\n",
      "flora  44\n",
      "negative interference starters  45\n",
      "instance  46\n",
      "l lactis  47\n",
      "['Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Phenotype' 'Habitat' 'Habitat' 'Microorganism' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Phenotype' 'Phenotype'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism']\n",
      "(31, 300) (31,)\n",
      "  text                                     words         entity norm\n",
      "0  T17              use strain c maltaromaticum         Habitat  NaN\n",
      "1  T18  lma cheese technology impact microflora         Habitat  NaN\n",
      "2  T19                              soft cheese         Habitat  NaN\n",
      "3  T20                           study presence         Habitat  NaN\n",
      "4  T21  decrease concentration psychrobacter sp   Microorganism  NaN\n",
      "BB-norm+ner-F-24010601-011.txt\n",
      "(31, 300)\n",
      "(31, 300) (31, 300)\n",
      "['NCBI_Taxonomy 12721' 'OntoBiotope OBT:000019' 'OntoBiotope OBT:001480'\n",
      " 'OntoBiotope OBT:000029' 'OntoBiotope OBT:001444'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003494'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 29412'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 244584'\n",
      " 'NCBI_Taxonomy 1269' 'OntoBiotope OBT:000152' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:001480' 'OntoBiotope OBT:000402'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:001480' 'OntoBiotope OBT:000113'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000113' 'NCBI_Taxonomy 1246']\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "total microbiota raw milk  15\n",
      "cheese rind  16\n",
      "composed species  17\n",
      "present types cheese  18\n",
      "corynebacterium casei psychrobacter spp lactococcus lactis ssp cremoris staphylococcus equorum vagococcus salmoninarum species  19\n",
      "present levels brevibacterium linens  20\n",
      "present low levels  21\n",
      "rind  22\n",
      "raw  23\n",
      "milk cheeses  24\n",
      "bacterium  25\n",
      "manufacturing process  26\n",
      "psychroflexus casei  27\n",
      "red smear raclette type cheese  28\n",
      "small proportions  29\n",
      "composition  30\n",
      "raw  31\n",
      "milk  32\n",
      "cheeses  33\n",
      "heart cheeses  34\n",
      "common species bacteria  35\n",
      "main species  36\n",
      "lactococcus lactis ssp cremoris psychrobacter spp staphylococcus equorum ssp equorum  37\n",
      "['Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Habitat'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism']\n",
      "(23, 300) (23,)\n",
      "  text                                              words         entity norm\n",
      "0  T15                         total microbiota raw milk         Habitat  NaN\n",
      "1  T16                                       cheese rind         Habitat  NaN\n",
      "2  T17                                  composed species         Habitat  NaN\n",
      "3  T18                              present types cheese         Habitat  NaN\n",
      "4  T19  corynebacterium casei psychrobacter spp lactoc...  Microorganism  NaN\n",
      "BB-norm+ner-F-25064656-000.txt\n",
      "(23, 300)\n",
      "(23, 300) (23, 300)\n",
      "['OntoBiotope OBT:000113' 'OntoBiotope OBT:001481'\n",
      " 'OntoBiotope OBT:002337' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:003220' 'OntoBiotope OBT:001481'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:001480'\n",
      " 'OntoBiotope OBT:001471' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 160386'\n",
      " 'OntoBiotope OBT:003494' 'NCBI_Taxonomy 1766' 'NCBI_Taxonomy 1051'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:001645'\n",
      " 'OntoBiotope OBT:001480' 'OntoBiotope OBT:003486'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 272630' 'NCBI_Taxonomy 47715']\n",
      "corynebacterium casei enterococcus  4\n",
      "faecalis prevalent raw milk cheeses  5\n",
      "whereas  6\n",
      "psychrobacter celer  7\n",
      "present  8\n",
      "milk cheeses  9\n",
      "['Microorganism' 'Habitat' 'Habitat' 'Microorganism' 'Habitat' 'Habitat']\n",
      "(6, 300) (6,)\n",
      "  text                                 words         entity norm\n",
      "0   T4   corynebacterium casei enterococcus   Microorganism  NaN\n",
      "1   T5  faecalis prevalent raw milk cheeses         Habitat  NaN\n",
      "2   T6                              whereas         Habitat  NaN\n",
      "3   T7                  psychrobacter celer   Microorganism  NaN\n",
      "4   T8                              present         Habitat  NaN\n",
      "BB-norm+ner-F-25064656-001.txt\n",
      "(6, 300)\n",
      "(6, 300) (6, 300)\n",
      "['NCBI_Taxonomy 1313' 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003269'\n",
      " 'NCBI_Taxonomy 1255614' 'NCBI_Taxonomy 12721' 'OntoBiotope OBT:001480']\n",
      "herve  7\n",
      "cheese soft cheese  8\n",
      "rind  9\n",
      "raw pasteurized milk texture firm  10\n",
      "smooth creamy minimum  11\n",
      "fat content  12\n",
      "characteristic flavor orange  13\n",
      "rind  14\n",
      "presence  15\n",
      "least bacterium brevibacterium linens  16\n",
      "['Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Microorganism']\n",
      "(10, 300) (10,)\n",
      "  text                               words   entity norm\n",
      "0   T7                              herve   Habitat  NaN\n",
      "1   T8                 cheese soft cheese   Habitat  NaN\n",
      "2   T9                               rind   Habitat  NaN\n",
      "3  T10  raw pasteurized milk texture firm   Habitat  NaN\n",
      "4  T11              smooth creamy minimum   Habitat  NaN\n",
      "BB-norm+ner-F-25064656-002.txt\n",
      "(10, 300)\n",
      "(10, 300) (10, 300)\n",
      "['OntoBiotope OBT:003494' 'OntoBiotope OBT:001480'\n",
      " 'OntoBiotope OBT:001481' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:001341' 'NCBI_Taxonomy 12721' 'NCBI_Taxonomy 28875'\n",
      " 'OntoBiotope OBT:001481' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000007']\n",
      "cheese  9\n",
      "raw milk  10\n",
      "safety reasons "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " 11\n",
      "due potential presence listeria monocytogenes  12\n",
      "herve  13\n",
      "cheese  14\n",
      "pasteurized milk  15\n",
      "dominant microbiota present  16\n",
      "red smear cheeses maroilles  17\n",
      "muenster limburger pont l v que  18\n",
      "gram  19\n",
      "positive bacteria  20\n",
      "micrococcus  21\n",
      "various coryneform species  22\n",
      "corynebacterium  23\n",
      "spp arthrobacter spp rhodococcus spp b  24\n",
      "brennan  25\n",
      "al feurer et al b  26\n",
      "microbiota  27\n",
      "specific herve  28\n",
      "cheese  29\n",
      "orange rind cheese  30\n",
      "particular bacterium b linens  31\n",
      "['Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Phenotype' 'Microorganism'\n",
      " 'Microorganism' 'Microorganism' 'Microorganism' 'Habitat' 'Microorganism'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat']\n",
      "(23, 300) (23,)\n",
      "  text                                           words   entity norm\n",
      "0   T9                                         cheese   Habitat  NaN\n",
      "1  T10                                       raw milk   Habitat  NaN\n",
      "2  T11                                 safety reasons   Habitat  NaN\n",
      "3  T12  due potential presence listeria monocytogenes   Habitat  NaN\n",
      "4  T13                                          herve   Habitat  NaN\n",
      "BB-norm+ner-F-25064656-003.txt\n",
      "(23, 300)\n",
      "(23, 300) (23, 300)\n",
      "['OntoBiotope OBT:001480' 'OntoBiotope OBT:001276' 'NCBI_Taxonomy 29340'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003494'\n",
      " 'OntoBiotope OBT:001480' 'OntoBiotope OBT:001645'\n",
      " 'OntoBiotope OBT:003188' 'OntoBiotope OBT:003494'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000103'\n",
      " 'OntoBiotope OBT:000061' 'NCBI_Taxonomy 1269' 'OntoBiotope OBT:000007'\n",
      " 'NCBI_Taxonomy 1716' 'NCBI_Taxonomy 56811' 'NCBI_Taxonomy 12721'\n",
      " 'NCBI_Taxonomy 260680' 'OntoBiotope OBT:000113' 'OntoBiotope OBT:000189'\n",
      " 'OntoBiotope OBT:001480' 'OntoBiotope OBT:001480'\n",
      " 'OntoBiotope OBT:000113']\n",
      "heart  3\n",
      "milk cheeses  4\n",
      "l lactis ssp cremoris staphylococcus  5\n",
      "equorum  6\n",
      "psychrobacter celer streptococcus salivarius ssp  7\n",
      "thermophilus heart  8\n",
      "milk  9\n",
      "l lactis ssp cremoris psychrobacter sp staph equorum  10\n",
      "['Habitat' 'Habitat' 'Microorganism' 'Microorganism' 'Microorganism'\n",
      " 'Microorganism' 'Habitat' 'Microorganism']\n",
      "(8, 300) (8,)\n",
      "  text                                              words         entity norm\n",
      "0   T3                                             heart         Habitat  NaN\n",
      "1   T4                                      milk cheeses         Habitat  NaN\n",
      "2   T5              l lactis ssp cremoris staphylococcus   Microorganism  NaN\n",
      "3   T6                                           equorum   Microorganism  NaN\n",
      "4   T7  psychrobacter celer streptococcus salivarius ssp   Microorganism  NaN\n",
      "BB-norm+ner-F-25064656-004.txt\n",
      "(8, 300)\n",
      "(8, 300) (8, 300)\n",
      "['NCBI_Taxonomy 260680' 'OntoBiotope OBT:001480' 'NCBI_Taxonomy 1359'\n",
      " 'NCBI_Taxonomy 246432' 'NCBI_Taxonomy 1255614' 'NCBI_Taxonomy 1582'\n",
      " 'OntoBiotope OBT:001645' 'NCBI_Taxonomy 1352']\n",
      "raw milk cheeses  3\n",
      "corynebacterium casei l lactis ssp cremoris staph equorum marinilactibacillus psychrotolerans  4\n",
      "vagococcus  5\n",
      "salmoninarum psychrobacter spp fusobacterium spp clostridiisalibacter  6\n",
      "many species  7\n",
      "present levels  8\n",
      "figure rind  9\n",
      "milk  10\n",
      "composed species  11\n",
      "raw milk cheeses  12\n",
      "psychrobacter  13\n",
      "l lactis ssp cremoris c casei v salmoninarum staph equorum vibrio spp fusobacterium spp psychrotolerans  14\n",
      "pseudoalteromonas  15\n",
      "spp vagococcus fluvialis  16\n",
      "strep  17\n",
      "salivarius species  18\n",
      "present  19\n",
      "b linens  20\n",
      "figure  21\n",
      "['Habitat' 'Microorganism' 'Microorganism' 'Microorganism' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism'\n",
      " 'Microorganism' 'Microorganism' 'Microorganism' 'Habitat' 'Microorganism'\n",
      " 'Habitat' 'Microorganism' 'Habitat']\n",
      "(19, 300) (19,)\n",
      "  text                                              words         entity norm\n",
      "0   T3                                  raw milk cheeses         Habitat  NaN\n",
      "1   T4  corynebacterium casei l lactis ssp cremoris st...  Microorganism  NaN\n",
      "2   T5                                        vagococcus   Microorganism  NaN\n",
      "3   T6  salmoninarum psychrobacter spp fusobacterium s...  Microorganism  NaN\n",
      "4   T7                                      many species         Habitat  NaN\n",
      "BB-norm+ner-F-25064656-005.txt\n",
      "(19, 300)\n",
      "(19, 300) (19, 300)\n",
      "['OntoBiotope OBT:001480' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000011' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000169' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:001645'\n",
      " 'OntoBiotope OBT:002337' 'OntoBiotope OBT:001480' 'NCBI_Taxonomy 497'\n",
      " 'NCBI_Taxonomy 1243' 'NCBI_Taxonomy 53246' 'OntoBiotope OBT:002732'\n",
      " 'OntoBiotope OBT:002806' 'NCBI_Taxonomy 69966' 'NCBI_Taxonomy 12721'\n",
      " 'NCBI_Taxonomy 1703' 'OntoBiotope OBT:001828']\n",
      "phylotypes  4\n",
      "raw milk  5\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "cheese rind  6\n",
      "corynebacterium phylotypes  7\n",
      "streptococcus staphylococcus vagococcus genera  8\n",
      "supplementary figure http dx doi org jds hand phylotypes  9\n",
      "milk  10\n",
      "cheese rind psychrobacter spp pseudoalteromonas spp vibrio  11\n",
      "spp supplementary figure  12\n",
      "['Microorganism' 'Habitat' 'Habitat' 'Microorganism' 'Microorganism'\n",
      " 'Habitat' 'Habitat' 'Microorganism' 'Habitat']\n",
      "(9, 300) (9,)\n",
      "  text                                            words         entity norm\n",
      "0   T4                                      phylotypes   Microorganism  NaN\n",
      "1   T5                                        raw milk         Habitat  NaN\n",
      "2   T6                                     cheese rind         Habitat  NaN\n",
      "3   T7                      corynebacterium phylotypes   Microorganism  NaN\n",
      "4   T8  streptococcus staphylococcus vagococcus genera   Microorganism  NaN\n",
      "BB-norm+ner-F-25064656-006.txt\n",
      "(9, 300)\n",
      "(9, 300) (9, 300)\n",
      "['OntoBiotope OBT:003269' 'OntoBiotope OBT:001276'\n",
      " 'OntoBiotope OBT:001481' 'OntoBiotope OBT:000061'\n",
      " 'OntoBiotope OBT:003269' 'NCBI_Taxonomy 1256511' 'OntoBiotope OBT:001645'\n",
      " 'OntoBiotope OBT:000648' 'OntoBiotope OBT:003269']\n",
      "corynebacterium casei  3\n",
      "present raw milk cheeses species  4\n",
      "p celer  5\n",
      "milk cheeses  6\n",
      "['Microorganism' 'Habitat' 'Microorganism' 'Habitat']\n",
      "(4, 300) (4,)\n",
      "  text                              words         entity norm\n",
      "0   T3             corynebacterium casei   Microorganism  NaN\n",
      "1   T4  present raw milk cheeses species         Habitat  NaN\n",
      "2   T5                           p celer   Microorganism  NaN\n",
      "3   T6                      milk cheeses         Habitat  NaN\n",
      "BB-norm+ner-F-25064656-007.txt\n",
      "(4, 300)\n",
      "(4, 300) (4, 300)\n",
      "['NCBI_Taxonomy 160386' 'OntoBiotope OBT:001480' 'NCBI_Taxonomy 306572'\n",
      " 'OntoBiotope OBT:001480']\n",
      "lactococcus lactis ssp cremoris  3\n",
      "cheeses  4\n",
      "ta  5\n",
      "bi  6\n",
      "al  7\n",
      "main species  8\n",
      "hearts  9\n",
      "raw milk  10\n",
      "milk cheeses  11\n",
      "lactococcus  12\n",
      "lactis ssp cremoris c casei  13\n",
      "main species  14\n",
      "raw milk  15\n",
      "cheese rinds  16\n",
      "whereas  17\n",
      "psychrobacter spp l lactis ssp cremoris c casei  18\n",
      "main species  19\n",
      "pasteurized milk  20\n",
      "cheese rinds  21\n",
      "['Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Microorganism' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Habitat' 'Habitat'\n",
      " 'Habitat']\n",
      "(19, 300) (19,)\n",
      "  text                             words         entity norm\n",
      "0   T3  lactococcus lactis ssp cremoris   Microorganism  NaN\n",
      "1   T4                          cheeses         Habitat  NaN\n",
      "2   T5                               ta         Habitat  NaN\n",
      "3   T6                               bi         Habitat  NaN\n",
      "4   T7                               al         Habitat  NaN\n",
      "BB-norm+ner-F-25064656-008.txt\n",
      "(19, 300)\n",
      "(19, 300) (19, 300)\n",
      "['OntoBiotope OBT:000007' 'OntoBiotope OBT:001480' 'NCBI_Taxonomy 1173061'\n",
      " 'NCBI_Taxonomy 1255591' 'OntoBiotope OBT:003146' 'NCBI_Taxonomy 272630'\n",
      " 'NCBI_Taxonomy 1255580' 'OntoBiotope OBT:001276' 'OntoBiotope OBT:001480'\n",
      " 'NCBI_Taxonomy 1357' 'NCBI_Taxonomy 160386' 'NCBI_Taxonomy 272630'\n",
      " 'OntoBiotope OBT:001276' 'OntoBiotope OBT:001481'\n",
      " 'OntoBiotope OBT:003269' 'NCBI_Taxonomy 1246' 'NCBI_Taxonomy 272630'\n",
      " 'OntoBiotope OBT:001645' 'OntoBiotope OBT:001481']\n",
      "psychrobacter celer vibrio  3\n",
      "spp populations  4\n",
      "milk cheeses  5\n",
      "p  6\n",
      "p  7\n",
      "figure  8\n",
      "whereas c casei enterococcus faecalis  9\n",
      "raw milk cheeses  10\n",
      "p figure  11\n",
      "['Microorganism' 'Habitat' 'Habitat' 'Microorganism' 'Microorganism'\n",
      " 'Habitat' 'Microorganism' 'Habitat' 'Habitat']\n",
      "(9, 300) (9,)\n",
      "  text                        words         entity norm\n",
      "0   T3  psychrobacter celer vibrio   Microorganism  NaN\n",
      "1   T4             spp populations         Habitat  NaN\n",
      "2   T5                milk cheeses         Habitat  NaN\n",
      "3   T6                           p   Microorganism  NaN\n",
      "4   T7                           p   Microorganism  NaN\n",
      "BB-norm+ner-F-25064656-009.txt\n",
      "(9, 300)\n",
      "(9, 300) (9, 300)\n",
      "['NCBI_Taxonomy 40269' 'OntoBiotope OBT:001480' 'OntoBiotope OBT:001480'\n",
      " 'NCBI_Taxonomy 585' 'NCBI_Taxonomy 585' 'OntoBiotope OBT:001828'\n",
      " 'NCBI_Taxonomy 1582' 'OntoBiotope OBT:001480' 'NCBI_Taxonomy 12721']\n",
      "presence p celer  9\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bacterium contributes  10\n",
      "aromatic properties  11\n",
      "cheese influences  12\n",
      "total volatile aroma production irlinger  13\n",
      "et  14\n",
      "al presence p celer  15\n",
      "cheeses  16\n",
      "presence  17\n",
      "saline water  18\n",
      "manufacturing process  19\n",
      "bacterium  20\n",
      "seawater yoon  21\n",
      "al vibrio spp  22\n",
      "cheese feurer  23\n",
      "al studies  24\n",
      "bacteria  25\n",
      "role  26\n",
      "process el baradei  27\n",
      "al  28\n",
      "new vibrio species  29\n",
      "vibrio  30\n",
      "isolated rind  31\n",
      "red smear  32\n",
      "et al presence c casei  33\n",
      "raw milk cheeses  34\n",
      "surprising  35\n",
      "et al  36\n",
      "['Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism'\n",
      " 'Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Phenotype' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Microorganism' 'Microorganism' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Habitat']\n",
      "(28, 300) (28,)\n",
      "  text                                      words         entity norm\n",
      "0   T9                          presence p celer   Microorganism  NaN\n",
      "1  T10                     bacterium contributes         Habitat  NaN\n",
      "2  T11                       aromatic properties         Habitat  NaN\n",
      "3  T12                         cheese influences         Habitat  NaN\n",
      "4  T13  total volatile aroma production irlinger         Habitat  NaN\n",
      "BB-norm+ner-F-25064656-010.txt\n",
      "(28, 300)\n",
      "(28, 300) (28, 300)\n",
      "['OntoBiotope OBT:000007' 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 1111734'\n",
      " 'OntoBiotope OBT:001480' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 56810' 'OntoBiotope OBT:001480'\n",
      " 'OntoBiotope OBT:003269' 'NCBI_Taxonomy 1582' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:001471' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1282'\n",
      " 'OntoBiotope OBT:000113' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000113' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 260680'\n",
      " 'OntoBiotope OBT:003146' 'NCBI_Taxonomy 28875' 'NCBI_Taxonomy 662'\n",
      " 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 210' 'OntoBiotope OBT:003428'\n",
      " 'OntoBiotope OBT:001480' 'NCBI_Taxonomy 471872' 'OntoBiotope OBT:000019']\n",
      "use p celer  5\n",
      "indicator  6\n",
      "milk cheese  7\n",
      "undertaken  8\n",
      "care  9\n",
      "['Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat']\n",
      "(5, 300) (5,)\n",
      "  text         words   entity norm\n",
      "0   T5  use p celer   Habitat  NaN\n",
      "1   T6    indicator   Habitat  NaN\n",
      "2   T7  milk cheese   Habitat  NaN\n",
      "3   T8   undertaken   Habitat  NaN\n",
      "4   T9         care   Habitat  NaN\n",
      "BB-norm+ner-F-25064656-011.txt\n",
      "(5, 300)\n",
      "(5, 300) (5, 300)\n",
      "['NCBI_Taxonomy 306572' 'OntoBiotope OBT:000360' 'OntoBiotope OBT:001480'\n",
      " 'OntoBiotope OBT:000113' 'NCBI_Taxonomy 260680']\n",
      "figure  10\n",
      "fecal contaminant e faecalis  11\n",
      "present raw milk cheeses  12\n",
      "present  13\n",
      "study counts  14\n",
      "low  15\n",
      "milk cheeses  16\n",
      "less differences  17\n",
      "due nature  18\n",
      "milk  19\n",
      "raw milk cheeses  20\n",
      "bacteria  21\n",
      "source typicality  22\n",
      "c  23\n",
      "casei e "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " 24\n",
      "faecalis microbiological hazard e faecalis  25\n",
      "common enterococci  26\n",
      "cheese e faecalis  27\n",
      "enterococcus  28\n",
      "faecium e faecium  29\n",
      "species dairy cows  30\n",
      "gelsomino  31\n",
      "al  32\n",
      "e  33\n",
      "faecium  34\n",
      "raw milk  35\n",
      "cheese samples addition  36\n",
      "enterococcus  37\n",
      "casseliflavus  38\n",
      "cheese gelsomino et al  39\n",
      "samples  40\n",
      "raw milk cheeses  41\n",
      "milk cheeses  42\n",
      "['Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Phenotype' 'Habitat'\n",
      " 'Microorganism' 'Microorganism' 'Microorganism' 'Microorganism' 'Habitat'\n",
      " 'Microorganism' 'Microorganism' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Microorganism' 'Habitat' 'Habitat' 'Microorganism'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat']\n",
      "(33, 300) (33,)\n",
      "  text                          words         entity norm\n",
      "0  T10                        figure         Habitat  NaN\n",
      "1  T11  fecal contaminant e faecalis   Microorganism  NaN\n",
      "2  T12      present raw milk cheeses         Habitat  NaN\n",
      "3  T13                       present         Habitat  NaN\n",
      "4  T14                  study counts         Habitat  NaN\n",
      "BB-norm+ner-F-25064656-012.txt\n",
      "(33, 300)\n",
      "(33, 300) (33, 300)\n",
      "['OntoBiotope OBT:001828' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:001480' 'NCBI_Taxonomy 12721' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:001480'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 260680' 'OntoBiotope OBT:001645'\n",
      " 'OntoBiotope OBT:001480' 'OntoBiotope OBT:000113' 'NCBI_Taxonomy 1697'\n",
      " 'NCBI_Taxonomy 263' 'NCBI_Taxonomy 160386' 'NCBI_Taxonomy 1173061'\n",
      " 'OntoBiotope OBT:002806' 'NCBI_Taxonomy 1351' 'NCBI_Taxonomy 1350'\n",
      " 'NCBI_Taxonomy 1352' 'NCBI_Taxonomy 1357' 'NCBI_Taxonomy 260680'\n",
      " 'OntoBiotope OBT:003146' 'NCBI_Taxonomy 1311' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:001276' 'OntoBiotope OBT:001480' 'NCBI_Taxonomy 1350'\n",
      " 'NCBI_Taxonomy 28985' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1400'\n",
      " 'OntoBiotope OBT:001480' 'OntoBiotope OBT:001480']\n",
      "phylotypes  9\n",
      "samples  10\n",
      "present study vagococcus  11\n",
      "vagococcus salmoninarum species  12\n",
      "salmonid fishes  13\n",
      "related carnobacterium genus wallbanks  14\n",
      "et  15\n",
      "several raw milk  16\n",
      "cheese samples  17\n",
      "vagococcus  18\n",
      "animal human sources  19\n",
      "et al pot  20\n",
      "al teixeira et  21\n",
      "cheese callon et al  22\n",
      "surprising identify  23\n",
      "pathogenic bacteria morganella morganii  24\n",
      "milk  25\n",
      "cheese tornadijo  26\n",
      "al delb paus et al  27\n",
      "bacterium  28\n",
      "able produce  29\n",
      "biogenic amines  30\n",
      "histamine  31\n",
      "responsible production  32\n",
      "cadaverine  33\n",
      "pressed cheeses  34\n",
      "delb  35\n",
      "paus  36\n",
      "al  37\n",
      "genus citricoccus  38\n",
      "bacterium  39\n",
      "cheese  40\n",
      "several times walls  41\n",
      "mold  42\n",
      "walls  43\n",
      "al sch fer  44\n",
      "al  45\n",
      "['Microorganism' 'Habitat' 'Habitat' 'Microorganism' 'Habitat'\n",
      " 'Microorganism' 'Microorganism' 'Habitat' 'Habitat' 'Microorganism'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism'\n",
      " 'Habitat' 'Habitat' 'Microorganism' 'Microorganism' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat']\n",
      "(37, 300) (37,)\n",
      "  text                             words         entity norm\n",
      "0   T9                       phylotypes   Microorganism  NaN\n",
      "1  T10                          samples         Habitat  NaN\n",
      "2  T11         present study vagococcus         Habitat  NaN\n",
      "3  T12  vagococcus salmoninarum species   Microorganism  NaN\n",
      "4  T13                  salmonid fishes         Habitat  NaN\n",
      "BB-norm+ner-F-25064656-013.txt\n",
      "(37, 300)\n",
      "(37, 300) (37, 300)\n",
      "['OntoBiotope OBT:003269' 'NCBI_Taxonomy 1400' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:002732' 'NCBI_Taxonomy 10508' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:001480'\n",
      " 'OntoBiotope OBT:001200' 'OntoBiotope OBT:000011' 'NCBI_Taxonomy 1288'\n",
      " 'OntoBiotope OBT:001030' 'OntoBiotope OBT:000526'\n",
      " 'OntoBiotope OBT:000113' 'NCBI_Taxonomy 471872' 'NCBI_Taxonomy 210'\n",
      " 'OntoBiotope OBT:001645' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 260680'\n",
      " 'OntoBiotope OBT:001471' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 246432' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:001480' 'NCBI_Taxonomy 1358'\n",
      " 'NCBI_Taxonomy 615' 'OntoBiotope OBT:003146' 'NCBI_Taxonomy 1301'\n",
      " 'OntoBiotope OBT:001471' 'OntoBiotope OBT:001480' 'NCBI_Taxonomy 1255655'\n",
      " 'OntoBiotope OBT:000143' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:001030' 'OntoBiotope OBT:003146']\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "addition presence  3\n",
      "halophilic alkaliphilic lactic acid bacteria  4\n",
      "present study psychrotolerans  5\n",
      "bacterium  6\n",
      "marine organisms  7\n",
      "certain areas  8\n",
      "japan  9\n",
      "ishikawa  10\n",
      "al  11\n",
      "cheese ishikawa  12\n",
      "al  13\n",
      "possess anti listeria effect  14\n",
      "roth  15\n",
      "al  16\n",
      "bacterium  17\n",
      "study genus halomonas  18\n",
      "smear ripened cheeses  19\n",
      "al identification samples  20\n",
      "pseudoalteromonas  21\n",
      "unexpected presence bacteria  22\n",
      "cheese  23\n",
      "ogier et  24\n",
      "al  25\n",
      "['Habitat' 'Phenotype' 'Habitat' 'Microorganism' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Microorganism' 'Microorganism' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Microorganism' 'Habitat']\n",
      "(23, 300) (23,)\n",
      "  text                                          words         entity norm\n",
      "0   T3                             addition presence         Habitat  NaN\n",
      "1   T4  halophilic alkaliphilic lactic acid bacteria       Phenotype  NaN\n",
      "2   T5                 present study psychrotolerans         Habitat  NaN\n",
      "3   T6                                     bacterium   Microorganism  NaN\n",
      "4   T7                              marine organisms         Habitat  NaN\n",
      "BB-norm+ner-F-25064656-014.txt\n",
      "(23, 300)\n",
      "(23, 300) (23, 300)\n",
      "['OntoBiotope OBT:000029' 'OntoBiotope OBT:000113'\n",
      " 'OntoBiotope OBT:000011' 'OntoBiotope OBT:001471'\n",
      " 'OntoBiotope OBT:000010' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1051' 'OntoBiotope OBT:003146'\n",
      " 'OntoBiotope OBT:001480' 'OntoBiotope OBT:003146'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000222'\n",
      " 'OntoBiotope OBT:003146' 'OntoBiotope OBT:001471' 'NCBI_Taxonomy 1173061'\n",
      " 'OntoBiotope OBT:003494' 'OntoBiotope OBT:000031' 'NCBI_Taxonomy 53246'\n",
      " 'OntoBiotope OBT:000801' 'OntoBiotope OBT:001480' 'NCBI_Taxonomy 1359'\n",
      " 'OntoBiotope OBT:003146']\n",
      "brachybacterium faecium  3\n",
      "study bacterium  4\n",
      "close brachybacterium species  5\n",
      "french  6\n",
      "beaufort cheeses  7\n",
      "schubert  8\n",
      "et al leucobacter spp alkalibacterium spp carnobacterium spp raoultella planticola  9\n",
      "milk  10\n",
      "cheese herbin  11\n",
      "al ishikawa  12\n",
      "al larpin laborde  13\n",
      "al  14\n",
      "raw milk cheeses  15\n",
      "current study  16\n",
      "['Microorganism' 'Phenotype' 'Microorganism' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat']\n",
      "(14, 300) (14,)\n",
      "  text                           words         entity norm\n",
      "0   T3        brachybacterium faecium   Microorganism  NaN\n",
      "1   T4                study bacterium       Phenotype  NaN\n",
      "2   T5  close brachybacterium species   Microorganism  NaN\n",
      "3   T6                         french         Habitat  NaN\n",
      "4   T7               beaufort cheeses         Habitat  NaN\n",
      "BB-norm+ner-F-25064656-015.txt\n",
      "(14, 300)\n",
      "(14, 300) (14, 300)\n",
      "['NCBI_Taxonomy 1716' 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 813'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:001480'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:001645' 'OntoBiotope OBT:001480'\n",
      " 'OntoBiotope OBT:002034' 'NCBI_Taxonomy 260680' 'OntoBiotope OBT:003146'\n",
      " 'OntoBiotope OBT:001480' 'OntoBiotope OBT:000019']\n",
      "previous study  12\n",
      "psychroflexus casei  13\n",
      "red smear color raclette type  14\n",
      "cheese seiler et  15\n",
      "al hypothesis conjunction b linens  16\n",
      "bacteria bacterium  17\n",
      "role orange  18\n",
      "coloration herve  19\n",
      "cheese  20\n",
      "raw  21\n",
      "cheeses  22\n",
      "b  23\n",
      "dominant bacteria  24\n",
      "raw  25\n",
      "rind microbiota  26\n",
      "bacteria  27\n",
      "process result agreement  28\n",
      "previous  29\n",
      "studies  30\n",
      "b linens  31\n",
      "significant member  32\n",
      "surface microbiota  33\n",
      "smear  34\n",
      "cheese  35\n",
      "major component  36\n",
      "cultures  37\n",
      "brennan  38\n",
      "et al gori et al  39\n",
      "possible preponderant bacteria psychrobacter  40\n",
      "staphylococcus  41\n",
      "role coloration  42\n",
      "smear  43\n",
      "cheeses  44\n",
      "hoppe seyler et  45\n",
      "yellow orange color rind cheeses  46\n",
      "interactions  47\n",
      "nonpigmented b linens  48\n",
      "yellow  49\n",
      "ogier et al leclercq  50\n",
      "perlat spinnler  51\n",
      "['Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Microorganism'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Microorganism'\n",
      " 'Phenotype' 'Habitat' 'Habitat' 'Phenotype' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Microorganism' 'Habitat' 'Microorganism' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat']\n",
      "(40, 300) (40,)\n",
      "  text                                words         entity norm\n",
      "0  T12                      previous study         Habitat  NaN\n",
      "1  T13                 psychroflexus casei   Microorganism  NaN\n",
      "2  T14       red smear color raclette type         Habitat  NaN\n",
      "3  T15                    cheese seiler et         Habitat  NaN\n",
      "4  T16  al hypothesis conjunction b linens         Habitat  NaN\n",
      "BB-norm+ner-F-25064656-016.txt\n",
      "(40, 300)\n",
      "(40, 300) (40, 300)\n",
      "['OntoBiotope OBT:003269' 'NCBI_Taxonomy 160386' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:003269' 'NCBI_Taxonomy 1703' 'OntoBiotope OBT:000061'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003483'\n",
      " 'OntoBiotope OBT:001480' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:001480' 'NCBI_Taxonomy 1359' 'OntoBiotope OBT:002806'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000061'\n",
      " 'OntoBiotope OBT:000113' 'OntoBiotope OBT:000031' 'NCBI_Taxonomy 1047'\n",
      " 'OntoBiotope OBT:000007' 'NCBI_Taxonomy 1703' 'NCBI_Taxonomy 1182512'\n",
      " 'OntoBiotope OBT:000113' 'OntoBiotope OBT:003494'\n",
      " 'OntoBiotope OBT:001480' 'NCBI_Taxonomy 37763' 'OntoBiotope OBT:001030'\n",
      " 'NCBI_Taxonomy 12721' 'OntoBiotope OBT:002034' 'NCBI_Taxonomy 1046'\n",
      " 'NCBI_Taxonomy 1279' 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003494'\n",
      " 'OntoBiotope OBT:001480' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:003473' 'NCBI_Taxonomy 12721' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:002161' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:002806']\n",
      "study  3\n",
      "presence bavariicoccus  4\n",
      "gen nov sp  5\n",
      "nov surface  6\n",
      "smear water  7\n",
      "german red smear  8\n",
      "soft cheese schmidt et al  9\n",
      "bacterium  10\n",
      "samples  11\n",
      "previous study  12\n",
      "color b  13\n",
      "yeast  14\n",
      "deacidification muenster  15\n",
      "cheese leclercq  16\n",
      "perlat  17\n",
      "al presence yeast  18\n",
      "study  19\n",
      "['Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat']\n",
      "(17, 300) (17,)\n",
      "  text                    words   entity norm\n",
      "0   T3                   study   Habitat  NaN\n",
      "1   T4  presence bavariicoccus   Habitat  NaN\n",
      "2   T5              gen nov sp   Habitat  NaN\n",
      "3   T6             nov surface   Habitat  NaN\n",
      "4   T7             smear water   Habitat  NaN\n",
      "BB-norm+ner-F-25064656-017.txt\n",
      "(17, 300)\n",
      "(17, 300) (17, 300)\n",
      "['OntoBiotope OBT:003269' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 83560'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:001341'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 260680' 'OntoBiotope OBT:001471'\n",
      " 'NCBI_Taxonomy 1400' 'OntoBiotope OBT:003269' 'OntoBiotope OBT:003066'\n",
      " 'OntoBiotope OBT:000113' 'NCBI_Taxonomy 1255655' 'OntoBiotope OBT:001480'\n",
      " 'NCBI_Taxonomy 1743' 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003269']\n",
      "vitamin b content soy yogurt lactobacillus reuteri  3\n",
      "['Habitat']\n",
      "(1, 300) (1,)\n",
      "  text                                              words   entity norm\n",
      "0   T3  vitamin b content soy yogurt lactobacillus reu...  Habitat  NaN\n",
      "BB-norm+ner-F-25955289-000.txt\n",
      "(1, 300)\n",
      "(1, 300) (1, 300)\n",
      "['OntoBiotope OBT:001480']\n",
      "glycerol fructose  3\n",
      "enhance production vitamin b glycerol fructose "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " 4\n",
      "induced expression cobt cbia glycerol fructose  5\n",
      "balance redox reaction  6\n",
      "high content  7\n",
      "fructose supplementation  8\n",
      "expression cobt vitamin b content soymilk  9\n",
      "fermented soybean food  10\n",
      "['Habitat' 'Habitat' 'Habitat' 'Phenotype' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat']\n",
      "(8, 300) (8,)\n",
      "  text                                            words     entity norm\n",
      "0   T3                               glycerol fructose     Habitat  NaN\n",
      "1   T4  enhance production vitamin b glycerol fructose     Habitat  NaN\n",
      "2   T5  induced expression cobt cbia glycerol fructose     Habitat  NaN\n",
      "3   T6                          balance redox reaction   Phenotype  NaN\n",
      "4   T7                                    high content     Habitat  NaN\n",
      "BB-norm+ner-F-25955289-001.txt\n",
      "(8, 300)\n",
      "(8, 300) (8, 300)\n",
      "['OntoBiotope OBT:000007' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:002712' 'NCBI_Taxonomy 1047'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019']\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Warning: parsing empty text\n",
      "attention  12\n",
      "vegetarians  13\n",
      "soy product  14\n",
      "due taste  15\n",
      "easy digestibility  16\n",
      "association  17\n",
      "health  18\n",
      "soy product defect  19\n",
      "low vitamin content  20\n",
      "water  21\n",
      "soluble vitamin b study  22\n",
      "co fermentation glycerol  23\n",
      "fructose soy yogurt enhance vitamin b production  24\n",
      "reuteri serial combination experiments  25\n",
      "co fermentation  26\n",
      "enhance production vitamin b g ml supplementations  27\n",
      "expression cobt cbia  28\n",
      "balance redox reaction  29\n",
      "high content  30\n",
      "fructose supplementation  31\n",
      "production vitamin b  32\n",
      "expression cobt bacteria  33\n",
      "vitamin b  34\n",
      "content soy yogurt  35\n",
      "soybean  36\n",
      "food  37\n",
      "alternative food  38\n",
      "vegetarians  39\n",
      "['Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Phenotype' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat']\n",
      "(28, 300) (28,)\n",
      "  text                words   entity norm\n",
      "0  T12           attention   Habitat  NaN\n",
      "1  T13         vegetarians   Habitat  NaN\n",
      "2  T14         soy product   Habitat  NaN\n",
      "3  T15           due taste   Habitat  NaN\n",
      "4  T16  easy digestibility   Habitat  NaN\n",
      "BB-norm+ner-F-25955289-002.txt\n",
      "(28, 300)\n",
      "(28, 300) (28, 300)\n",
      "['NCBI_Taxonomy 1173061' 'OntoBiotope OBT:000094' 'OntoBiotope OBT:000827'\n",
      " 'OntoBiotope OBT:002806' 'NCBI_Taxonomy 10508' 'OntoBiotope OBT:003269'\n",
      " 'OntoBiotope OBT:001231' 'OntoBiotope OBT:003220'\n",
      " 'OntoBiotope OBT:003220' 'OntoBiotope OBT:000468'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:003269' 'NCBI_Taxonomy 1752'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000360'\n",
      " 'OntoBiotope OBT:002712' 'NCBI_Taxonomy 1047' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:001480' 'NCBI_Taxonomy 1246'\n",
      " 'OntoBiotope OBT:000094' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000094']\n",
      "watanabe et al  18\n",
      "japanese natto korean soybean products  19\n",
      "contain  20\n",
      "g  21\n",
      "g samples  22\n",
      "microbiological assay  23\n",
      "analogs  24\n",
      "lactobacillus plantarum sm propionibacterium freudenreichii  25\n",
      "cofermentation whey  26\n",
      "g cobalamin  27\n",
      "ml hugenschmidt  28\n",
      "al watanabe et al  29\n",
      "addition propionibacteria cabbage sauerkraut production  30\n",
      "higher concentrations  31\n",
      "b  32\n",
      "g ml  33\n",
      "less vitamin b fermentations  34\n",
      "samples  35\n",
      "pungent odor  36\n",
      "propionic acid work cofermentation glycerol fructose  37\n",
      "positive effect production vitamin b table glycerol  38\n",
      "vital interesting topic vitamin b production  39\n",
      "growth cells  40\n",
      "glycerol  41\n",
      "inducer vitamin b synthesis cluster  42\n",
      "roth  43\n",
      "al fig cobt cbia genes  44\n",
      "glycerol cbia  45\n",
      "cobt increase expression  46\n",
      "accelerated synthesis vitamin b  47\n",
      "higher production vitamin b condition vitamin b synthesis  48\n",
      "production vitamin b analogs vitamin b  49\n",
      "dependent enzymes  50\n",
      "l  51\n",
      "reuteri  52\n",
      "conversion glycerol hpa order regenerate nad  53\n",
      "atp generation researchers  54\n",
      "glycerol  55\n",
      "serves  56\n",
      "external hydrogen acceptor glycerol  57\n",
      "fermentation l reuteri work carbon source sriramulu  58\n",
      "al work glycerol  59\n",
      "external hydrogen acceptor  60\n",
      "table agreement results  61\n",
      "obvious shift end products  62\n",
      "ethanol  63\n",
      "acetate  64\n",
      "results  65\n",
      "table similar results  66\n",
      "work l thi peng et al  67\n",
      "coworkers  68\n",
      "sufficient supplementation glucose  69\n",
      "glycerol  70\n",
      "generation propanediol  71\n",
      "acetate  72\n",
      "accumulation hpa lactate vitamin b production table  73\n",
      "supplementation glycerol  74\n",
      "glucose  75\n",
      "utilized  76\n",
      "efficient pathway generate atp glucose  77\n",
      "redox balance  78\n",
      "high concentrations  79\n",
      "glycerol supplementations  80\n",
      "work  81\n",
      "inhibited growth  82\n",
      "cells  83\n",
      "activity glycerol dehydratase  84\n",
      "quorum  85\n",
      "reuterin bauer et al  86\n",
      "['Habitat' 'Habitat' 'Habitat' 'Microorganism' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Phenotype' 'Microorganism' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Phenotype' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Habitat' 'Phenotype' 'Habitat'\n",
      " 'Phenotype' 'Habitat' 'Habitat']\n",
      "(69, 300) (69,)\n",
      "  text                                    words         entity norm\n",
      "0  T18                          watanabe et al         Habitat  NaN\n",
      "1  T19  japanese natto korean soybean products         Habitat  NaN\n",
      "2  T20                                 contain         Habitat  NaN\n",
      "3  T21                                       g   Microorganism  NaN\n",
      "4  T22                               g samples         Habitat  NaN\n",
      "BB-norm+ner-F-25955289-003.txt\n",
      "(69, 300)\n",
      "(69, 300) (69, 300)\n",
      "['NCBI_Taxonomy 1359' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1255655'\n",
      " 'NCBI_Taxonomy 1173061' 'OntoBiotope OBT:003269' 'OntoBiotope OBT:000011'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:001480' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:002034'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 28038' 'NCBI_Taxonomy 1359'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1400'\n",
      " 'OntoBiotope OBT:002806' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000061' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000222'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1597' 'OntoBiotope OBT:000007'\n",
      " 'NCBI_Taxonomy 63186' 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 29412' 'OntoBiotope OBT:003451'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000064'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000113' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000038' 'OntoBiotope OBT:000031'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000006'\n",
      " 'OntoBiotope OBT:000061' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000031' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000007'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000007' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000006' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000801' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000143' 'OntoBiotope OBT:000019']\n",
      "staphylococcus  3\n",
      "inhibition  4\n",
      "higher lactococcus garvieae  5\n",
      "hydrogen peroxide h  6\n",
      "['Microorganism' 'Phenotype' 'Microorganism' 'Habitat']\n",
      "(4, 300) (4,)\n",
      "  text                         words         entity norm\n",
      "0   T3               staphylococcus   Microorganism  NaN\n",
      "1   T4                   inhibition       Phenotype  NaN\n",
      "2   T5  higher lactococcus garvieae   Microorganism  NaN\n",
      "3   T6          hydrogen peroxide h         Habitat  NaN\n",
      "BB-norm+ner-F-26187841-000.txt\n",
      "(4, 300)\n",
      "(4, 300) (4, 300)\n",
      "['NCBI_Taxonomy 1279' 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 186765'\n",
      " 'OntoBiotope OBT:000019']\n",
      "growth  6\n",
      "foodborne pathogen staphylococcus aureus  7\n",
      "milk  8\n",
      "cheese hydrogen  9\n",
      "lactococcus garvieae  10\n",
      "n dairy strain  11\n",
      "transcriptomic responses  12\n",
      "aureus  13\n",
      "strains  14\n",
      "sa dairy strain  15\n",
      "mw human pathogenic strain growth  16\n",
      "inhibition  17\n",
      "brain heart infusion  18\n",
      "high  19\n",
      "low aeration level  20\n",
      "['Habitat' 'Microorganism' 'Habitat' 'Habitat' 'Microorganism' 'Habitat'\n",
      " 'Habitat' 'Microorganism' 'Microorganism' 'Habitat' 'Habitat' 'Phenotype'\n",
      " 'Habitat' 'Habitat' 'Habitat']\n",
      "(15, 300) (15,)\n",
      "  text                                      words         entity norm\n",
      "0   T6                                    growth         Habitat  NaN\n",
      "1   T7  foodborne pathogen staphylococcus aureus   Microorganism  NaN\n",
      "2   T8                                      milk         Habitat  NaN\n",
      "3   T9                           cheese hydrogen         Habitat  NaN\n",
      "4  T10                      lactococcus garvieae   Microorganism  NaN\n",
      "BB-norm+ner-F-26187841-001.txt\n",
      "(15, 300)\n",
      "(15, 300) (15, 300)\n",
      "['OntoBiotope OBT:000019' 'OntoBiotope OBT:002669'\n",
      " 'OntoBiotope OBT:001645' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000061' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:003269' 'NCBI_Taxonomy 1280' 'OntoBiotope OBT:000061'\n",
      " 'OntoBiotope OBT:000019' 'NCBI_Taxonomy 1773' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000019' 'OntoBiotope OBT:000019'\n",
      " 'OntoBiotope OBT:000328']\n",
      "lab lactococcus garvieae  3\n",
      "dairy products  4\n",
      "inhibit  5\n",
      "aureus growth  6\n",
      "milk cheese  7\n",
      "raw pasteurized milk alomar  8\n",
      "al alomar  9\n",
      "al b delbes paus et al  10\n",
      "['Microorganism' 'Habitat' 'Phenotype' 'Microorganism' 'Habitat' 'Habitat'\n",
      " 'Habitat' 'Microorganism']\n",
      "(8, 300) (8,)\n",
      "  text                      words         entity norm\n",
      "0   T3  lab lactococcus garvieae   Microorganism  NaN\n",
      "1   T4            dairy products         Habitat  NaN\n",
      "2   T5                   inhibit       Phenotype  NaN\n",
      "3   T6             aureus growth   Microorganism  NaN\n",
      "4   T7               milk cheese         Habitat  NaN\n",
      "BB-norm+ner-F-26187841-002.txt\n",
      "(8, 300)\n",
      "(8, 300) (8, 300)\n",
      "['OntoBiotope OBT:000019' 'OntoBiotope OBT:001208' 'NCBI_Taxonomy 1743'\n",
      " 'NCBI_Taxonomy 1496' 'OntoBiotope OBT:001480' 'NCBI_Taxonomy 777'\n",
      " 'OntoBiotope OBT:003188' 'NCBI_Taxonomy 260680']\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\ipykernel_launcher.py:11: FutureWarning: read_table is deprecated, use read_csv instead, passing sep='\\t'.\n",
      "  # This is added back by InteractiveShellApp.init_path()\n"
     ]
    }
   ],
   "source": [
    "path = 'BioNLP-OST-2019_BB-norm+ner_test'\n",
    "predict_test_data(path)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "pre = model_class.predict(X_test)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "data.to_csv(\"a.a2\",index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## cross validation"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 77,
   "metadata": {},
   "outputs": [],
   "source": [
    "from sklearn.model_selection import GridSearchCV"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 87,
   "metadata": {},
   "outputs": [],
   "source": [
    "parameters = {'criterion':['gini'],'class_weight':['balanced']}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 88,
   "metadata": {},
   "outputs": [],
   "source": [
    "clf = RandomForestClassifier()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 121,
   "metadata": {},
   "outputs": [],
   "source": [
    "rand_clf = GridSearchCV(clf, parameters, cv=3, scoring = 'accuracy')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 122,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\sklearn\\model_selection\\_split.py:652: Warning: The least populated class in y has only 1 members, which is too few. The minimum number of members in any class cannot be less than n_splits=3.\n",
      "  % (min_groups, self.n_splits)), Warning)\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\sklearn\\ensemble\\forest.py:246: FutureWarning: The default value of n_estimators will change from 10 in version 0.20 to 100 in 0.22.\n",
      "  \"10 in version 0.20 to 100 in 0.22.\", FutureWarning)\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\sklearn\\ensemble\\forest.py:246: FutureWarning: The default value of n_estimators will change from 10 in version 0.20 to 100 in 0.22.\n",
      "  \"10 in version 0.20 to 100 in 0.22.\", FutureWarning)\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\sklearn\\ensemble\\forest.py:246: FutureWarning: The default value of n_estimators will change from 10 in version 0.20 to 100 in 0.22.\n",
      "  \"10 in version 0.20 to 100 in 0.22.\", FutureWarning)\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\sklearn\\model_selection\\_search.py:841: DeprecationWarning: The default of the `iid` parameter will change from True to False in version 0.22 and will be removed in 0.24. This will change numeric results when test-set sizes are unequal.\n",
      "  DeprecationWarning)\n",
      "C:\\ProgramData\\Anaconda3\\lib\\site-packages\\sklearn\\ensemble\\forest.py:246: FutureWarning: The default value of n_estimators will change from 10 in version 0.20 to 100 in 0.22.\n",
      "  \"10 in version 0.20 to 100 in 0.22.\", FutureWarning)\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "GridSearchCV(cv=3, error_score='raise-deprecating',\n",
       "       estimator=RandomForestClassifier(bootstrap=True, class_weight=None, criterion='gini',\n",
       "            max_depth=None, max_features='auto', max_leaf_nodes=None,\n",
       "            min_impurity_decrease=0.0, min_impurity_split=None,\n",
       "            min_samples_leaf=1, min_samples_split=2,\n",
       "            min_weight_fraction_leaf=0.0, n_estimators='warn', n_jobs=None,\n",
       "            oob_score=False, random_state=None, verbose=0,\n",
       "            warm_start=False),\n",
       "       fit_params=None, iid='warn', n_jobs=None,\n",
       "       param_grid={'criterion': ['gini'], 'class_weight': ['balanced']},\n",
       "       pre_dispatch='2*n_jobs', refit=True, return_train_score='warn',\n",
       "       scoring='accuracy', verbose=0)"
      ]
     },
     "execution_count": 122,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "rand_clf.fit(X_train,Y)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 127,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0.9256425948592412"
      ]
     },
     "execution_count": 127,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "rand_clf.score(X_train,Y)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>T3\tHabitat 61 70\tpediatric</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>T4\\tHabitat 178 189\\trespiratory</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>T5\\tHabitat 256 267\\trespiratory</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>T6\\tHabitat 281 290\\tpediatric</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>T7\\tHabitat 339 372\\tchildren less than 2 year...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>T8\\tHabitat 418 488\\tchildren greater than 9 y...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>T9\\tHabitat 469 480\\trespiratory</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>T10\\tHabitat 490 495\\tMales</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>T11\\tHabitat 543 550\\tfemales</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>T12\\tHabitat 573 578\\tmales</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>T13\\tPhenotype 600 621\\tnonbacterial pathogen</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>T14\\tHabitat 648 666\\tpatients with WARI</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>T15\\tHabitat 668 679\\trespiratory</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>T16\\tMicroorganism 668 695\\trespiratory syncyt...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>T17\\tMicroorganism 697 724\\tparainfluenza viru...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>T18\\tMicroorganism 697 722;729 730\\tparainflue...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>T19\\tMicroorganism 732 744\\tadenoviruses</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>T20\\tMicroorganism 750 771\\tMycoplasma pneumoniae</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>T21\\tHabitat 807 814\\tPatient</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>T22\\tPhenotype 887 933\\tcause of WARI in child...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>T23\\tHabitat 904 933\\tchildren under 5 years o...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>T24\\tMicroorganism 938 941\\tRSV</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>T25\\tMicroorganism 950 971\\tMycoplasma pneumoniae</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>T26\\tHabitat 1007 1048\\tschool age children wi...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>T27\\tHabitat 1124 1138\\tdiagnosticians</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>T28\\tHabitat 1146 1157\\tresearchers</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>T29\\tHabitat 1193 1204\\trespiratory</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>N1\\tOntoBiotope Annotation:T3 Referent:OBT:003188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>N2\\tOntoBiotope Annotation:T3 Referent:OBT:003220</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>N3\\tOntoBiotope Annotation:T4 Referent:OBT:000407</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>N4\\tOntoBiotope Annotation:T5 Referent:OBT:000407</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>N9\\tOntoBiotope Annotation:T8 Referent:OBT:003220</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>N10\\tOntoBiotope Annotation:T8 Referent:OBT:00...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>N11\\tOntoBiotope Annotation:T9 Referent:OBT:00...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>N12\\tOntoBiotope Annotation:T10 Referent:OBT:0...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>N13\\tOntoBiotope Annotation:T10 Referent:OBT:0...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>N14\\tOntoBiotope Annotation:T11 Referent:OBT:0...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>N15\\tOntoBiotope Annotation:T11 Referent:OBT:0...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>N16\\tOntoBiotope Annotation:T12 Referent:OBT:0...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>N17\\tOntoBiotope Annotation:T12 Referent:OBT:0...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>N18\\tOntoBiotope Annotation:T13 Referent:OBT:0...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>N19\\tOntoBiotope Annotation:T14 Referent:OBT:0...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45</th>\n",
       "      <td>N20\\tOntoBiotope Annotation:T14 Referent:OBT:0...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>46</th>\n",
       "      <td>N21\\tOntoBiotope Annotation:T15 Referent:OBT:0...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47</th>\n",
       "      <td>N22\\tNCBI_Taxonomy Annotation:T16 Referent:12814</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>48</th>\n",
       "      <td>N23\\tNCBI_Taxonomy Annotation:T17 Referent:12730</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>49</th>\n",
       "      <td>N24\\tNCBI_Taxonomy Annotation:T18 Referent:11216</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>50</th>\n",
       "      <td>N25\\tNCBI_Taxonomy Annotation:T19 Referent:10508</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>51</th>\n",
       "      <td>N26\\tNCBI_Taxonomy Annotation:T20 Referent:2104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>52</th>\n",
       "      <td>N27\\tOntoBiotope Annotation:T21 Referent:OBT:0...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>53</th>\n",
       "      <td>N28\\tOntoBiotope Annotation:T21 Referent:OBT:0...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>54</th>\n",
       "      <td>N29\\tOntoBiotope Annotation:T22 Referent:OBT:0...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>55</th>\n",
       "      <td>N30\\tOntoBiotope Annotation:T23 Referent:OBT:0...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>56</th>\n",
       "      <td>N31\\tOntoBiotope Annotation:T23 Referent:OBT:0...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>57</th>\n",
       "      <td>N32\\tNCBI_Taxonomy Annotation:T24 Referent:12814</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>58</th>\n",
       "      <td>N33\\tNCBI_Taxonomy Annotation:T25 Referent:2104</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>59</th>\n",
       "      <td>N34\\tOntoBiotope Annotation:T26 Referent:OBT:0...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>60</th>\n",
       "      <td>N35\\tOntoBiotope Annotation:T26 Referent:OBT:0...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>61</th>\n",
       "      <td>N36\\tOntoBiotope Annotation:T27 Referent:OBT:0...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>62</th>\n",
       "      <td>N37\\tOntoBiotope Annotation:T28 Referent:OBT:0...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>63</th>\n",
       "      <td>N38\\tOntoBiotope Annotation:T29 Referent:OBT:0...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>64 rows  1 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                         T3\\tHabitat 61 70\\tpediatric\n",
       "0                    T4\\tHabitat 178 189\\trespiratory\n",
       "1                    T5\\tHabitat 256 267\\trespiratory\n",
       "2                      T6\\tHabitat 281 290\\tpediatric\n",
       "3   T7\\tHabitat 339 372\\tchildren less than 2 year...\n",
       "4   T8\\tHabitat 418 488\\tchildren greater than 9 y...\n",
       "5                    T9\\tHabitat 469 480\\trespiratory\n",
       "6                         T10\\tHabitat 490 495\\tMales\n",
       "7                       T11\\tHabitat 543 550\\tfemales\n",
       "8                         T12\\tHabitat 573 578\\tmales\n",
       "9       T13\\tPhenotype 600 621\\tnonbacterial pathogen\n",
       "10           T14\\tHabitat 648 666\\tpatients with WARI\n",
       "11                  T15\\tHabitat 668 679\\trespiratory\n",
       "12  T16\\tMicroorganism 668 695\\trespiratory syncyt...\n",
       "13  T17\\tMicroorganism 697 724\\tparainfluenza viru...\n",
       "14  T18\\tMicroorganism 697 722;729 730\\tparainflue...\n",
       "15           T19\\tMicroorganism 732 744\\tadenoviruses\n",
       "16  T20\\tMicroorganism 750 771\\tMycoplasma pneumoniae\n",
       "17                      T21\\tHabitat 807 814\\tPatient\n",
       "18  T22\\tPhenotype 887 933\\tcause of WARI in child...\n",
       "19  T23\\tHabitat 904 933\\tchildren under 5 years o...\n",
       "20                    T24\\tMicroorganism 938 941\\tRSV\n",
       "21  T25\\tMicroorganism 950 971\\tMycoplasma pneumoniae\n",
       "22  T26\\tHabitat 1007 1048\\tschool age children wi...\n",
       "23             T27\\tHabitat 1124 1138\\tdiagnosticians\n",
       "24                T28\\tHabitat 1146 1157\\tresearchers\n",
       "25                T29\\tHabitat 1193 1204\\trespiratory\n",
       "26  N1\\tOntoBiotope Annotation:T3 Referent:OBT:003188\n",
       "27  N2\\tOntoBiotope Annotation:T3 Referent:OBT:003220\n",
       "28  N3\\tOntoBiotope Annotation:T4 Referent:OBT:000407\n",
       "29  N4\\tOntoBiotope Annotation:T5 Referent:OBT:000407\n",
       "..                                                ...\n",
       "34  N9\\tOntoBiotope Annotation:T8 Referent:OBT:003220\n",
       "35  N10\\tOntoBiotope Annotation:T8 Referent:OBT:00...\n",
       "36  N11\\tOntoBiotope Annotation:T9 Referent:OBT:00...\n",
       "37  N12\\tOntoBiotope Annotation:T10 Referent:OBT:0...\n",
       "38  N13\\tOntoBiotope Annotation:T10 Referent:OBT:0...\n",
       "39  N14\\tOntoBiotope Annotation:T11 Referent:OBT:0...\n",
       "40  N15\\tOntoBiotope Annotation:T11 Referent:OBT:0...\n",
       "41  N16\\tOntoBiotope Annotation:T12 Referent:OBT:0...\n",
       "42  N17\\tOntoBiotope Annotation:T12 Referent:OBT:0...\n",
       "43  N18\\tOntoBiotope Annotation:T13 Referent:OBT:0...\n",
       "44  N19\\tOntoBiotope Annotation:T14 Referent:OBT:0...\n",
       "45  N20\\tOntoBiotope Annotation:T14 Referent:OBT:0...\n",
       "46  N21\\tOntoBiotope Annotation:T15 Referent:OBT:0...\n",
       "47   N22\\tNCBI_Taxonomy Annotation:T16 Referent:12814\n",
       "48   N23\\tNCBI_Taxonomy Annotation:T17 Referent:12730\n",
       "49   N24\\tNCBI_Taxonomy Annotation:T18 Referent:11216\n",
       "50   N25\\tNCBI_Taxonomy Annotation:T19 Referent:10508\n",
       "51    N26\\tNCBI_Taxonomy Annotation:T20 Referent:2104\n",
       "52  N27\\tOntoBiotope Annotation:T21 Referent:OBT:0...\n",
       "53  N28\\tOntoBiotope Annotation:T21 Referent:OBT:0...\n",
       "54  N29\\tOntoBiotope Annotation:T22 Referent:OBT:0...\n",
       "55  N30\\tOntoBiotope Annotation:T23 Referent:OBT:0...\n",
       "56  N31\\tOntoBiotope Annotation:T23 Referent:OBT:0...\n",
       "57   N32\\tNCBI_Taxonomy Annotation:T24 Referent:12814\n",
       "58    N33\\tNCBI_Taxonomy Annotation:T25 Referent:2104\n",
       "59  N34\\tOntoBiotope Annotation:T26 Referent:OBT:0...\n",
       "60  N35\\tOntoBiotope Annotation:T26 Referent:OBT:0...\n",
       "61  N36\\tOntoBiotope Annotation:T27 Referent:OBT:0...\n",
       "62  N37\\tOntoBiotope Annotation:T28 Referent:OBT:0...\n",
       "63  N38\\tOntoBiotope Annotation:T29 Referent:OBT:0...\n",
       "\n",
       "[64 rows x 1 columns]"
      ]
     },
     "execution_count": 26,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pd.read_csv('BioNLP-OST-2019_BB-norm+ner_train/BB-norm+ner-448557.a2')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>N1\tOntoBiotope Annotation:T3 Referent:OBT:000019</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>N2\\tNCBI_Taxonomy Annotation:T4 Referent:208964</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>N3\\tOntoBiotope Annotation:T5 Referent:OBT:000019</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>N4\\tNCBI_Taxonomy Annotation:T6 Referent:1010796</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   N1\\tOntoBiotope Annotation:T3 Referent:OBT:000019\n",
       "0    N2\\tNCBI_Taxonomy Annotation:T4 Referent:208964\n",
       "1  N3\\tOntoBiotope Annotation:T5 Referent:OBT:000019\n",
       "2   N4\\tNCBI_Taxonomy Annotation:T6 Referent:1010796"
      ]
     },
     "execution_count": 27,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pd.read_csv('C:/Users/user/Desktop/BioNLP-OST-2019_BB-norm_test/BB-norm-132046.a2')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
